<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 1.5.4">
<meta name="author" content="Natalie Catlett">
<title>BEL 2.0 Specification</title>
<style>
@import url(http://red-hat-overpass-fonts.s3.amazonaws.com/overpass.css);

// Theming built off rocket-panda.css stylesheet. Thanks!

/*! normalize.css v2.1.2 | MIT License | git.io/normalize */


/* ========================================================================== HTML5 display definitions ========================================================================== */


/** Correct `block` display not defined in IE 8/9. */

article,
aside,
details,
figcaption,
figure,
footer,
header,
hgroup,
main,
nav,
section,
summary {
    display: block;
}


/** Correct `inline-block` display not defined in IE 8/9. */

audio,
canvas,
video {
    display: inline-block;
}


/** Prevent modern browsers from displaying `audio` without controls. Remove excess height in iOS 5 devices. */

audio:not([controls]) {
    display: none;
    height: 0;
}


/** Address `[hidden]` styling not present in IE 8/9. Hide the `template` element in IE, Safari, and Firefox < 22. */

[hidden],
template {
    display: none;
}

script {
    display: none !important;
}


/* ========================================================================== Base ========================================================================== */


/** 1. Set default font family to sans-serif. 2. Prevent iOS text size adjust after orientation change, without disabling user zoom. */

html {
    font-family: sans-serif;
    /* 1 */
    -ms-text-size-adjust: 100%;
    /* 2 */
    -webkit-text-size-adjust: 100%;
    /* 2 */
}


/** Remove default margin. */

body {
    margin: 0;
}


/* ========================================================================== Links ========================================================================== */


/** Remove the gray background color from active links in IE 10. */

a {
    background: transparent;
}


/** Address `outline` inconsistency between Chrome and other browsers. */

a:focus {
    outline: thin dotted;
}


/** Improve readability when focused and also mouse hovered in all browsers. */

a:active,
a:hover {
    outline: 0;
}


/* ========================================================================== Typography ========================================================================== */


/** Address variable `h1` font-size and margin within `section` and `article` contexts in Firefox 4+, Safari 5, and Chrome. */

h1 {
    font-size: 2em;
    margin: 0.67em 0;
}


/** Address styling not present in IE 8/9, Safari 5, and Chrome. */

abbr[title] {
    border-bottom: 1px dotted;
}


/** Address style set to `bolder` in Firefox 4+, Safari 5, and Chrome. */

b,
strong {
    font-weight: bold;
}


/** Address styling not present in Safari 5 and Chrome. */

dfn {
    font-style: italic;
}


/** Address differences between Firefox and other browsers. */

hr {
    -moz-box-sizing: content-box;
    box-sizing: content-box;
    height: 0;
}


/** Address styling not present in IE 8/9. */

mark {
    background: #ff0;
    color: #000;
}


/** Correct font family set oddly in Safari 5 and Chrome. */

code,
kbd,
pre,
samp {
    font-family: monospace, serif;
    font-size: 1em;
}


/** Improve readability of pre-formatted text in all browsers. */

pre {
    white-space: pre-wrap;
}


/** Set consistent quote types. */

q {
    quotes: "\201C" "\201D" "\2018" "\2019";
}


/** Address inconsistent and variable font size in all browsers. */

small {
    font-size: 80%;
}


/** Prevent `sub` and `sup` affecting `line-height` in all browsers. */

sub,
sup {
    font-size: 75%;
    line-height: 0;
    position: relative;
    vertical-align: baseline;
}

sup {
    top: -0.5em;
}

sub {
    bottom: -0.25em;
}


/* ========================================================================== Embedded content ========================================================================== */


/** Remove border when inside `a` element in IE 8/9. */

img {
    border: 0;
}


/** Correct overflow displayed oddly in IE 9. */

svg:not(:root) {
    overflow: hidden;
}


/* ========================================================================== Figures ========================================================================== */


/** Address margin not present in IE 8/9 and Safari 5. */

figure {
    margin: 0;
}


/* ========================================================================== Forms ========================================================================== */


/** Define consistent border, margin, and padding. */

fieldset {
    border: 1px solid #c0c0c0;
    margin: 0 2px;
    padding: 0.35em 0.625em 0.75em;
}


/** 1. Correct `color` not being inherited in IE 8/9. 2. Remove padding so people aren't caught out if they zero out fieldsets. */

legend {
    border: 0;
    /* 1 */
    padding: 0;
    /* 2 */
}


/** 1. Correct font family not being inherited in all browsers. 2. Correct font size not being inherited in all browsers. 3. Address margins set differently in Firefox 4+, Safari 5, and Chrome. */

button,
input,
select,
textarea {
    font-family: inherit;
    /* 1 */
    font-size: 100%;
    /* 2 */
    margin: 0;
    /* 3 */
}


/** Address Firefox 4+ setting `line-height` on `input` using `!important` in the UA stylesheet. */

button,
input {
    line-height: normal;
}


/** Address inconsistent `text-transform` inheritance for `button` and `select`. All other form control elements do not inherit `text-transform` values. Correct `button` style inheritance in Chrome, Safari 5+, and IE 8+. Correct `select` style inheritance in Firefox 4+ and Opera. */

button,
select {
    text-transform: none;
}


/** 1. Avoid the WebKit bug in Android 4.0.* where (2) destroys native `audio` and `video` controls. 2. Correct inability to style clickable `input` types in iOS. 3. Improve usability and consistency of cursor style between image-type `input` and others. */

button,
html input[type="button"],
input[type="reset"],
input[type="submit"] {
    -webkit-appearance: button;
    /* 2 */
    cursor: pointer;
    /* 3 */
}


/** Re-set default cursor for disabled elements. */

button[disabled],
html input[disabled] {
    cursor: default;
}


/** 1. Address box sizing set to `content-box` in IE 8/9. 2. Remove excess padding in IE 8/9. */

input[type="checkbox"],
input[type="radio"] {
    box-sizing: border-box;
    /* 1 */
    padding: 0;
    /* 2 */
}


/** 1. Address `appearance` set to `searchfield` in Safari 5 and Chrome. 2. Address `box-sizing` set to `border-box` in Safari 5 and Chrome (include `-moz` to future-proof). */

input[type="search"] {
    -webkit-appearance: textfield;
    /* 1 */
    -moz-box-sizing: content-box;
    -webkit-box-sizing: content-box;
    /* 2 */
    box-sizing: content-box;
}


/** Remove inner padding and search cancel button in Safari 5 and Chrome on OS X. */

input[type="search"]::-webkit-search-cancel-button,
input[type="search"]::-webkit-search-decoration {
    -webkit-appearance: none;
}


/** Remove inner padding and border in Firefox 4+. */

button::-moz-focus-inner,
input::-moz-focus-inner {
    border: 0;
    padding: 0;
}


/** 1. Remove default vertical scrollbar in IE 8/9. 2. Improve readability and alignment in all browsers. */

textarea {
    overflow: auto;
    /* 1 */
    vertical-align: top;
    /* 2 */
}


/* ========================================================================== Tables ========================================================================== */


/** Remove most spacing between table cells. */

table {
    border-collapse: collapse;
    border-spacing: 0;
}

meta.foundation-mq-small {
    font-family: "only screen and (min-width: 768px)";
    width: 768px;
}

meta.foundation-mq-medium {
    font-family: "only screen and (min-width:1280px)";
    width: 1280px;
}

meta.foundation-mq-large {
    font-family: "only screen and (min-width:1440px)";
    width: 1440px;
}

*,
*:before,
*:after {
    -moz-box-sizing: border-box;
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
}

html,
body {
    font-size: 100%;
}

body {
    background: #e5e5e5;
    color: #222222;
    padding: 0;
    margin: 0;
    font-family: "Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif;
    font-weight: normal;
    font-style: normal;
    line-height: 1;
    position: relative;
    cursor: auto;
}

a:hover {
    cursor: pointer;
}

img,
object,
embed {
    max-width: 100%;
    height: auto;
}

object,
embed {
    height: 100%;
}

img {
    -ms-interpolation-mode: bicubic;
}

#map_canvas img,
#map_canvas embed,
#map_canvas object,
.map_canvas img,
.map_canvas embed,
.map_canvas object {
    max-width: none !important;
}

.left {
    float: left !important;
}

.right {
    float: right !important;
}

.text-left {
    text-align: left !important;
}

.text-right {
    text-align: right !important;
}

.text-center {
    text-align: center !important;
}

.text-justify {
    text-align: justify !important;
}

.hide {
    display: none;
}

.antialiased,
body {
    -webkit-font-smoothing: antialiased;
}

img {
    display: inline-block;
    vertical-align: middle;
}

textarea {
    height: auto;
    min-height: 50px;
}

select {
    width: 100%;
}

object,
svg {
    display: inline-block;
    vertical-align: middle;
}

.center {
    margin-left: auto;
    margin-right: auto;
}

.spread {
    width: 100%;
}

p.lead,
.paragraph.lead > p,
#preamble > .sectionbody > .paragraph:first-of-type p {
    font-size: 1.15625em;
    line-height: 1.6;
}

.subheader,
.admonitionblock td.content > .title,
.audioblock > .title,
.exampleblock > .title,
.imageblock > .title,
.listingblock > .title,
.literalblock > .title,
.stemblock > .title,
.openblock > .title,
.paragraph > .title,
.quoteblock > .title,
table.tableblock > .title,
.verseblock > .title,
.videoblock > .title,
.dlist > .title,
.olist > .title,
.ulist > .title,
.qlist > .title,
.hdlist > .title {
    line-height: 1.4;
    color: #4d4d4d;
    font-weight: 300;
    margin-top: 0.2em;
    margin-bottom: 0.5em;
}


/* Typography resets */

div,
dl,
dt,
dd,
ul,
ol,
li,
h1,
h2,
h3,
#toctitle,
.sidebarblock > .content > .title,
h4,
h5,
h6,
pre,
form,
p,
blockquote,
th,
td {
    margin: 0;
    padding: 0;
    direction: ltr;
}


/* Default Link Styles */

a {
    color: #006699;
    text-decoration: none;
    line-height: inherit;
}

a:hover,
a:focus {
    color: #ea0011;
}

a img {
    border: none;
}


/* Default paragraph styles */

p {
    font-family: "Helvetica Neue", Helvetica, "Liberation Sans", Arial, sans-serif;
    font-weight: normal;
    font-size: 0.9375em;
    line-height: 1.4;
    margin-bottom: 1.375em;
    text-rendering: optimizeLegibility;
}

p aside {
    font-size: 0.875em;
    line-height: 1.35;
    font-style: italic;
}


/* Default header styles */

h1,
h2,
h3,
#toctitle,
.sidebarblock > .content > .title,
h4,
h5,
h6 {
    font-family: Overpass, "Helvetica Neue", Helvetica, Arial, sans-serif;
    font-weight: normal;
    font-style: normal;
    color: black;
    text-rendering: optimizeLegibility;
    margin-top: 1em;
    margin-bottom: 0.5em;
    line-height: 1.2125em;
}

h1 small,
h2 small,
h3 small,
#toctitle small,
.sidebarblock > .content > .title small,
h4 small,
h5 small,
h6 small {
    font-size: 60%;
    color: #4d4d4d;
    line-height: 0;
}

h1 {
    font-size: 2.125em;
}

h2 {
    font-size: 1.6875em;
}

h3,
#toctitle,
.sidebarblock > .content > .title {
    font-size: 1.375em;
}

h4 {
    font-size: 1.125em;
}

h5 {
    font-size: 1.125em;
}

h6 {
    font-size: 1em;
}

hr {
    border: solid #dddddd;
    border-width: 1px 0 0;
    clear: both;
    margin: 1.25em 0 1.1875em;
    height: 0;
}


/* Helpful Typography Defaults */

em,
i {
    font-style: italic;
    line-height: inherit;
}

strong,
b {
    font-weight: bold;
    line-height: inherit;
}

small {
    font-size: 60%;
    line-height: inherit;
}

code {
    font-family: Menlo, Monaco, "Liberation Mono", Consolas, monospace;
    font-weight: normal;
    color: #3b3b3b;
}


/* Lists */

ul,
ol,
dl {
    font-size: 0.9375em;
    line-height: 1.4;
    margin-bottom: 1.375em;
    list-style-position: outside;
    font-family: "Helvetica Neue", Helvetica, "Liberation Sans", Arial, sans-serif;
}

ul,
ol {
    margin-left: 1.625em;
}

ul.no-bullet,
ol.no-bullet {
    margin-left: 1.625em;
}


/* Unordered Lists */

ul li ul,
ul li ol {
    margin-left: 1.25em;
    margin-bottom: 0;
    font-size: 1em;
    /* Override nested font-size change */
}

ul.square li ul,
ul.circle li ul,
ul.disc li ul {
    list-style: inherit;
}

ul.square {
    list-style-type: square;
}

ul.circle {
    list-style-type: circle;
}

ul.disc {
    list-style-type: disc;
}

ul.no-bullet {
    list-style: none;
}


/* Ordered Lists */

ol li ul,
ol li ol {
    margin-left: 1.25em;
    margin-bottom: 0;
}


/* Definition Lists */

dl dt {
    margin-bottom: 0.3em;
    font-weight: bold;
}

dl dd {
    margin-bottom: 0.75em;
}


/* Abbreviations */

abbr,
acronym {
    text-transform: uppercase;
    font-size: 90%;
    color: #404040;
    border-bottom: 1px dotted #dddddd;
    cursor: help;
}

abbr {
    text-transform: none;
}


/* Blockquotes */

blockquote {
    margin: 0 0 1.375em;
    padding: 0 0 0 1em;
    border-left: 5px solid #ededed;
}

blockquote cite {
    display: block;
    font-size: 0.8125em;
    color: #333333;
}

blockquote cite:before {
    content: "\2014 \0020";
}

blockquote cite a,
blockquote cite a:visited {
    color: #333333;
}

blockquote,
blockquote p {
    line-height: 1.4;
    color: #404040;
}


/* Microformats */

.vcard {
    display: inline-block;
    margin: 0 0 1.25em 0;
    border: 1px solid #dddddd;
    padding: 0.625em 0.75em;
}

.vcard li {
    margin: 0;
    display: block;
}

.vcard .fn {
    font-weight: bold;
    font-size: 0.9375em;
}

.vevent .summary {
    font-weight: bold;
}

.vevent abbr {
    cursor: auto;
    text-decoration: none;
    font-weight: bold;
    border: none;
    padding: 0 0.0625em;
}

@media only screen and (min-width: 768px) {
    h1,
    h2,
    h3,
    #toctitle,
    .sidebarblock > .content > .title,
    h4,
    h5,
    h6 {
        line-height: 1.4;
    }
    h1 {
        font-size: 2.75em;
    }
    h2 {
        font-size: 2.3125em;
    }
    h3,
    #toctitle,
    .sidebarblock > .content > .title {
        font-size: 1.6875em;
    }
    h4 {
        font-size: 1.4375em;
    }
}


/* Tables */

table {
    background: white;
    margin-bottom: 1.25em;
    border: solid 1px #dddddd;
}

table thead,
table tfoot {
    background: whitesmoke;
    font-weight: bold;
}

table thead tr th,
table thead tr td,
table tfoot tr th,
table tfoot tr td {
    padding: 0.5em 0.625em 0.625em;
    font-size: inherit;
    color: #222222;
    text-align: left;
}

table tr th,
table tr td {
    padding: 0.5625em 0.625em;
    font-size: inherit;
    color: #222222;
}

table tr.even,
table tr.alt,
table tr:nth-of-type(even) {
    background: #f9f9f9;
}

table thead tr th,
table tfoot tr th,
table tbody tr td,
table tr td,
table tfoot tr td {
    display: table-cell;
    line-height: 1.4;
}

body {
    tab-size: 4;
}

h1,
h2,
h3,
#toctitle,
.sidebarblock > .content > .title,
h4,
h5,
h6 {
    line-height: 1.4;
}

a:hover,
a:focus {
    text-decoration: underline;
}

.clearfix:before,
.clearfix:after,
.float-group:before,
.float-group:after {
    content: " ";
    display: table;
}

.clearfix:after,
.float-group:after {
    clear: both;
}

*:not(pre) > code {
    font-size: 0.9375em;
    font-style: normal !important;
    letter-spacing: 0;
    padding: 1px 4px;
    background-color: #eff1f1;
    border: 1px solid #d4d9d9;
    -webkit-border-radius: 0;
    border-radius: 0;
    line-height: inherit;
}

pre,
pre > code {
    line-height: 1.6;
    color: #3b3b3b;
    font-family: Menlo, Monaco, "Liberation Mono", Consolas, monospace;
    font-weight: normal;
}

.keyseq {
    color: #737373;
}

kbd {
    font-family: Menlo, Monaco, "Liberation Mono", Consolas, monospace;
    display: inline-block;
    color: #404040;
    font-size: 0.65em;
    line-height: 1.45;
    background-color: #f7f7f7;
    border: 1px solid #ccc;
    -webkit-border-radius: 3px;
    border-radius: 3px;
    -webkit-box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 0.1em white inset;
    box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 0.1em white inset;
    margin: 0 0.15em;
    padding: 0.2em 0.5em;
    vertical-align: middle;
    position: relative;
    top: -0.1em;
    white-space: nowrap;
}

.keyseq kbd:first-child {
    margin-left: 0;
}

.keyseq kbd:last-child {
    margin-right: 0;
}

.menuseq,
.menu {
    color: #272727;
}

b.button:before,
b.button:after {
    position: relative;
    top: -1px;
    font-weight: normal;
}

b.button:before {
    content: "[";
    padding: 0 3px 0 2px;
}

b.button:after {
    content: "]";
    padding: 0 2px 0 3px;
}

#header,
#content,
#footnotes,
#footer {
    width: 100%;
    margin-left: auto;
    margin-right: auto;
    margin-top: 0;
    margin-bottom: 0;
    max-width: 62.5em;
    *zoom: 1;
    position: relative;
    padding-left: 0.9375em;
    padding-right: 0.9375em;
}

#header:before,
#header:after,
#content:before,
#content:after,
#footnotes:before,
#footnotes:after,
#footer:before,
#footer:after {
    content: " ";
    display: table;
}

#header:after,
#content:after,
#footnotes:after,
#footer:after {
    clear: both;
}

#content {
    margin-top: 1.25em;
}

#content:before {
    content: none;
}

#header > h1:first-child {
    color: black;
    margin-top: 2.25rem;
    margin-bottom: 0;
}

#header > h1:first-child + #toc {
    margin-top: 8px;
    border-top: 1px solid #dddddd;
}

#header > h1:only-child,
body.toc2 #header > h1:nth-last-child(2) {
    border-bottom: 1px solid #dddddd;
    padding-bottom: 8px;
}

#header .details {
    border-bottom: 1px solid #dddddd;
    line-height: 1.45;
    padding-top: 0.25em;
    padding-bottom: 0.25em;
    padding-left: 0.25em;
    color: #333333;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    -ms-flex-flow: row wrap;
    -webkit-flex-flow: row wrap;
    flex-flow: row wrap;
}

#header .details span:first-child {
    margin-left: -0.125em;
}

#header .details span.email a {
    color: #404040;
}

#header .details br {
    display: none;
}

#header .details br + span:before {
    content: "\00a0\2013\00a0";
}

#header .details br + span.author:before {
    content: "\00a0\22c5\00a0";
    color: #404040;
}

#header .details br + span#revremark:before {
    content: "\00a0|\00a0";
}

#header #revnumber {
    text-transform: capitalize;
}

#header #revnumber:after {
    content: "\00a0";
}

#content > h1:first-child:not([class]) {
    color: black;
    border-bottom: 1px solid #dddddd;
    padding-bottom: 8px;
    margin-top: 0;
    padding-top: 1rem;
    margin-bottom: 1.25rem;
}

#toc {
    border-bottom: 0 solid #dddddd;
    padding-bottom: 0.5em;
}

#toc > ul {
    margin-left: 0.125em;
}

#toc ul.sectlevel0 > li > a {
    font-style: italic;
}

#toc ul.sectlevel0 ul.sectlevel1 {
    margin: 0.5em 0;
}

#toc ul {
    font-family: Overpass, "Helvetica Neue", Helvetica, Arial, sans-serif;
    list-style-type: none;
}

#toc li {
    line-height: 1.3334;
    margin-top: 0.3334em;
}

#toc a {
    text-decoration: none;
}

#toc a:active {
    text-decoration: underline;
}

#toctitle {
    color: #4d4d4d;
    font-size: 1.2em;
}

@media only screen and (min-width: 768px) {
    #toctitle {
        font-size: 1.375em;
    }
    body.toc2 {
        padding-left: 15em;
        padding-right: 0;
    }
    #toc.toc2 {
        margin-top: 0 !important;
        background-color: #e5e5e5;
        position: fixed;
        width: 15em;
        left: 0;
        top: 0;
        border-right: 1px solid #dddddd;
        border-top-width: 0 !important;
        border-bottom-width: 0 !important;
        z-index: 1000;
        padding: 1.25em 1em;
        height: 100%;
        overflow: auto;
    }
    #toc.toc2 #toctitle {
        margin-top: 0;
        margin-bottom: 0.8rem;
        font-size: 1.2em;
    }
    #toc.toc2 > ul {
        font-size: 0.9em;
        margin-bottom: 0;
    }
    #toc.toc2 ul ul {
        margin-left: 0;
        padding-left: 1em;
    }
    #toc.toc2 ul.sectlevel0 ul.sectlevel1 {
        padding-left: 0;
        margin-top: 0.5em;
        margin-bottom: 0.5em;
    }
    body.toc2.toc-right {
        padding-left: 0;
        padding-right: 15em;
    }
    body.toc2.toc-right #toc.toc2 {
        border-right-width: 0;
        border-left: 1px solid #dddddd;
        left: auto;
        right: 0;
    }
}

@media only screen and (min-width: 1280px) {
    body.toc2 {
        padding-left: 20em;
        padding-right: 0;
    }
    #toc.toc2 {
        width: 20em;
    }
    #toc.toc2 #toctitle {
        font-size: 1.375em;
    }
    #toc.toc2 > ul {
        font-size: 0.95em;
    }
    #toc.toc2 ul ul {
        padding-left: 1.25em;
    }
    body.toc2.toc-right {
        padding-left: 0;
        padding-right: 20em;
    }
}

#content #toc {
    border-style: solid;
    border-width: 1px;
    border-color: #cccccc;
    margin-bottom: 1.25em;
    padding: 1.25em;
    background: #e5e5e5;
    -webkit-border-radius: 0;
    border-radius: 0;
}

#content #toc >:first-child {
    margin-top: 0;
}

#content #toc >:last-child {
    margin-bottom: 0;
}

#footer {
    max-width: 100%;
    background-color: white;
    padding: 1.25em;
}

#footer-text {
    color: #666666;
    line-height: 1.26;
}

.sect1 {
    padding-bottom: 0.625em;
}

@media only screen and (min-width: 768px) {
    .sect1 {
        padding-bottom: 1.25em;
    }
}

.sect1 + .sect1 {
    border-top: 0 solid #dddddd;
}

#content h1 > a.anchor,
h2 > a.anchor,
h3 > a.anchor,
#toctitle > a.anchor,
.sidebarblock > .content > .title > a.anchor,
h4 > a.anchor,
h5 > a.anchor,
h6 > a.anchor {
    position: absolute;
    z-index: 1001;
    width: 1.5ex;
    margin-left: -1.5ex;
    display: block;
    text-decoration: none !important;
    visibility: hidden;
    text-align: center;
    font-weight: normal;
}

#content h1 > a.anchor:before,
h2 > a.anchor:before,
h3 > a.anchor:before,
#toctitle > a.anchor:before,
.sidebarblock > .content > .title > a.anchor:before,
h4 > a.anchor:before,
h5 > a.anchor:before,
h6 > a.anchor:before {
    content: "\00A7";
    font-size: 0.85em;
    display: block;
    padding-top: 0.1em;
}

#content h1:hover > a.anchor,
#content h1 > a.anchor:hover,
h2:hover > a.anchor,
h2 > a.anchor:hover,
h3:hover > a.anchor,
#toctitle:hover > a.anchor,
.sidebarblock > .content > .title:hover > a.anchor,
h3 > a.anchor:hover,
#toctitle > a.anchor:hover,
.sidebarblock > .content > .title > a.anchor:hover,
h4:hover > a.anchor,
h4 > a.anchor:hover,
h5:hover > a.anchor,
h5 > a.anchor:hover,
h6:hover > a.anchor,
h6 > a.anchor:hover {
    visibility: visible;
}

#content h1 > a.link,
h2 > a.link,
h3 > a.link,
#toctitle > a.link,
.sidebarblock > .content > .title > a.link,
h4 > a.link,
h5 > a.link,
h6 > a.link {
    color: black;
    text-decoration: none;
}

#content h1 > a.link:hover,
h2 > a.link:hover,
h3 > a.link:hover,
#toctitle > a.link:hover,
.sidebarblock > .content > .title > a.link:hover,
h4 > a.link:hover,
h5 > a.link:hover,
h6 > a.link:hover {
    color: black;
}

.audioblock,
.imageblock,
.literalblock,
.listingblock,
.stemblock,
.videoblock {
    margin-bottom: 1.25em;
}

.admonitionblock td.content > .title,
.audioblock > .title,
.exampleblock > .title,
.imageblock > .title,
.listingblock > .title,
.literalblock > .title,
.stemblock > .title,
.openblock > .title,
.paragraph > .title,
.quoteblock > .title,
table.tableblock > .title,
.verseblock > .title,
.videoblock > .title,
.dlist > .title,
.olist > .title,
.ulist > .title,
.qlist > .title,
.hdlist > .title {
    text-rendering: optimizeLegibility;
    text-align: left;
}

table.tableblock > caption.title {
    white-space: nowrap;
    overflow: visible;
    max-width: 0;
}

.paragraph.lead > p,
#preamble > .sectionbody > .paragraph:first-of-type p {
    color: black;
}

table.tableblock #preamble > .sectionbody > .paragraph:first-of-type p {
    font-size: inherit;
}

.admonitionblock > table {
    border-collapse: separate;
    border: 0;
    background: none;
    width: 100%;
}

.admonitionblock > table td.icon {
    text-align: center;
    width: 80px;
}

.admonitionblock > table td.icon img {
    max-width: none;
}

.admonitionblock > table td.icon .title {
    font-weight: bold;
    font-family: Overpass, "Helvetica Neue", Helvetica, Arial, sans-serif;
    text-transform: uppercase;
}

.admonitionblock > table td.content {
    padding-left: 1.125em;
    padding-right: 1.25em;
    border-left: 1px solid #dddddd;
    color: #333333;
}

.admonitionblock > table td.content >:last-child >:last-child {
    margin-bottom: 0;
}

.exampleblock > .content {
    border-style: solid;
    border-width: 1px;
    border-color: #cccccc;
    margin-bottom: 1.25em;
    padding: 1.25em;
    background: #e5e5e5;
    -webkit-border-radius: 0;
    border-radius: 0;
}

.exampleblock > .content >:first-child {
    margin-top: 0;
}

.exampleblock > .content >:last-child {
    margin-bottom: 0;
}

.sidebarblock {
    border-style: solid;
    border-width: 1px;
    border-color: #cccccc;
    margin-bottom: 1.25em;
    padding: 1.25em;
    background: #e5e5e5;
    -webkit-border-radius: 0;
    border-radius: 0;
}

.sidebarblock >:first-child {
    margin-top: 0;
}

.sidebarblock >:last-child {
    margin-bottom: 0;
}

.sidebarblock > .content > .title {
    color: #4d4d4d;
    margin-top: 0;
}

.exampleblock > .content >:last-child >:last-child,
.exampleblock > .content .olist > ol > li:last-child >:last-child,
.exampleblock > .content .ulist > ul > li:last-child >:last-child,
.exampleblock > .content .qlist > ol > li:last-child >:last-child,
.sidebarblock > .content >:last-child >:last-child,
.sidebarblock > .content .olist > ol > li:last-child >:last-child,
.sidebarblock > .content .ulist > ul > li:last-child >:last-child,
.sidebarblock > .content .qlist > ol > li:last-child >:last-child {
    margin-bottom: 0;
}

.literalblock pre,
.listingblock pre:not(.highlight),
.listingblock pre[class="highlight"],
.listingblock pre[class^="highlight "],
.listingblock pre.CodeRay,
.listingblock pre.prettyprint {
    background: #eff1f1;
}

.sidebarblock .literalblock pre,
.sidebarblock .listingblock pre:not(.highlight),
.sidebarblock .listingblock pre[class="highlight"],
.sidebarblock .listingblock pre[class^="highlight "],
.sidebarblock .listingblock pre.CodeRay,
.sidebarblock .listingblock pre.prettyprint {
    background: #f2f1f1;
}

.literalblock pre,
.literalblock pre[class],
.listingblock pre,
.listingblock pre[class] {
    border: 0 0 0 2px solid rgba(120, 120, 120, 0.35);
    -webkit-border-radius: 0;
    border-radius: 0;
    word-wrap: break-word;
    padding: 1em 1.5em 0.875em 1.5em;
    font-size: 0.8125em;
}

.literalblock pre.nowrap,
.literalblock pre[class].nowrap,
.listingblock pre.nowrap,
.listingblock pre[class].nowrap {
    overflow-x: auto;
    white-space: pre;
    word-wrap: normal;
}

@media only screen and (min-width: 768px) {
    .literalblock pre,
    .literalblock pre[class],
    .listingblock pre,
    .listingblock pre[class] {
        font-size: 0.90625em;
    }
}

@media only screen and (min-width: 1280px) {
    .literalblock pre,
    .literalblock pre[class],
    .listingblock pre,
    .listingblock pre[class] {
        font-size: 1em;
    }
}

.literalblock.output pre {
    color: #eff1f1;
    background-color: #3b3b3b;
}

.listingblock pre.highlightjs {
    padding: 0;
}

.listingblock pre.highlightjs > code {
    padding: 1em 1.5em 0.875em 1.5em;
    -webkit-border-radius: 0;
    border-radius: 0;
}

.listingblock > .content {
    position: relative;
}

.listingblock code[data-lang]:before {
    display: none;
    content: attr(data-lang);
    position: absolute;
    font-size: 0.75em;
    top: 0.425rem;
    right: 0.5rem;
    line-height: 1;
    text-transform: uppercase;
    color: #999;
}

.listingblock:hover code[data-lang]:before {
    display: block;
}

.listingblock.terminal pre .command:before {
    content: attr(data-prompt);
    padding-right: 0.5em;
    color: #999;
}

.listingblock.terminal pre .command:not([data-prompt]):before {
    content: "$";
}

table.pyhltable {
    border-collapse: separate;
    border: 0;
    margin-bottom: 0;
    background: none;
}

table.pyhltable td {
    vertical-align: top;
    padding-top: 0;
    padding-bottom: 0;
    line-height: 1.6;
}

table.pyhltable td.code {
    padding-left: .75em;
    padding-right: 0;
}

pre.pygments .lineno,
table.pyhltable td:not(.code) {
    color: #999;
    padding-left: 0;
    padding-right: .5em;
    border-right: 1px solid #dddddd;
}

pre.pygments .lineno {
    display: inline-block;
    margin-right: .25em;
}

table.pyhltable .linenodiv {
    background: none !important;
    padding-right: 0 !important;
}

.quoteblock {
    margin: 0 1em 1.375em 1.5em;
    display: table;
}

.quoteblock > .title {
    margin-left: -1.5em;
    margin-bottom: 0.75em;
}

.quoteblock blockquote,
.quoteblock blockquote p {
    color: #404040;
    font-size: 1.15rem;
    line-height: 1.75;
    word-spacing: 0.1em;
    letter-spacing: 0;
    font-style: italic;
    text-align: justify;
}

.quoteblock blockquote {
    margin: 0;
    padding: 0;
    border: 0;
}

.quoteblock blockquote:before {
    content: "\201c";
    float: left;
    font-size: 2.75em;
    font-weight: bold;
    line-height: 0.6em;
    margin-left: -0.6em;
    color: #4d4d4d;
    text-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
}

.quoteblock blockquote > .paragraph:last-child p {
    margin-bottom: 0;
}

.quoteblock .attribution {
    margin-top: 0.5em;
    margin-right: 0.5ex;
    text-align: right;
}

.quoteblock .quoteblock {
    margin-left: 0;
    margin-right: 0;
    padding: 0.5em 0;
    border-left: 3px solid #333333;
}

.quoteblock .quoteblock blockquote {
    padding: 0 0 0 0.75em;
}

.quoteblock .quoteblock blockquote:before {
    display: none;
}

.verseblock {
    margin: 0 1em 1.375em 1em;
}

.verseblock pre {
    font-family: "Open Sans", "DejaVu Sans", sans;
    font-size: 1.15rem;
    color: #404040;
    font-weight: 300;
    text-rendering: optimizeLegibility;
}

.verseblock pre strong {
    font-weight: 400;
}

.verseblock .attribution {
    margin-top: 1.25rem;
    margin-left: 0.5ex;
}

.quoteblock .attribution,
.verseblock .attribution {
    font-size: 0.8125em;
    line-height: 1.45;
    font-style: italic;
}

.quoteblock .attribution br,
.verseblock .attribution br {
    display: none;
}

.quoteblock .attribution cite,
.verseblock .attribution cite {
    display: block;
    letter-spacing: -0.025em;
    color: #333333;
}

.quoteblock.abstract {
    margin: 0 0 1.375em 0;
    display: block;
}

.quoteblock.abstract blockquote,
.quoteblock.abstract blockquote p {
    text-align: left;
    word-spacing: 0;
}

.quoteblock.abstract blockquote:before,
.quoteblock.abstract blockquote p:first-of-type:before {
    display: none;
}

table.tableblock {
    max-width: 100%;
    border-collapse: separate;
}

table.tableblock td > .paragraph:last-child p > p:last-child,
table.tableblock th > p:last-child,
table.tableblock td > p:last-child {
    margin-bottom: 0;
}

table.tableblock,
th.tableblock,
td.tableblock {
    border: 0 solid #dddddd;
}

table.grid-all th.tableblock,
table.grid-all td.tableblock {
    border-width: 0 1px 1px 0;
}

table.grid-all tfoot > tr > th.tableblock,
table.grid-all tfoot > tr > td.tableblock {
    border-width: 1px 1px 0 0;
}

table.grid-cols th.tableblock,
table.grid-cols td.tableblock {
    border-width: 0 1px 0 0;
}

table.grid-all * > tr > .tableblock:last-child,
table.grid-cols * > tr > .tableblock:last-child {
    border-right-width: 0;
}

table.grid-rows th.tableblock,
table.grid-rows td.tableblock {
    border-width: 0 0 1px 0;
}

table.grid-all tbody > tr:last-child > th.tableblock,
table.grid-all tbody > tr:last-child > td.tableblock,
table.grid-all thead:last-child > tr > th.tableblock,
table.grid-rows tbody > tr:last-child > th.tableblock,
table.grid-rows tbody > tr:last-child > td.tableblock,
table.grid-rows thead:last-child > tr > th.tableblock {
    border-bottom-width: 0;
}

table.grid-rows tfoot > tr > th.tableblock,
table.grid-rows tfoot > tr > td.tableblock {
    border-width: 1px 0 0 0;
}

table.frame-all {
    border-width: 1px;
}

table.frame-sides {
    border-width: 0 1px;
}

table.frame-topbot {
    border-width: 1px 0;
}

th.halign-left,
td.halign-left {
    text-align: left;
}

th.halign-right,
td.halign-right {
    text-align: right;
}

th.halign-center,
td.halign-center {
    text-align: center;
}

th.valign-top,
td.valign-top {
    vertical-align: top;
}

th.valign-bottom,
td.valign-bottom {
    vertical-align: bottom;
}

th.valign-middle,
td.valign-middle {
    vertical-align: middle;
}

table thead th,
table tfoot th {
    font-weight: bold;
}

tbody tr th {
    display: table-cell;
    line-height: 1.4;
    background: whitesmoke;
}

tbody tr th,
tbody tr th p,
tfoot tr th,
tfoot tr th p {
    color: #222222;
    font-weight: bold;
}

p.tableblock > code:only-child {
    background: none;
    padding: 0;
}

p.tableblock {
    font-size: 1em;
}

td > div.verse {
    white-space: pre;
}

ol {
    margin-left: 1.875em;
}

ul li ol {
    margin-left: 1.625em;
}

dl dd {
    margin-left: 2em;
}

dl dd:last-child,
dl dd:last-child >:last-child {
    margin-bottom: 0;
}

ol > li p,
ul > li p,
ul dd,
ol dd,
.olist .olist,
.ulist .ulist,
.ulist .olist,
.olist .ulist {
    margin-bottom: 0.6875em;
}

ul.unstyled,
ol.unnumbered,
ul.checklist,
ul.none {
    list-style-type: none;
}

ul.unstyled,
ol.unnumbered,
ul.checklist {
    margin-left: 0.625em;
}

ul.checklist li > p:first-child > .fa-square-o:first-child,
ul.checklist li > p:first-child > .fa-check-square-o:first-child {
    width: 1em;
    font-size: 0.85em;
}

ul.checklist li > p:first-child > input[type="checkbox"]:first-child {
    width: 1em;
    position: relative;
    top: 1px;
}

ul.inline {
    margin: 0 auto 0.6875em auto;
    margin-left: -1.375em;
    margin-right: 0;
    padding: 0;
    list-style: none;
    overflow: hidden;
}

ul.inline > li {
    list-style: none;
    float: left;
    margin-left: 1.375em;
    display: block;
}

ul.inline > li > * {
    display: block;
}

.unstyled dl dt {
    font-weight: normal;
    font-style: normal;
}

ol.arabic {
    list-style-type: decimal;
}

ol.decimal {
    list-style-type: decimal-leading-zero;
}

ol.loweralpha {
    list-style-type: lower-alpha;
}

ol.upperalpha {
    list-style-type: upper-alpha;
}

ol.lowerroman {
    list-style-type: lower-roman;
}

ol.upperroman {
    list-style-type: upper-roman;
}

ol.lowergreek {
    list-style-type: lower-greek;
}

.hdlist > table,
.colist > table {
    border: 0;
    background: none;
}

.hdlist > table > tbody > tr,
.colist > table > tbody > tr {
    background: none;
}

td.hdlist1,
td.hdlist2 {
    vertical-align: top;
    padding: 0 0.625em;
}

td.hdlist1 {
    font-weight: bold;
    padding-bottom: 1.375em;
}

.literalblock + .colist,
.listingblock + .colist {
    margin-top: -0.5em;
}

.colist > table tr > td:first-of-type {
    padding: 0 0.75em;
    line-height: 1;
}

.colist > table tr > td:last-of-type {
    padding: 0.25em 0;
}

.thumb,
.th {
    line-height: 0;
    display: inline-block;
    border: solid 4px white;
    -webkit-box-shadow: 0 0 0 1px #dddddd;
    box-shadow: 0 0 0 1px #dddddd;
}

.imageblock.left,
.imageblock[style*="float: left"] {
    margin: 0.25em 0.625em 1.25em 0;
}

.imageblock.right,
.imageblock[style*="float: right"] {
    margin: 0.25em 0 1.25em 0.625em;
}

.imageblock > .title {
    margin-bottom: 0;
}

.imageblock.thumb,
.imageblock.th {
    border-width: 6px;
}

.imageblock.thumb > .title,
.imageblock.th > .title {
    padding: 0 0.125em;
}

.image.left,
.image.right {
    margin-top: 0.25em;
    margin-bottom: 0.25em;
    display: inline-block;
    line-height: 0;
}

.image.left {
    margin-right: 0.625em;
}

.image.right {
    margin-left: 0.625em;
}

a.image {
    text-decoration: none;
    display: inline-block;
}

a.image object {
    pointer-events: none;
}

sup.footnote,
sup.footnoteref {
    font-size: 0.875em;
    position: static;
    vertical-align: super;
}

sup.footnote a,
sup.footnoteref a {
    text-decoration: none;
}

sup.footnote a:active,
sup.footnoteref a:active {
    text-decoration: underline;
}

#footnotes {
    padding-top: 0.75em;
    padding-bottom: 0.75em;
    margin-bottom: 0.625em;
}

#footnotes hr {
    width: 20%;
    min-width: 6.25em;
    margin: -0.25em 0 0.75em 0;
    border-width: 1px 0 0 0;
}

#footnotes .footnote {
    padding: 0 0.375em 0 0.225em;
    line-height: 1.3334;
    font-size: 0.875em;
    margin-left: 1.2em;
    text-indent: -1.05em;
    margin-bottom: 0.2em;
}

#footnotes .footnote a:first-of-type {
    font-weight: bold;
    text-decoration: none;
}

#footnotes .footnote:last-of-type {
    margin-bottom: 0;
}

#content #footnotes {
    margin-top: -0.625em;
    margin-bottom: 0;
    padding: 0.75em 0;
}

.gist .file-data > table {
    border: 0;
    background: #fff;
    width: 100%;
    margin-bottom: 0;
}

.gist .file-data > table td.line-data {
    width: 99%;
}

div.unbreakable {
    page-break-inside: avoid;
}

.big {
    font-size: larger;
}

.small {
    font-size: smaller;
}

.underline {
    text-decoration: underline;
}

.overline {
    text-decoration: overline;
}

.line-through {
    text-decoration: line-through;
}

.aqua {
    color: #00bfbf;
}

.aqua-background {
    background-color: #00fafa;
}

.black {
    color: black;
}

.black-background {
    background-color: black;
}

.blue {
    color: #0000bf;
}

.blue-background {
    background-color: #0000fa;
}

.fuchsia {
    color: #bf00bf;
}

.fuchsia-background {
    background-color: #fa00fa;
}

.gray {
    color: #606060;
}

.gray-background {
    background-color: #7d7d7d;
}

.green {
    color: #006000;
}

.green-background {
    background-color: #007d00;
}

.lime {
    color: #00bf00;
}

.lime-background {
    background-color: #00fa00;
}

.maroon {
    color: #600000;
}

.maroon-background {
    background-color: #7d0000;
}

.navy {
    color: #000060;
}

.navy-background {
    background-color: #00007d;
}

.olive {
    color: #606000;
}

.olive-background {
    background-color: #7d7d00;
}

.purple {
    color: #600060;
}

.purple-background {
    background-color: #7d007d;
}

.red {
    color: #bf0000;
}

.red-background {
    background-color: #fa0000;
}

.silver {
    color: #909090;
}

.silver-background {
    background-color: #bcbcbc;
}

.teal {
    color: #006060;
}

.teal-background {
    background-color: #007d7d;
}

.white {
    color: #bfbfbf;
}

.white-background {
    background-color: #fafafa;
}

.yellow {
    color: #bfbf00;
}

.yellow-background {
    background-color: #fafa00;
}

span.icon > .fa {
    cursor: default;
}

.admonitionblock td.icon [class^="fa icon-"] {
    font-size: 2.5em;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
    cursor: default;
}

.admonitionblock td.icon .icon-note:before {
    content: "\f05a";
    color: #004c73;
}

.admonitionblock td.icon .icon-tip:before {
    content: "\f0eb";
    text-shadow: 1px 1px 2px rgba(155, 155, 0, 0.8);
    color: #111;
}

.admonitionblock td.icon .icon-warning:before {
    content: "\f071";
    color: #bf6900;
}

.admonitionblock td.icon .icon-caution:before {
    content: "\f06d";
    color: #bf3400;
}

.admonitionblock td.icon .icon-important:before {
    content: "\f06a";
    color: #bf0000;
}

.conum[data-value] {
    display: inline-block;
    color: #fff !important;
    background-color: #404040;
    -webkit-border-radius: 100px;
    border-radius: 100px;
    text-align: center;
    font-size: 0.75em;
    width: 1.67em;
    height: 1.67em;
    line-height: 1.67em;
    font-family: "Open Sans", "DejaVu Sans", sans-serif;
    font-style: normal;
    font-weight: bold;
}

.conum[data-value] * {
    color: #fff !important;
}

.conum[data-value] + b {
    display: none;
}

.conum[data-value]:after {
    content: attr(data-value);
}

pre .conum[data-value] {
    position: relative;
    top: -0.125em;
}

b.conum * {
    color: inherit !important;
}

.conum:not([data-value]):empty {
    display: none;
}

#header,
#content,
#footnotes {
    background: white;
    padding-left: 2.5em;
    padding-right: 2.5em;
}

#header {
    margin-bottom: 0;
}

#header > h1 {
    border-bottom: none;
}

.literalblock pre,
.listingblock pre {
    background: #eff1f1;
}

#footnotes {
    margin-bottom: 2em;
}

.sect1 {
    padding-bottom: 0;
}

ol > li p,
ul > li p,
ul dd,
ol dd,
.olist .olist,
.ulist .olist,
.olist .ulist {
    margin-bottom: 0.34375em;
}

// Custom OpenBEL theming:

@media only screen and (min-width: 768px) {
	#toc.toc2 {
		background-color: #f4f4f4;
	}
}
#content {
	margin-top: 0;
}
h1, h2, h3, h4, h5, h6 {
	color: #25777D;
}
h1, h2, h3 {
	border-bottom: 1px dotted #FDC975;
}
a {
	color: #0C91A5;
}
a:hover,
a:focus,
a:active {
	text-decoration: none;
	color: #25777D;
	border-bottom: 1px dotted #FDC975;
}
#toc a:active {
    text-decoration: none;
}
#header h1 {
	border-bottom: none;
}
body {
	background-color: #fff;
}
#content {
	border-top: 1px dashed #FDC975;
}


</style>
</head>
<body class="article toc2 toc-right">
<div id="header">
<h1>BEL 2.0 Specification</h1>
<div class="details">
<span id="author" class="author">Natalie Catlett</span><br>
<span id="email" class="email"><a href="mailto:info@openbel.org">info@openbel.org</a></span><br>
<span id="revnumber">version 2.0,</span>
<span id="revdate">2014-06</span>
</div>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#overview">Overview</a>
<ul class="sectlevel2">
<li><a href="#_bel_history">BEL History</a></li>
<li><a href="#_bel_version_history">BEL Version History</a></li>
</ul>
</li>
<li><a href="#_summary_of_changes_for_bel_v2_0">Summary of Changes for BEL v2.0</a>
<ul class="sectlevel2">
<li><a href="#_variants">Variants</a></li>
<li><a href="#_protein_cleavage_fragments">Protein Cleavage Fragments</a></li>
<li><a href="#_post_translational_protein_modifications">Post-Translational Protein Modifications</a></li>
<li><a href="#_translocations_and_cellular_location">Translocations and Cellular Location</a></li>
<li><a href="#_activity_functions">Activity Functions</a></li>
<li><a href="#_regulates_relationship">Regulates Relationship</a></li>
<li><a href="#_bel_script_format_changes">BEL Script Format Changes</a></li>
</ul>
</li>
<li><a href="#LanguageStructure">1. Language Structure</a>
<ul class="sectlevel2">
<li><a href="#_namespaces">1.1. Namespaces</a>
<ul class="sectlevel3">
<li><a href="#_equivalencing_between_namespaces">1.1.1. Equivalencing between Namespaces</a></li>
</ul>
</li>
<li><a href="#_terms">1.2. Terms</a>
<ul class="sectlevel3">
<li><a href="#_abundances">1.2.1. Abundances</a></li>
<li><a href="#_processes">1.2.2. Processes</a></li>
<li><a href="#_bel_terms_as_functional_expressions">1.2.3. BEL Terms as Functional Expressions</a></li>
</ul>
</li>
<li><a href="#_statements">1.3. Statements</a>
<ul class="sectlevel3">
<li><a href="#_example_bel_statements">1.3.1. Example BEL Statements</a>
<ul class="sectlevel4">
<li><a href="#_subject_term_only">Subject Term Only</a></li>
<li><a href="#_causal">Causal</a></li>
<li><a href="#_causal_2">Causal</a></li>
<li><a href="#_nested_statement_object_term_is_statement">Nested Statement - Object Term is Statement</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_annotations">1.4. Annotations</a></li>
</ul>
</li>
<li><a href="#_bel_functions">2. BEL Functions</a>
<ul class="sectlevel2">
<li><a href="#_abundance_functions">2.1. Abundance Functions</a>
<ul class="sectlevel3">
<li><a href="#Xabundancea">2.1.1. abundance(), a()</a>
<ul class="sectlevel4">
<li><a href="#_examples_small_molecule_and_chemical">Examples - small molecule and chemical</a></li>
</ul>
</li>
<li><a href="#XcomplexA">2.1.2. complexAbundance(), complex()</a>
<ul class="sectlevel4">
<li><a href="#_example_named_complex">Example - named complex</a></li>
<li><a href="#_example_composed_complex">Example - composed complex</a></li>
</ul>
</li>
<li><a href="#XcompositeA">2.1.3. compositeAbundance(), composite()</a>
<ul class="sectlevel4">
<li><a href="#_example_bel_statement_with_compositeabundance_term">Example - BEL Statement with compositeAbundance term</a></li>
</ul>
</li>
<li><a href="#XgeneA">2.1.4. geneAbundance(), g()</a>
<ul class="sectlevel4">
<li><a href="#_example_promoter_binding_event_represented_using_geneabundance">Example - promoter binding event represented using geneAbundance</a></li>
</ul>
</li>
<li><a href="#XmicroRNAA">2.1.5. microRNAAbundance(), m()</a>
<ul class="sectlevel4">
<li><a href="#_example_microrna_abundance">Example - microRNA abundance</a></li>
</ul>
</li>
<li><a href="#XproteinA">2.1.6. proteinAbundance(), p()</a>
<ul class="sectlevel4">
<li><a href="#_examples_protein_abundances">Examples - protein abundances</a></li>
</ul>
</li>
<li><a href="#XrnaA">2.1.7. rnaAbundance(), r()</a>
<ul class="sectlevel4">
<li><a href="#_example_rna_abundance">Example - RNA abundance</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_abundance_modifier_functions">2.2. Abundance Modifier Functions</a>
<ul class="sectlevel3">
<li><a href="#_protein_modifications">2.2.1. Protein Modifications</a>
<ul class="sectlevel4">
<li><a href="#_proteinmodification_pmod">proteinModification(),  pmod()</a></li>
<li><a href="#_examples">Examples</a></li>
<li><a href="#_modification_types_provided_in_default_bel_namespace">Modification Types Provided in Default BEL Namespace</a></li>
<li><a href="#_supported_one_and_three_letter_amino_acid_codes">Supported One- and Three-letter Amino Acid Codes</a></li>
</ul>
</li>
<li><a href="#_variants_2">2.2.2. Variants</a>
<ul class="sectlevel4">
<li><a href="#_variant_var">variant(), var()</a></li>
<li><a href="#_protein_examples">Protein examples</a></li>
<li><a href="#_dna_gene_examples">DNA (gene) examples</a></li>
<li><a href="#_rna_examples">RNA examples</a></li>
<li><a href="#_rna_reference_sequence">RNA reference sequence</a></li>
</ul>
</li>
<li><a href="#_proteolytic_fragments">2.2.3. Proteolytic fragments</a>
<ul class="sectlevel4">
<li><a href="#_fragment_frag">fragment(), frag()</a></li>
<li><a href="#_examples_2">Examples</a></li>
</ul>
</li>
<li><a href="#_cellular_location">2.2.4. Cellular location</a>
<ul class="sectlevel4">
<li><a href="#_location_loc">location(), loc()</a></li>
<li><a href="#_examples_3">Examples</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_process_functions">2.3. Process Functions</a>
<ul class="sectlevel3">
<li><a href="#_biologicalprocess_bp">2.3.1. biologicalProcess(), bp()</a>
<ul class="sectlevel4">
<li><a href="#_examples_4">Examples</a></li>
</ul>
</li>
<li><a href="#_pathology_path">2.3.2. pathology(), path()</a>
<ul class="sectlevel4">
<li><a href="#_examples_5">Examples</a></li>
</ul>
</li>
<li><a href="#Xactivity">2.3.3. activity(), act()</a>
<ul class="sectlevel4">
<li><a href="#_example">Example</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_process_modifier_function">2.4. Process Modifier Function</a>
<ul class="sectlevel3">
<li><a href="#XmolecularA">2.4.1. molecularActivity(), ma()</a>
<ul class="sectlevel4">
<li><a href="#_examples_6">Examples</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_transformation_functions">2.5. Transformation Functions</a>
<ul class="sectlevel3">
<li><a href="#_translocations">2.5.1. Translocations</a>
<ul class="sectlevel4">
<li><a href="#_translocation_tloc">translocation(), tloc()</a></li>
<li><a href="#_example_2">Example</a></li>
<li><a href="#_cellsecretion_sec">cellSecretion(), sec()</a></li>
<li><a href="#_cellsurfaceexpression_surf">cellSurfaceExpression(), surf()</a></li>
</ul>
</li>
<li><a href="#_degradation_deg">2.5.2. degradation(), deg()</a></li>
<li><a href="#_reaction_rxn">2.5.3. reaction(), rxn()</a>
<ul class="sectlevel4">
<li><a href="#_example_3">Example</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_other_functions">2.6. Other Functions</a>
<ul class="sectlevel3">
<li><a href="#_fusion_fus">2.6.1. fusion(), fus()</a>
<ul class="sectlevel4">
<li><a href="#_example_4">Example</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li><a href="#_bel_relationships">3. BEL Relationships</a>
<ul class="sectlevel2">
<li><a href="#_causal_relationships">3.1. Causal Relationships</a>
<ul class="sectlevel3">
<li><a href="#Xincreases">3.1.1. increases, &#8594;</a></li>
<li><a href="#XdIncreases">3.1.2. directlyIncreases, &#8658;</a></li>
<li><a href="#Xdecreases">3.1.3. decreases, -|</a></li>
<li><a href="#XdDecreases">3.1.4. directlyDecreases, =|</a></li>
<li><a href="#_ratelimitingstepof">3.1.5. rateLimitingStepOf</a>
<ul class="sectlevel4">
<li><a href="#_example_5">Example</a></li>
</ul>
</li>
<li><a href="#Xcnc">3.1.6. causesNoChange, cnc</a></li>
<li><a href="#_regulates_reg">3.1.7. regulates, reg</a>
<ul class="sectlevel4">
<li><a href="#__strong_direct_relationships_strong"><strong>Direct Relationships</strong></a></li>
<li><a href="#_direct_causal_relationships">Direct causal relationships</a></li>
<li><a href="#_examples_7">Examples</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#XcorRels">3.2. Correlative Relationships</a>
<ul class="sectlevel3">
<li><a href="#XnegCor">3.2.1. negativeCorrelation, neg</a></li>
<li><a href="#XposCor">3.2.2. positiveCorrelation, pos</a></li>
<li><a href="#Xassociation">3.2.3. association,&#8201;&#8212;&#8201;</a></li>
</ul>
</li>
<li><a href="#_genomic_relationships">3.3. Genomic Relationships</a>
<ul class="sectlevel3">
<li><a href="#_orthologous">3.3.1. orthologous</a></li>
<li><a href="#_transcribedto">3.3.2. transcribedTo, :&gt;</a></li>
<li><a href="#_translatedto">3.3.3. translatedTo, &gt;&gt;</a></li>
</ul>
</li>
<li><a href="#_other_relationships">3.4. Other Relationships</a>
<ul class="sectlevel3">
<li><a href="#_hasmember">3.4.1. hasMember</a></li>
<li><a href="#_hasmembers">3.4.2. hasMembers</a></li>
<li><a href="#_hascomponent">3.4.3. hasComponent</a></li>
<li><a href="#_hascomponents">3.4.4. hasComponents</a></li>
<li><a href="#_isa">3.4.5. isA</a></li>
<li><a href="#_subprocessof">3.4.6. subProcessOf</a></li>
</ul>
</li>
<li><a href="#_deprecated_relationships">3.5. Deprecated Relationships</a>
<ul class="sectlevel3">
<li><a href="#_analogous">3.5.1. analogous</a></li>
<li><a href="#_biomarkerfor">3.5.2. biomarkerFor</a></li>
<li><a href="#_prognosticbiomarkerfor">3.5.3. prognosticBiomarkerFor</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_appendices">4. Appendices</a>
<ul class="sectlevel2">
<li><a href="#_namespaces_used_in_examples">4.1. Namespaces Used in Examples</a></li>
<li><a href="#_bel_examples">4.2. BEL Examples</a>
<ul class="sectlevel3">
<li><a href="#_bel_term_examples">4.2.1. BEL Term Examples</a>
<ul class="sectlevel4">
<li><a href="#_abundance_term_examples">Abundance Term Examples</a></li>
<li><a href="#_activity_term_examples">Activity Term Examples</a></li>
<li><a href="#_binding_interaction_term_examples">Binding Interaction Term Examples</a></li>
<li><a href="#_biological_processes_and_pathologies_term_examples">Biological Processes and Pathologies Term Examples</a></li>
<li><a href="#_post_translationally_modified_protein_term_examples">Post-Translationally Modified Protein Term Examples</a></li>
<li><a href="#Xtte">Transformation Term Examples (Reactions, Translocations, Degradation)</a></li>
<li><a href="#_variant_mutant_protein_examples">Variant (Mutant) Protein Examples</a></li>
</ul>
</li>
<li><a href="#_bel_statement_examples">4.2.2. BEL Statement Examples</a>
<ul class="sectlevel4">
<li><a href="#_causal_statement_examples">Causal Statement Examples</a></li>
<li><a href="#_correlative_statement_examples">Correlative Statement Examples</a></li>
<li><a href="#_direct_causal_statement_examples">Direct Causal Statement Examples</a></li>
<li><a href="#_nested_statement_example">Nested Statement Example</a></li>
</ul>
</li>
<li><a href="#_other_examples">4.2.3. Other Examples</a>
<ul class="sectlevel4">
<li><a href="#_bel_statement_annotation_examples">BEL Statement Annotation Examples</a></li>
<li><a href="#_membership_assignment_examples">Membership Assignment Examples</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_bel_best_practices_updated_for_bel_v2">4.3. BEL Best Practices - Updated for BEL v2</a>
<ul class="sectlevel3">
<li><a href="#_representation_of_experimental_data">4.3.1. Representation of Experimental Data</a>
<ul class="sectlevel4">
<li><a href="#_subject_terms_perturbations">Subject Terms (Perturbations)</a></li>
<li><a href="#_relationships">Relationships</a></li>
<li><a href="#_object_terms_measurements">Object Terms (Measurements)</a></li>
</ul>
</li>
<li><a href="#_statement_annotations">4.3.2. Statement Annotations</a>
<ul class="sectlevel4">
<li><a href="#_how_do_i_annotate_a_relationship_observed_in_multiple_biological_contexts">How do I annotate a relationship observed in multiple biological contexts?</a></li>
</ul>
</li>
<li><a href="#_modified_proteins">4.3.3. Modified Proteins</a>
<ul class="sectlevel4">
<li><a href="#_how_do_i_represent_a_protein_modification_when_specific_information_is_not_available">How do I represent a protein modification when specific information is not available?</a></li>
<li><a href="#_how_do_i_represent_a_protein_modification_within_a_complex">How do I represent a protein modification within a complex?</a></li>
<li><a href="#Xmultphosphforprotein">How do I represent a situation where multiple phosphorylations are required for a protein&#8217;s activity?</a></li>
<li><a href="#_how_do_i_represent_a_situation_where_one_protein_modification_initiates_additional_modifications">How do I represent a situation where one protein modification initiates additional modifications?</a></li>
<li><a href="#Xremovalofproteinmod">How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?</a></li>
</ul>
</li>
<li><a href="#_reactions_2">4.3.4. Reactions</a>
<ul class="sectlevel4">
<li><a href="#_how_can_i_represent_a_reversible_metabolic_reaction">How can I represent a reversible metabolic reaction?</a></li>
<li><a href="#_when_and_why_should_i_use_a_reaction_term">When and why should I use a reaction term?</a></li>
</ul>
</li>
<li><a href="#_protein_protein_interactions_2">4.3.5. Protein-Protein Interactions</a>
<ul class="sectlevel4">
<li><a href="#_how_do_i_represent_a_physical_interaction_between_two_entities">How do I represent a physical interaction between two entities?</a></li>
</ul>
</li>
<li><a href="#_protein_families_2">4.3.6. Protein Families</a>
<ul class="sectlevel4">
<li><a href="#_when_should_i_use_a_protein_family_instead_of_a_specific_protein">When should I use a protein family instead of a specific protein?</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li><a href="#_implementation">5. Implementation</a>
<ul class="sectlevel2">
<li><a href="#_formats">5.1. Formats</a>
<ul class="sectlevel3">
<li><a href="#_bel_script">5.1.1. BEL Script</a></li>
<li><a href="#_xbel">5.1.2. XBEL</a></li>
<li><a href="#_evidence_json">5.1.3. Evidence JSON</a></li>
<li><a href="#_json_graph_format_jgf">5.1.4. JSON Graph Format (JGF)</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#_tools">6. Tools</a>
<ul class="sectlevel2">
<li><a href="#_java">6.1. Java</a></li>
<li><a href="#_ruby">6.2. Ruby</a></li>
</ul>
</li>
</ul>
</div>
</div>
<div id="content">
<div id="preamble">
<div class="sectionbody">
<div class="paragraph">
<p>This is the home of the BEL Language Documentation v2.0.</p>
</div>
<div class="paragraph">
<p><em>Please cite the BEL v2.0 language documentation as "BEL v2.0 Language Documentation, <a href="https://github.com/OpenBEL/language" class="bare">https://github.com/OpenBEL/language</a> ", along with the date accessed.</em></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="overview">Overview</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Biological Expression Language (BEL) is a language for representing biological observations in a computable form, along with contextual information. BEL is intended as a knowledge capture and interchange medium. BEL is used to qualitatively represent causal and correlative relationships involving biological measurements (e.g., RNA, protein, phosphorylated proteins). Each BEL statement stands alone as an individual observation or fact, and can be integrated with related observations into a cohesive network.</p>
</div>
<div class="paragraph">
<p>BEL is a human-readable and -writable language designed to be easy for life scientists to learn and use.  BEL is comprised of a relatively small set of function and relationships types that can be used in conjunction with widely used vocabularies like HGNC human gene symbols, Gene Ontology, ChEBI, and MeSH. As a language of discourse for biological findings, BEL is designed to be "white-boardable" as well as written.</p>
</div>
<div class="sect2">
<h3 id="_bel_history">BEL History</h3>
<div class="paragraph">
<p>BEL was initially designed in 2003 at Selventa (operating as Genstruct) by Dexter Pratt. BEL was designed with a focus on capturing qualitative causal relationships that could be used for inference.  From 2003 to 2010, BEL evolved in response to daily use by scientists representing findings derived from tens of thousands of abstracts and full-text articles.</p>
</div>
<div class="paragraph">
<p>In 2011, it was proposed to make BEL an open standard.  BEL has been refined, formalized, and extended to meet the needs of a broader community to represent, manage, and share scientific findings in the life sciences. BEL and associated software was released as open-source technology in 2012. OpenBEL became a Linux Foundation Collaborative project in 2013.</p>
</div>
</div>
<div class="sect2">
<h3 id="_bel_version_history">BEL Version History</h3>
<div class="ulist">
<ul>
<li>
<p>BEL v1.0  initial open source release, 2012</p>
</li>
<li>
<p>BEL v2.0  major revisions and refinements, 2014</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_summary_of_changes_for_bel_v2_0">Summary of Changes for BEL v2.0</h2>
<div class="sectionbody">
<div class="paragraph">
<p>These additions and modifications enhance the BEL language by providing new representation capability (e.g., DNA and RNA variants, protein cleavage fragments, cellular location of abundances) and enabling the use of external vocabularies (post-translational modifications, activities).</p>
</div>
<div class="sect2">
<h3 id="_variants">Variants</h3>
<div class="ulist">
<ul>
<li>
<p>Now represent sequence variants at DNA, RNA, and protein levels.</p>
</li>
<li>
<p>Now represent multiple substitutions within the same gene/RNA/protein</p>
</li>
<li>
<p>New BEL abundance modifier function <a href="http://wiki.openbel.org/display/BLVD/Variants"><code>variant()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Variants"><code>var()</code></a> is used for most variant types, replacing <code>substitution()</code> / <code>sub()</code> and <code>truncation()</code> / <code>trunc()</code>. <a href="http://www.hgvs.org/rec.html">Human Genome Variation Society</a> (HGVS) nomenclature adopted to describe variants (<a href="http://onlinelibrary.wiley.com/doi/10.1002/%28SICI%291098-1004%28200001%2915:1%3C7::AID-HUMU4%3E3.0.CO;2-N/pdf">Dunnen and Antonarakis, 2000</a>) within the <code>var()</code> modifier function, expanding supported types of variation to include insertions, deletions, duplications as well as non-specific variants.</p>
</li>
<li>
<p>Usage of <a href="http://wiki.openbel.org/display/BLVD/Other+Functions"><code>fus()</code></a> changed. Instead of a modifier function for a gene/RNA/protein abundance, <code>fus()</code> is used to compose new entities that can be used in place of a namespace value for abundance functions.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_protein_cleavage_fragments">Protein Cleavage Fragments</h3>
<div class="ulist">
<ul>
<li>
<p>New abundance modifier function <a href="http://wiki.openbel.org/display/BLVD/Proteolytic+fragments"><code>fragment()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Proteolytic+fragments"><code>frag()</code></a> to be used within protein abundances to specify protein fragments based on amino acid sequence range.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_post_translational_protein_modifications">Post-Translational Protein Modifications</h3>
<div class="ulist">
<ul>
<li>
<p>The <a href="http://wiki.openbel.org/display/BLVD/Protein+Modifications"><code>proteinModification()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Protein+Modifications"><code>pmod()</code></a> abundance modifier function can now use external vocabularies (e.g., <a href="http://psidev.cvs.sourceforge.net/viewvc/psidev/psi/mod/data/PSI-MOD.obo">PSI-MOD</a>) for modification types, enabling users to add types without requiring a language change.</p>
</li>
<li>
<p>Now multiple <code>pmod()</code> expressions can be used within a protein abundance.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_translocations_and_cellular_location">Translocations and Cellular Location</h3>
<div class="ulist">
<ul>
<li>
<p>New abundance modifier function to specify location - <a href="http://wiki.openbel.org/display/BLVD/Cellular+location"><code>location()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Cellular+location"><code>loc()</code></a>.</p>
</li>
<li>
<p>Change in <a href="http://wiki.openbel.org/display/BLVD/Transformation+Functions"><code>translocation()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Transformation+Functions"><code>tloc()</code></a> function format, to explicitly add BEL location functions to location arguments.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_activity_functions">Activity Functions</h3>
<div class="ulist">
<ul>
<li>
<p>The ten distinct BEL activity functions, e.g., <code>kinaseActivity()</code> / <code>kin()</code>, <code>catalyticActivity()</code> / <code>cat()</code>, <code>transcriptionalActivity()</code> / <code>tscript()</code>, are consolidated to a single activity function <a href="http://wiki.openbel.org/display/BLVD/Process+Functions#ProcessFunctions-act()"><code>activity()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Process+Functions#ProcessFunctions-act()"><code>act()</code></a>.</p>
</li>
<li>
<p>New modifier function <a href="http://wiki.openbel.org/display/BLVD/Process+Modifier+Function"><code>molecularActivity()</code></a> / <a href="http://wiki.openbel.org/display/BLVD/Process+Modifier+Function"><code>ma()</code></a> can be used to specify specific activity types, using external vocabularies, e.g., <a href="http://www.geneontology.org/GO.function.guidelines.shtml">GO Molecular Function</a>, or a default BEL vocabulary.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_regulates_relationship">Regulates Relationship</h3>
<div class="ulist">
<ul>
<li>
<p>New causal relationship <code>regulates</code> to represent cases where A is reported to affect B, but it cannot be determined if A increases or decreases B.</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_bel_script_format_changes">BEL Script Format Changes</h3>
<div class="ulist">
<ul>
<li>
<p>Citation annotation requirement removed for <em>Name</em> field</p>
</li>
<li>
<p>Citation annotation <em>DOI</em> and <em>URL</em> added as accepted types</p>
</li>
<li>
<p><em>BEL Script Evidence</em> Annotation renamed to <em>Support</em></p>
</li>
<li>
<p>BEL version set in document header</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="LanguageStructure">1. Language Structure</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Knowledge in BEL is expressed as <a href="http://wiki.openbel.org/display/BLVD/Statements">BEL Statements</a>.  Generally, BEL Statements have the form of a <em>subject</em> - <em>predicate</em> - <em>object</em> triple, where the subject is a BEL Term, the predicate is one of the BEL relationship types (e.g., <code><em>increases</em></code>),  and the object can be either a BEL Term or a BEL Statement. A BEL Statement may also be comprised of a subject term only.</p>
</div>
<div class="paragraph">
<p><a href="http://wiki.openbel.org/display/BLVD/Terms">BEL Terms</a> are composed of BEL Functions applied to concepts referenced using <a href="http://wiki.openbel.org/display/BLVD/Namespaces">Namespace</a> identifiers. Each BEL Term represents either an abundance of a biological entity, e.g., human AKT1 protein, or a process such as apoptosis.</p>
</div>
<div class="paragraph">
<p><a href="http://wiki.openbel.org/display/BLVD/Annotations">BEL Annotations</a> are applied to BEL Statements to optionally express additional information about the statement itself such as the citation for the publication reporting the observation, or the context in which the observation was made (e.g., species, tissue, cell line).</p>
</div>
<div class="sect2">
<h3 id="_namespaces">1.1. Namespaces</h3>
<div class="paragraph">
<p>BEL is specifically designed to adopt external vocabularies and ontologies, and represent life-science knowledge in the language and schema of the organization collecting or using the knowledge. Thus, BEL Terms are defined by reference to concepts in external vocabularies, which provide a set of well-known domain values, such as the official human gene symbols provided by HGNC (<a href="http://www.genenames.org/">http://www.genenames.org/</a>) . While we consider it good practice to define biological entities with respect to well-defined domains such as public ontologies, no specific vocabulary is essential to the use of BEL, and users are free to define and reference their own vocabularies as needed.</p>
</div>
<div class="paragraph">
<p>BEL uses Namespaces to unambiguously reference concepts. The user associates a Namespace prefix with an external vocabulary and uses the prefix to refer to elements of the vocabulary. For example, if we associate the Namespace prefix HGNC with the vocabulary of symbols managed by the HGNC committee, we can then compose BEL Terms by referencing the HGNC Namespace prefix and any concept from the HGNC namespace together with a relevant BEL Function, e.g., <code>proteinAbundance(HGNC:AKT1)</code> or <code>rnaAbundance(HGNC:TNF)</code>.</p>
</div>
<div class="sect3">
<h4 id="_equivalencing_between_namespaces">1.1.1. Equivalencing between Namespaces</h4>
<div class="paragraph">
<p>Values from different Namespaces may correspond to the same biological concept. For example, the name AKT1 in the HGNC Namespace refers to the same gene referenced with ID 207 in the EGID (Entrez Gene Identifier) Namespace. The BEL Framework assembles knowledge into a cohesive network, mapping equivalent BEL Terms, e.g., <code>proteinAbundance(HGNC:AKT1)</code> and <code>proteinAbundance(EGID:207)</code>, to a single node in the network. This correspondence of Namespace values is handled in the BEL Framework separately from BEL knowledge representation.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_terms">1.2. Terms</h3>
<div class="paragraph">
<p>Two general categories of biological entities are represented as BEL Terms: <strong>abundances</strong> and <strong>processes</strong>.</p>
</div>
<div class="sect3">
<h4 id="_abundances">1.2.1. Abundances</h4>
<div class="paragraph">
<p>Life science experiments often measure the abundance of a type of thing in a given sample or set of samples. BEL Abundance Terms represent classes of abundance, the abundances of specific types of things. Examples include the <em>protein abundance of TP53</em>, the <em>RNA abundance of CCND1</em>, the <em>abundance of the protein AKT1 phosphorylated at serine 21</em>, or the <em>abundance of the complex of the proteins CCND1 and CDK4</em>.</p>
</div>
</div>
<div class="sect3">
<h4 id="_processes">1.2.2. Processes</h4>
<div class="paragraph">
<p>BEL Process Terms represent classes of complex phenomena taking place at the level of the cell or the organism, such as the biological process of <em>cell cycle</em> or a disease process such as <em>Cardiomyopathy</em>. In other cases, BEL Terms may represent classes of specific molecular activities, such as the kinase activity of the AKT1 protein, or a specific chemical reaction like conversion of superoxides to hydrogen peroxide and oxygen.</p>
</div>
<div class="paragraph">
<p>Measurable biological parameters such as <em>Blood Pressure</em> or <em>Body Temperature</em> are represented as process BEL Terms. These BEL Terms denote biological activities that, when measured, are reduced to an output parameter.</p>
</div>
</div>
<div class="sect3">
<h4 id="_bel_terms_as_functional_expressions">1.2.3. BEL Terms as Functional Expressions</h4>
<div class="paragraph">
<p>BEL Terms are denoted by expressions composed of a BEL Function and a list of arguments. BEL v2.0 specifies a set of approximately 20 functions allowed in term expressions.</p>
</div>
<div class="paragraph">
<p>The combination of a BEL function and its arguments fully specifies a BEL Term. The BEL Term expression <code>f(a)</code> denotes a BEL Term defined by function <code>f()</code> applied to an argument <code>a</code>. Wherever the same function is applied to the same arguments, the resulting BEL Term references the same biological entity.</p>
</div>
<div class="paragraph">
<p>The semantics of a BEL Term are determined by the function used in the term expression. For example, the function <code>proteinAbundance()</code> is defined such that any term expression using <code>proteinAbundance()</code> represents a class of abundance of protein. Many BEL functions take only single values as arguments, providing a structured method of using ontologies and vocabularies in BEL. For example, values in the HUGO Gene Nomenclature Committee (HGNC) vocabulary of official human gene symbols can be used to designate gene, RNA, and protein abundances. The function <code>proteinAbundance()</code> could then be applied to an HGNC gene symbol, <em>AKT1</em> for example, to indicate the class of protein abundances produced by the AKT1 gene, producing the BEL Term <code>proteinAbundance(HGNC:AKT1)</code>.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_statements">1.3. Statements</h3>
<div class="paragraph">
<p>A BEL Statement represents an experimental observation, generally reported in a scientific publication or unpublished experimental data. Generally, BEL Statements express a causal or correlative relationship between two biological entities. Because BEL Terms are functionally composed, a BEL Statement can consist of a single BEL Term; this simple statement indicates that the biological entity represented by the term has been observed.</p>
</div>
<div class="sect3">
<h4 id="_example_bel_statements">1.3.1. Example BEL Statements</h4>
<div class="sect4">
<h5 id="_subject_term_only">Subject Term Only</h5>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:CCND1), p(HGNC:CDK4))</pre>
</div>
</div>
<div class="paragraph">
<p>The abundance of a complex formed from protein abundances designated by <em>CCND1</em> and <em>CDK4</em> in the HGNC namespace. This is a subject term only statement, and indicates that the entity specified by the term has been observed.</p>
</div>
</div>
<div class="sect4">
<h5 id="_causal">Causal</h5>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CCND1) =&gt; act(p(HGNC:CDK4))</pre>
</div>
</div>
<div class="paragraph">
<p>The abundance of the protein designated by <em>CCND1</em> in the HGNC namespace directly increases the activity of the abundance of the protein designated by <em>CDK4</em> in the HGNC namespace.</p>
</div>
</div>
<div class="sect4">
<h5 id="_causal_2">Causal</h5>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:BCL2)-| bp(MESHPP:Apoptosis)</pre>
</div>
</div>
<div class="paragraph">
<p>The abundance of the protein designated by <em>BCL2</em> in the HGNC namespace decreases the biological process designated by <em>apoptosis</em> in the MESHPP (phenomena and processes) namespace.</p>
</div>
</div>
<div class="sect4">
<h5 id="_nested_statement_object_term_is_statement">Nested Statement - Object Term is Statement</h5>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:GATA1) =&gt; ( act(p(HGNC:ZBTB16)) =&gt; r(HGNC:MPL) )</pre>
</div>
</div>
<div class="paragraph">
<p>The abundance of the protein designated by <em>GATA1</em> in the HGNC namespace directly increases the process in which the activity of the protein abundance designated by <em>ZBTB16</em> in the HGNC namespace directly increases the abundance of RNA designated by <em>MPL</em> in the HGNC namespace.</p>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_annotations">1.4. Annotations</h3>
<div class="paragraph">
<p>Each BEL Statement can optionally be annotated to express knowledge about the statement itself. Some important uses of annotations are to specify information about the:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>biological system in which the observation represented by the statement was made</p>
</li>
<li>
<p>experimental methods used to demonstrate the observation</p>
</li>
<li>
<p>knowledge source on which the statement is based, such as the citation and specific text supporting the statement</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Examples of annotations that could be associated with a BEL Statement are the:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>PubMed ID specifying the publication in which the observation was reported,</p>
</li>
<li>
<p>Species, tissue, and cellular location in which the observations were made, and</p>
</li>
<li>
<p>Dosage, exposure and recovery time associated with the observation.</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_bel_functions">2. BEL Functions</h2>
<div class="sectionbody">
<div class="paragraph">
<p>This section provides a listing and explanation of all BEL functions that are included in the BEL v2.0 Language Specification.</p>
</div>
<div class="sect2">
<h3 id="_abundance_functions">2.1. Abundance Functions</h3>
<div class="paragraph">
<p>The following BEL Functions represent classes of abundances of specific types of biological entities like RNAs, proteins, post-translationally modified proteins, and small molecules. Biological experiments frequently involve the manipulation and measurement of entities in samples. These BEL functions specify the type of entity referred to by a namespace value. For example,<code>geneAbundance(HGNC:AKT1)</code>, <code>rnaAbundance(HGNC:AKT1)</code>, and <code>proteinAbundance(HGNC:AKT1)</code>, represent the abundances of the AKT1 gene, RNA, and protein, respectively.</p>
</div>
<div class="sect3">
<h4 id="Xabundancea">2.1.1. abundance(), a()</h4>
<div class="paragraph">
<p><code>abundance(ns:v)</code> or <code>a(ns:v)</code> denotes the abundance of the entity designated by the value <code>v</code> in the namespace <code>ns</code>. abundance is a general abundance term that can be used for chemicals or other molecules not defined by a more specific abundance function. Gene, RNA, protein, and microRNA abundances should be represented using the appropriate specific abundance function.</p>
</div>
<div class="sect4">
<h5 id="_examples_small_molecule_and_chemical">Examples - small molecule and chemical</h5>
<div class="literalblock">
<div class="content">
<pre>a(CHEBI:"oxygen atom")
a(CHEBI:thapsigargin)</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XcomplexA">2.1.2. complexAbundance(), complex()</h4>
<div class="paragraph">
<p>The <code>complexAbundance()</code> or <code>complex()</code> function can be used with either a namespace value or with a list of abundance terms.</p>
</div>
<div class="paragraph">
<p><code>complexAbundance(ns:v)</code> or <code>complex(ns:v)</code> denotes the abundance of the molecular complex designated by the value <code>v</code> in the namespace <code>ns</code>. This form is generally used to identify abundances of named complexes.</p>
</div>
<div class="sect4">
<h5 id="_example_named_complex">Example - named complex</h5>
<div class="literalblock">
<div class="content">
<pre>complex(SCOMP:"AP-1 Complex")</pre>
</div>
</div>
<div class="paragraph">
<p><code>complexAbundance(&lt;abundance term list&gt;)</code> denotes the abundance of the molecular complex of members of the abundances denoted by <code>&lt;abundance term list&gt;</code>, a list of abundance terms supplied as arguments. The list is unordered, thus different orderings of the arguments should be interpreted as the same term. Members of a molecular complex retain their individual identities. The <code>complexAbundance()</code> function does not specify the duration or stability of the interaction of the members of the complex.</p>
</div>
</div>
<div class="sect4">
<h5 id="_example_composed_complex">Example - composed complex</h5>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:FOS), p(HGNC:JUN))</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XcompositeA">2.1.3. compositeAbundance(), composite()</h4>
<div class="paragraph">
<p>The <code>compositeAbundance(&lt;abundance term list&gt;)</code> function takes a list of abundance terms.
The <code>compositeAbundance()</code> or <code>composite()</code> function is used to represent cases where multiple abundances synergize to produce an effect. The list is unordered, thus different orderings of the arguments should be interpreted as the same term. This function should not be used if any of the abundances alone are reported to cause the effect. <code>compositeAbundance()</code> terms should be used only as subjects of statements, not as objects.</p>
</div>
<div class="sect4">
<h5 id="_example_bel_statement_with_compositeabundance_term">Example - BEL Statement with compositeAbundance term</h5>
<div class="literalblock">
<div class="content">
<pre>composite(p(HGNC:IL6), complex(GOCC:"interleukin-23 complex")) increases bp(GOBP:"T-helper 17 cell differentiation")</pre>
</div>
</div>
<div class="paragraph">
<p>In the above example, IL-6 and IL-23 synergistically induce Th17 differentiation.</p>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XgeneA">2.1.4. geneAbundance(), g()</h4>
<div class="paragraph">
<p><code>geneAbundance(ns:v)</code> or <code>g(ns:v)</code> denotes the abundance of the gene designated by the value v in the namespace ns. <code>geneAbundance()</code> terms are used to represent the DNA encoding the specified gene. <code>geneAbundance()</code> is considered decreased in the case of a homozygous or heterozygous gene deletion, and increased in the case of a DNA amplification mutation. Events in which a protein binds to the promoter of a gene can be represented using the <code>geneAbundance()</code> function.</p>
</div>
<div class="sect4">
<h5 id="_example_promoter_binding_event_represented_using_geneabundance">Example - promoter binding event represented using geneAbundance</h5>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:TP53), g(HGNC:CDKN1A))</pre>
</div>
</div>
<div class="paragraph">
<p>In the above example, the p53 protein binds the CDKN1A gene.</p>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XmicroRNAA">2.1.5. microRNAAbundance(), m()</h4>
<div class="paragraph">
<p><code>microRNAAbundance(ns:v)</code> or <code>m(ns:v)</code> denotes the abundance of the processed, functional microRNA designated by the value v in the namespace ns.</p>
</div>
<div class="sect4">
<h5 id="_example_microrna_abundance">Example - microRNA abundance</h5>
<div class="literalblock">
<div class="content">
<pre>m(HGNC:MIR21)</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XproteinA">2.1.6. proteinAbundance(), p()</h4>
<div class="paragraph">
<p><code>proteinAbundance(ns:v)</code> or <code>p(ns:v)</code> denotes the abundance of the protein designated by the value v in the namespace ns, where v references a gene or a named protein family.</p>
</div>
<div class="sect4">
<h5 id="_examples_protein_abundances">Examples - protein abundances</h5>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1)
p(SFAM:"AKT Family")</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="XrnaA">2.1.7. rnaAbundance(), r()</h4>
<div class="paragraph">
<p><code>rnaAbundance(ns:v)</code> or <code>r(ns:v)</code> denotes the abundance of the RNA designated by the value v in the namespace ns, where v references a gene. This function refers to all RNA designated by ns:v, regardless of splicing, editing, or polyadenylation stage.</p>
</div>
<div class="sect4">
<h5 id="_example_rna_abundance">Example - RNA abundance</h5>
<div class="literalblock">
<div class="content">
<pre>r(HGNC:AKT1)</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_abundance_modifier_functions">2.2. Abundance Modifier Functions</h3>
<div class="paragraph">
<p>The following BEL functions are special functions that can be used only as an argument within an abundance function. These functions modify the abundance to specify sequence variations (gene, RNA, microRNA, protein), post-translational modifications (protein), fragment resulting from proteolytic processing (protein), or cellular location (most abundance types).</p>
</div>
<div class="sect3">
<h4 id="_protein_modifications">2.2.1. Protein Modifications</h4>
<div class="sect4">
<h5 id="_proteinmodification_pmod">proteinModification(),  pmod()</h5>
<div class="paragraph">
<p>The <code>proteinModification()</code> or <code>pmod()</code> function can be used only as an argument within a <code>proteinAbundance()</code> function to indicate modification of the specified protein. Multiple modifications can be applied to the same protein abundance. Modified protein abundance term expressions have the general form:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(ns:protein_value, pmod(ns:type_value, &lt;code&gt;, &lt;pos&gt;))</pre>
</div>
</div>
<div class="paragraph">
<p><code>type_value</code> (required) is a namespace value for the type of modification , <strong><code>&lt;code&gt;</code></strong> (optional) is a <a href="#_supported_one_and_three_letter_amino_acid_codes">single-letter or three-letter code</a> for one of the twenty standard amino acids, and <code>&lt;pos&gt;</code> (optional) is the position at which the modification occurs based on the reference sequence for the protein. If <strong><code>&lt;pos&gt;</code></strong> is omitted, then the position of the modification is unspecified. If both <strong><code>&lt;code&gt;</code></strong> and <strong><code>&lt;pos&gt;</code></strong> are omitted, then the residue and position of the modification are unspecified. NOTE - A <a href="http://wiki.openbel.org/display/BLVD/Modification+Types+Provided+in+Default+BEL+Namespace">default BEL namespace</a> includes commonly used protein modification types.</p>
</div>
</div>
<div class="sect4">
<h5 id="_examples">Examples</h5>
<div class="sect5">
<h6 id="_akt1_phosphorylated_at_serine_473">AKT1 phosphorylated at Serine 473</h6>
<div class="paragraph">
<p>default BEL namespace and 1-letter amino acid code:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(Ph, S, 473))</pre>
</div>
</div>
<div class="paragraph">
<p>default BEL namespace and 3-letter amino acid code:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(Ph, Ser, 473))</pre>
</div>
</div>
<div class="paragraph">
<p><a href="http://psidev.cvs.sourceforge.net/viewvc/psidev/psi/mod/data/PSI-MOD.obo">PSI-MOD</a> namespace and 3-letter amino acid code:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(MOD:PhosRes, Ser, 473))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_mapk1_phosphorylated_at_both_threonine_185_and_tyrosine_187">MAPK1 phosphorylated at both Threonine 185 and Tyrosine 187</h6>
<div class="paragraph">
<p>default BEL namespace and 3-letter amino acid code:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:MAPK1, pmod(Ph, Thr, 185), pmod(Ph, Tyr, 187))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_palmitoylated_hras">Palmitoylated HRAS</h6>
<div class="paragraph">
<p>HRAS palmitoylated at an unspecified residue. Default BEL namespace:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:HRAS, pmod(Palm))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_modification_types_provided_in_default_bel_namespace">Modification Types Provided in Default BEL Namespace</h5>
<div class="paragraph">
<p>Additional modification types can be requested as needed, or an external vocabulary can be used. Like other BEL namespace values, these modification types can be equivalenced to values in other vocabularies.</p>
</div>
<table class="tableblock frame-all grid-all spread">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Label</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Synonym</strong></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ac</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">acetylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ADPRib</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ADP-ribosylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ADP-rybosylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">adenosine diphosphoribosyl</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Farn</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">farnesylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gerger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">geranylgeranylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glyco</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">glycosylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Hy</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">hydroxylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ISG</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ISGylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ISG15-protein conjugation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Me</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">methylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Me1</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">monomethylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">mono-methylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Me2</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">dimethylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">di-methylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Me3</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">trimethylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">tri-methylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Myr</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">myristoylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nedd</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">neddylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">NGlyco</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">N-linked glycosylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">NO</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Nitrosylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">OGlyco</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">O-linked glycosylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Palm</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">palmitoylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ph</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">phosphorylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Sulf</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulfation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulphation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulfur addition</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulphur addition</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulfonation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">sulphonation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Sumo</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SUMOylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ub</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ubiquitination</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ubiquitinylation</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">ubiquitylation</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">UbK48</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lysine 48-linked polyubiquitination</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">UbK63</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lysine 63-linked polyubiquitination</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">UbMono</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">monoubiquitination</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">UbPoly</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect4">
<h5 id="_supported_one_and_three_letter_amino_acid_codes">Supported One- and Three-letter Amino Acid Codes</h5>
<table class="tableblock frame-all grid-all spread">
<colgroup>
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 33.3334%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Amino Acid</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>1-Letter Code</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>3-Letter Code</strong></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Alanine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ala</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Arginine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">R</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Arg</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Asparagine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">N</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Asn</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Aspartic Acid</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">D</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Asp</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cysteine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">C</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Cys</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glutamic Acid</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">E</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glu</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glutamine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gln</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glycine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">G</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gly</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Histidine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">H</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">His</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Isoleucine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">I</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ile</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Leucine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">L</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Leu</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lysine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">K</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lys</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Methionine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">M</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Met</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phenylalanine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">F</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Phe</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Proline</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">P</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Pro</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Serine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">S</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Ser</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Threonine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">T</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Thr</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Tryptophan</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">W</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Trp</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Tyrosine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Y</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Tyr</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Valine</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">V</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Val</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect3">
<h4 id="_variants_2">2.2.2. Variants</h4>
<div class="sect4">
<h5 id="_variant_var">variant(), var()</h5>
<div class="paragraph">
<p>The <code>variant(&lt;expression&gt;)</code> or <code>var(&lt;expression&gt;)</code> function can be used as an argument within a <code>geneAbundance()</code>, <code>rnaAbundance()</code>, <code>microRNAAbundance()</code>, or <code>proteinAbundance()</code> to indicate a sequence variant of the specified abundance. The <code>var()</code> function takes <a href="http://www.hgvs.org/mutnomen/">HGVS</a> variant description expression, e.g., for a substitution, insertion, or deletion variant. Multiple <code>var()</code> arguments may be applied to an abundance term.</p>
</div>
</div>
<div class="sect4">
<h5 id="_protein_examples">Protein examples</h5>
<div class="sect5">
<h6 id="_reference_allele">reference allele</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CFTR, var(=))</pre>
</div>
</div>
<div class="paragraph">
<p>This is different than <code>p(HGNC:CFTR)</code>, the root protein abundance, which includes all variants.</p>
</div>
</div>
<div class="sect5">
<h6 id="_unspecified_variant">unspecified variant</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CFTR, var(?))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_substitution">substitution</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CFTR, var(p.Gly576Ala))
p(REF:NP_000483.3, var(p.Gly576Ala))</pre>
</div>
</div>
<div class="paragraph">
<p>CFTR substitution variant Glycine 576 Alanine (HGVS <em>NP_000483.3:p.Gly576Ala</em>). Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>p.</em> within the <code>var()</code> expression indicates that the numbering is based on a protein sequence.</p>
</div>
</div>
<div class="sect5">
<h6 id="_deletion">deletion</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CFTR, var(p.Phe508del))
p(REF:NP_000483.3, var(p.Phe508del))</pre>
</div>
</div>
<div class="paragraph">
<p>CFTR F508 variant (HGVS <em>NP_000483.3:p.Phe508del</em>). Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>p.</em> within the <code>var()</code> expression indicates that the numbering is based on a protein reference sequence.</p>
</div>
</div>
<div class="sect5">
<h6 id="_frameshift">frameshift</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CFTR, var(p.Thr1220Lysfs))
p(REF:NP_000483.3, var(p.Thr1220Lysfs))</pre>
</div>
</div>
<div class="paragraph">
<p>CFTR frameshift variant <em>(</em>HGVS<em> NP_000483.3:p.Thr1220Lysfs*7). </em>Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>p.</em> within the <code>var()</code> expression indicates that the numbering is based on a protein reference sequence.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_dna_gene_examples">DNA (gene) examples</h5>
<div class="paragraph">
<p>These are all representations of CFTR <em>F508</em>.</p>
</div>
<div class="sect5">
<h6 id="_snp">SNP</h6>
<div class="literalblock">
<div class="content">
<pre>g(SNP:rs113993960, var(delCTT))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_chromosome">chromosome</h6>
<div class="literalblock">
<div class="content">
<pre>g(REF:NC_000007.13, var(g.117199646_117199648delCTT))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_gene_coding_dna_reference_sequence">gene - coding DNA reference sequence</h6>
<div class="literalblock">
<div class="content">
<pre>g(HGNC:CFTR, var(c.1521_1523delCTT))
g(REF:NM_000492.3, var(c.1521_1523delCTT))</pre>
</div>
</div>
<div class="paragraph">
<p>Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>c.</em> within the <code>var()</code> expression indicates that the numbering is based on a coding DNA reference sequence.The coding DNA reference sequence covers the part of the transcript that is translated into protein; numbering starts at the A of the initiating ATG codon, and ends at the last nucleotide of the translation stop codon.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_rna_examples">RNA examples</h5>
<div class="paragraph">
<p>These are all representations of CFTR <em>F508</em>.</p>
</div>
<div class="sect5">
<h6 id="_coding_reference_sequence">coding reference sequence</h6>
<div class="literalblock">
<div class="content">
<pre>r(HGNC:CFTR, var(c.1521_1523delCTT))
r(REF:NM_000492.3, var(c.1521_1523delCTT))</pre>
</div>
</div>
<div class="paragraph">
<p>Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>c.</em> within the <code>var()</code> expression indicates that the numbering is based on a coding DNA reference sequence. The coding DNA reference sequence covers the part of the transcript that is translated into protein; numbering starts at the A of the initiating ATG codon, and ends at the last nucleotide of the translation stop codon.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_rna_reference_sequence">RNA reference sequence</h5>
<div class="literalblock">
<div class="content">
<pre>r(HGNC:CFTR, var(r.1653_1655delcuu))
r(REF:NM_000492.3, var(r.1653_1655delcuu))</pre>
</div>
</div>
<div class="paragraph">
<p>Because a specific position is referenced, a namespace value for a non-ambiguous sequence like the <a href="http://www.ncbi.nlm.nih.gov/refseq/about/">RefSeq</a> ID in the lower example is preferred over the HGNC gene symbol. The <em>r.</em> within the <code>var()</code> expression indicates that the numbering is based on an RNA reference sequence. The RNA reference sequence covers the entire transcript except for the poly A-tail; numbering starts at the trascrption initiation site and ends at the transcription termination site.</p>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_proteolytic_fragments">2.2.3. Proteolytic fragments</h4>
<div class="sect4">
<h5 id="_fragment_frag">fragment(), frag()</h5>
<div class="paragraph">
<p>The <code>fragment()</code> or <code>frag()</code> function can be used within a <code>proteinAbundance()</code> term to specify a protein fragment, e.g., a product of proteolytic cleavage. Protein fragment expressions take the general form:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(ns:v, frag(&lt;range&gt;, &lt;descriptor&gt;))</pre>
</div>
</div>
<div class="paragraph">
<p>where <code>&lt;range&gt;</code> (required) is an amino acid range, and <code>&lt;descriptor&gt;</code> (optional) is any additional distinguishing information like fragment size or name.</p>
</div>
</div>
<div class="sect4">
<h5 id="_examples_2">Examples</h5>
<div class="paragraph">
<p>For these examples, <em>HGNC:YFG</em> is your favorite gene. For the first four examples, only the <code>&lt;range&gt;</code> argument is used. The last examples include use of the optional <code>&lt;descriptor&gt;</code>.</p>
</div>
<div class="sect5">
<h6 id="_fragment_with_known_start_stop">fragment with known start/stop</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:YFG, frag(5_20))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_amino_terminal_fragment_of_unknown_length">amino-terminal fragment of unknown length</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:YFG, frag(1_?))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_carboxyl_terminal_fragment_of_unknown_length">carboxyl-terminal fragment of unknown length</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:YFG, frag(?_*))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_fragment_with_unknown_start_stop">fragment with unknown start/stop</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:YFG, frag(?))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_fragment_with_unknown_start_stop_and_a_descriptor">fragment with unknown start/stop and a descriptor</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:YFG, frag(?, 55kD))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_cellular_location">2.2.4. Cellular location</h4>
<div class="sect4">
<h5 id="_location_loc">location(), loc()</h5>
<div class="paragraph">
<p><code>location()</code> or <code>loc()</code> can be used as an argument within any abundance function except <code>compositeAbundance()</code> to represent a distinct subset of the abundance at that location. Location subsets of abundances have the general form:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>f(ns:v, loc(ns:v))</pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_examples_3">Examples</h5>
<div class="sect5">
<h6 id="_cytoplasmic_pool_of_akt1_protein">Cytoplasmic pool of AKT1 protein</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, loc(MESHCS:Cytoplasm))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_endoplasmic_reticulum_pool_of_ca_sup_2_sup">Endoplasmic Reticulum pool of Ca<sup>2+</sup></h6>
<div class="literalblock">
<div class="content">
<pre>a(CHEBI:"calcium(2+)", loc(GOCC:"endoplasmic reticulum"))</pre>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_process_functions">2.3. Process Functions</h3>
<div class="paragraph">
<p>The following BEL Functions represent classes of events or phenomena taking place at the level of the cell or the organism which do not correspond to molecular abundances, but instead to a biological process like angiogenesis or a pathology like cancer.</p>
</div>
<div class="sect3">
<h4 id="_biologicalprocess_bp">2.3.1. biologicalProcess(), bp()</h4>
<div class="paragraph">
<p><code>biologicalProcess(ns:v)</code> or <code>bp(ns:v)</code> denotes the process or population of events designated by the value v in the namespace ns.</p>
</div>
<div class="sect4">
<h5 id="_examples_4">Examples</h5>
<div class="literalblock">
<div class="content">
<pre>bp(GOBP:"cell cycle arrest")
bp(GOBP:angiogenesis)</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_pathology_path">2.3.2. pathology(), path()</h4>
<div class="paragraph">
<p><code>pathology(ns:v)</code> or <code>path(ns:v)</code> denotes the disease or pathology process designated by the value v in the namespace ns. The +pathology()` function is included to facilitate the distinction of pathologies from other biological processes because of their importance in many potential applications in the life sciences.</p>
</div>
<div class="sect4">
<h5 id="_examples_5">Examples</h5>
<div class="literalblock">
<div class="content">
<pre>pathology(MESHD:"Pulmonary Disease, Chronic Obstructive")
pathology(MESHD:adenocarcinoma)</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="Xactivity">2.3.3. activity(), act()</h4>
<div class="paragraph">
<p><code>activity(&lt;abundance&gt;)</code> or <code>act(&lt;abundance)</code> is used to specify events resulting from the molecular activity of an abundance. The <code>activity()</code> function provides distinct terms that enable differentiation of the increase or decrease of the molecular activity of a protein from changes in the abundance of the protein. <code>activity()</code> can be applied to a protein, complex, or RNA abundance term, and modified with a <a href="http://wiki.openbel.org/display/BLVD/Process+Modifier+Function#ProcessModifierFunction-ma()">molecularActivity()</a> argument to indicate a specific type of molecular activity.</p>
</div>
<div class="sect4">
<h5 id="_example">Example</h5>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:AKT1))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_process_modifier_function">2.4. Process Modifier Function</h3>
<div class="sect3">
<h4 id="XmolecularA">2.4.1. molecularActivity(), ma()</h4>
<div class="paragraph">
<p><code>molecularActivity(ns:v)</code> or <code>ma(ns:v)</code> is used to denote a specific type of activity function within an <code>activity()</code> term.</p>
</div>
<div class="paragraph">
<p>NOTE - The default BEL namespace includes commonly used molecular activity types, mapping directly to the BEL v1.0 activity functions.</p>
</div>
<div class="sect4">
<h5 id="_examples_6">Examples</h5>
<div class="sect5">
<h6 id="_default_bel_namespace_transcriptional_activity">default BEL namespace, transcriptional activity</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:FOXO1), ma(tscript))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_go_molecular_function_namespace_transcriptional_activity">GO molecular function namespace, transcriptional activity</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:FOXO1), ma(GO:"nucleic acid binding transcription factor activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_default_bel_namespace_kinase_activity">default BEL namespace, kinase activity</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:AKT1), ma(kin))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_go_molecular_function_namespace_kinase_activity">GO molecular function namespace, kinase activity</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:AKT1), ma(GO:"kinase activity"))</pre>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_transformation_functions">2.5. Transformation Functions</h3>
<div class="paragraph">
<p>The following BEL functions represent transformations. Transformations are processes or events in which one class of abundance is transformed or changed into a second class of abundance by translocation, degradation, or participation in a reaction. All types of abundance terms except compositeAbundance() may be used within these transformation functions.</p>
</div>
<div class="sect3">
<h4 id="_translocations">2.5.1. Translocations</h4>
<div class="paragraph">
<p>BEL translocation functions include <code>translocation()</code> as well as <code>cellSurfaceExpression()</code> and <code>cellSecretion()</code>, two functions intended to provide a simple, standard means of expressing commonly represented translocations.</p>
</div>
<div class="sect4">
<h5 id="_translocation_tloc">translocation(), tloc()</h5>
<div class="paragraph">
<p>For the abundance term A, <code>translocation(&lt;abundance&gt;, fromLocation(ns1:v1), toLocation(ns2:v2))</code> or <code>tloc(&lt;abundance&gt;, fromLoc(ns1:v1), toLoc(ns2:v2))</code> denotes the frequency or number of events in which members of <code>&lt;abundance&gt;</code> move from the location designated by the value v1 in the namespace ns1 to the location designated by the value <code>v2</code> in the namespace <code>ns2</code>. Translocation is applied to represent events on the cellular scale, like endocytosis and movement of transcription factors from the cytoplasm to the nucleus.  Special case translocations are handled by the BEL functions: <code>cellSecretion()</code>, <code>cellSurfaceExpression()</code>.</p>
</div>
</div>
<div class="sect4">
<h5 id="_example_2">Example</h5>
<div class="paragraph">
<p>endocytosis (translocation from the cell surface to the endosome) of the epidermal growth factor receptor (EGFR) protein can be represented as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>tloc(p(HGNC:EGFR), fromLoc(GOCC:"cell surface"), toLoc(GOCC:endosome))</pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_cellsecretion_sec">cellSecretion(), sec()</h5>
<div class="paragraph">
<p>For the abundance term A, <code>cellSecretion(&lt;abundance&gt;)</code> or <code>sec(&lt;abundance&gt;)</code> denotes the frequency or number of events in which members of <code>&lt;abundance&gt;</code> move from cells to regions outside of the cells. <code>cellSecretion(&lt;abundance&gt;</code> can be equivalently expressed as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>tloc(&lt;abundance&gt;, fromLoc(GOCC:intracellular), toLoc(GOCC:"extracellular space"))</pre>
</div>
</div>
<div class="paragraph">
<p>The intent of the <code>cellSecretion()</code> function is to provide a simple, standard means of expressing a commonly represented translocation.</p>
</div>
</div>
<div class="sect4">
<h5 id="_cellsurfaceexpression_surf">cellSurfaceExpression(), surf()</h5>
<div class="paragraph">
<p><code>cellSurfaceExpression(&lt;abundance&gt;)</code> or <code>surf(&lt;abundance&gt;)</code> denotes the frequency or abundance of events in which members of <code>&lt;abundance&gt;</code> move to the surface of cells. <code>cellSurfaceExpression(&lt;abundance&gt;)</code> can be equivalently expressed as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>tloc(&lt;abundance&gt;, fromLoc(GOCC:intracellular), toLoc(GOCC:"cell surface"))</pre>
</div>
</div>
<div class="paragraph">
<p>The intent of the <code>cellSurfaceExpression()</code> function is to provide a simple, standard means of expressing a commonly represented translocation.</p>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_degradation_deg">2.5.2. degradation(), deg()</h4>
<div class="paragraph">
<p><code>degradation(&lt;abundance&gt;)</code> or <code>deg(&lt;abundance&gt;)</code> denotes the frequency or number of events in which a member of <code>&lt;abundance&gt;</code> is degraded in some way such that it is no longer a member of <code>&lt;abundance&gt;</code>. For example, <code>degradation()</code> is used to represent proteasome-mediated proteolysis. The BEL Framework automatically connects deg(&lt;abundance&gt;) to <code>&lt;abundance&gt;</code> such that:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>deg(&lt;abundance&gt;) directlyDecreases &lt;abundance&gt;</pre>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_reaction_rxn">2.5.3. reaction(), rxn()</h4>
<div class="paragraph">
<p><code>reaction(reactants(&lt;abundance term list1&gt;), products(&lt;abundance term list2&gt;))</code> denotes the frequency or abundance of events in which members of the abundances in <code>&lt;abundance term list1&gt;</code> (the reactants) are transformed into members of the abundances in <code>&lt;abundance term list2&gt;</code> (the products).</p>
</div>
<div class="sect4">
<h5 id="_example_3">Example</h5>
<div class="paragraph">
<p>The reaction in which superoxides are dismutated into oxygen and hydrogen peroxide can be represented as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>rxn(reactants(a(CHEBI:superoxide)),products(a(CHEBI:"hydrogen peroxide"), a(CHEBI: "oxygen"))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_other_functions">2.6. Other Functions</h3>
<div class="sect3">
<h4 id="_fusion_fus">2.6.1. fusion(), fus()</h4>
<div class="paragraph">
<p><code>fusion()</code> or <code>fus()</code> expressions can be used in place of a namespace value within a gene, RNA, or protein abundance function to represent a hybrid gene, or gene product formed from two previously separate genes. <code>fusion()</code> expressions take the general form:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>fus(ns5':v5', range5', ns3':v3', range3')</pre>
</div>
</div>
<div class="paragraph">
<p>where <code>ns5':v5'</code> is a namespace and value for the 5' fusion partner, <code>range5'</code> is the sequence coordinates of the 5' partner, <code>ns3':v3'</code> is a namespace and value for the 3' partner, and <code>range3'</code> is the sequence coordinates for the 3' partner.</p>
</div>
<div class="sect4">
<h5 id="_example_4">Example</h5>
<div class="sect5">
<h6 id="_rna_abundance_of_fusion_with_known_breakpoints">RNA abundance of fusion with known breakpoints</h6>
<div class="literalblock">
<div class="content">
<pre>r(fus(HGNC:TMPRSS2, r.1_79, HGNC:ERG, r.312_5034))</pre>
</div>
</div>
<div class="paragraph">
<p>The <em>r.</em> designation in the range fields indicates that the numbering uses the RNA sequence as the reference. RNA sequence numbering starts at the transcription initiation site.  You use <em>c.</em>_ for g() fusions and <em>p.</em>_ for p() fusions.  These <em>r.</em>, <em>c.</em>, and <em>p.</em> designations come from <a href="http://www.hgvs.org">HGVS variation description</a> convention.</p>
</div>
</div>
<div class="sect5">
<h6 id="_rna_abundance_of_fusion_with_unspecified_breakpoints">RNA abundance of fusion with unspecified breakpoints</h6>
<div class="literalblock">
<div class="content">
<pre>r(fus(HGNC:TMPRSS2, ?, HGNC:ERG, ?))</pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_bel_relationships">3. BEL Relationships</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The following BEL Relationship types are included in the BEL v2.0 language specification:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_causal_relationships">Causal Relationships</a></p>
</li>
<li>
<p><a href="#XcorRels">Correlative Relationships</a></p>
</li>
<li>
<p><a href="#_genomic_relationships">Genomic Relationships</a></p>
</li>
<li>
<p><a href="#_other_relationships">Other Relationships</a></p>
</li>
<li>
<p><a href="#_deprecated_relationships">Deprecated Relationships</a></p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The most used BEL relationships should be the <a href="#_causal_relationships">causal</a> and <a href="#XcorRels">correlative</a> relationship categories. Relationships not used in the written BEL language, but introduced by the BEL Framework during compilation of a BEL network are not covered in this document.</p>
</div>
<div class="sect2">
<h3 id="_causal_relationships">3.1. Causal Relationships</h3>
<div class="paragraph">
<p>These relationship types denote a causal relationship, or the absence of a causal relationship between a subject and an object term.</p>
</div>
<div class="sect3">
<h4 id="Xincreases">3.1.1. increases, &#8594;</h4>
<div class="paragraph">
<p>For terms A and B, <code>A increases B</code> or <code>A &#8594; B</code> indicate that increases in A have been observed to cause increases in B.</p>
</div>
<div class="paragraph">
<p><code>A increases B</code> also represents cases where decreases in A have been observed to cause decreases in B, for example, in recording the results of gene deletion or other inhibition experiments.</p>
</div>
<div class="paragraph">
<p>A is a BEL Term and B is either a BEL Term or a BEL Statement.</p>
</div>
<div class="paragraph">
<p>The <code>increases</code> relationship does not indicate that the changes in A are either necessary for changes in B, nor does it indicate that changes in A are sufficient to cause changes in B.</p>
</div>
</div>
<div class="sect3">
<h4 id="XdIncreases">3.1.2. directlyIncreases, &#8658;</h4>
<div class="paragraph">
<p>For terms A and B, <code>A directlyIncreases B</code> or <code>A &#8658; B</code> indicates that increases in A have been observed to cause increases in B and that the mechanism of the causal relationship is based on physical interaction of entities related to A and B. This is a <a href="#Direct Relationships">direct</a> version of the increases relationship.</p>
</div>
</div>
<div class="sect3">
<h4 id="Xdecreases">3.1.3. decreases, -|</h4>
<div class="paragraph">
<p>For terms A and B, <code>A decreases B</code> or <code>A -| B</code> indicate that increases in A have been observed to cause decreases in B.</p>
</div>
<div class="paragraph">
<p><code>A decreases B</code> also represents cases where decreases in A have been observed to cause increases in B, for example, in recording the results of gene deletion or other inhibition experiments.</p>
</div>
<div class="paragraph">
<p>A is a BEL Term and B is either a BEL Term or a BEL Statement.</p>
</div>
<div class="paragraph">
<p>The <code>decreases</code> relationship does not indicate that the changes in A are either necessary for changes in B, nor does it indicate that changes in A are sufficient to cause changes in B.</p>
</div>
</div>
<div class="sect3">
<h4 id="XdDecreases">3.1.4. directlyDecreases, =|</h4>
<div class="paragraph">
<p>For terms A and B, <code>A directlyDecreases B</code> or <code>A =| B</code> indicates that increases in A have been observed to cause decreases in B and that the mechanism of the causal relationship is based on physical interaction of entities related to A and B. This is a <a href="#Direct Relationships">direct</a> version of the decreases relationship.</p>
</div>
</div>
<div class="sect3">
<h4 id="_ratelimitingstepof">3.1.5. rateLimitingStepOf</h4>
<div class="paragraph">
<p>For process, activity, or transformation term A and process term P, <code>A rateLimitingStepOf P</code> indicates both:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>A subProcessOf B
A -&gt; B</pre>
</div>
</div>
<div class="sect4">
<h5 id="_example_5">Example</h5>
<div class="paragraph">
<p>The catalytic activity of HMG CoA reductase is a rate-limiting step for cholesterol biosynthesis:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:HMGCR), ma(cat)) rateLimitingStepOf bp(GOBP:"cholesterol biosynthetic process")</pre>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="Xcnc">3.1.6. causesNoChange, cnc</h4>
<div class="paragraph">
<p>For terms A and B, <code>A causesNoChange B</code> or <code>A cnc B</code> indicate that B was observed not to change in response to changes in A.</p>
</div>
<div class="paragraph">
<p>Statements using this relationship correspond to cases where explicit measurement of B demonstrates lack of significant change, not for cases where the state of B is unknown.</p>
</div>
</div>
<div class="sect3">
<h4 id="_regulates_reg">3.1.7. regulates, reg</h4>
<div class="paragraph">
<p>For terms A and B, <code>A regulates B</code> or <code>A reg B</code> indicate that A is reported to have an effect on B, but information is missing about whether A increases B or A decreases B. This relationship provides more information than <a href="#Xassociation">association</a>, because the upstream entity (source term) and downstream entity (target term) can be assigned.</p>
</div>
<div class="sect4">
<h5 id="__strong_direct_relationships_strong"><strong>Direct Relationships</strong></h5>
<div class="paragraph">
<p>Direct relationships include direct causal relationships and non-causal relationships that are considered direct because they are self-referential.</p>
</div>
</div>
<div class="sect4">
<h5 id="_direct_causal_relationships">Direct causal relationships</h5>
<div class="paragraph">
<p>The direct casual relationships included in BEL v2.0 are <code>directlyIncreases</code> (<code>&#8658;</code>) and <code>directlyDecreases</code> (<code>=|</code>).</p>
</div>
<div class="paragraph">
<p>The direct casual relationships are causal relationships where the mechanism of the causal relationship is based on the physical interaction of entities related to the BEL Statement subject and object terms.</p>
</div>
<div class="paragraph">
<p>If A or B is an <a href="#_abundance_functions">abundance</a>, then members of the abundance are part of the interaction. If A or B are <a href="#Xactivity">activities</a> activities, then members of the abundances performing the activities physically interact.</p>
</div>
</div>
<div class="sect4">
<h5 id="_examples_7">Examples</h5>
<div class="sect5">
<h6 id="_abundances_and_activities">Abundances and activities</h6>
<div class="paragraph">
<p>Inhibition of the Patched 1 receptor signaling activity by Hedgehog is represented as direct, because Hedgehog and Patched 1 physically interact:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(PFH:"Hedgehog Family") =| act(p(HGNC:PTCH1))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_transcription">Transcription</h6>
<div class="paragraph">
<p>In the case of transcriptional activity, if the protein performing the transcriptional activity interacts with the gene that the RNA is transcribed from, the relationship is considered direct. For example, repression of the transcription of miR-21 by FOXO3 protein transcriptional activity is represented as direct because FOXO3 binds the miR-21 promoter:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:FOXO3),ma(tscript)) =| r(HGNC:MIR21)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_target_term_is_bel_statement">Target term is BEL statement</h6>
<div class="paragraph">
<p>If B is a BEL Statement, the relationship is considered direct if the subject abundance term for B physically interacts with the abundance term for A. For example, for the BEL Statement:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CLSPN) =&gt; (act(p(HGNC:ATR), ma(kin)) =&gt; p(HGNC:CHEK1, pmod(P)))</pre>
</div>
</div>
<div class="paragraph">
<p>CLSPN protein is considered to directly activate the phosphorylation of CHEK1 protein by the kinase activity of ATR, because the CLSPN and ATR proteins physically interact.</p>
</div>
</div>
<div class="sect5">
<h6 id="_self_referential_relationships">Self-referential relationships</h6>
<div class="paragraph">
<p>Self-referential causal relationships are generally represented as direct. For example, phosphorylation of GSK3B at serine 9 inhibiting the kinase activity of GSK3B can be represented as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:GSK3B, pmod(P, S, 9)) =| act(p(HGNC:GSK3B), ma(kin))</pre>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="XcorRels">3.2. Correlative Relationships</h3>
<div class="paragraph">
<p>These relationship types link abundances and biological processes when no causal relationship is known. The order of subject and object terms does not matter in a statement with a correlative relationship, unlike a statement with a causal relationship.</p>
</div>
<div class="sect3">
<h4 id="XnegCor">3.2.1. negativeCorrelation, neg</h4>
<div class="paragraph">
<p>For terms A and B, <code>A negativeCorrelation B</code> or <code>A neg B</code> indicates that changes in A and B have been observed to be negatively correlated. The order of the subject and object does not affect the interpretation of the statement, thus <code>B negativeCorrelation A</code> is equivalent to <code>A negativeCorrelation B</code>.</p>
</div>
</div>
<div class="sect3">
<h4 id="XposCor">3.2.2. positiveCorrelation, pos</h4>
<div class="paragraph">
<p>For terms A and B, <code>A positiveCorrelation B</code> or <code>A pos B</code> indicates that changes in A and B have been observed to be positively correlated. The order of the subject and object does not affect the interpretation of the statement, thus <code>B positiveCorrelation A</code> is equivalent to <code>A positiveCorrelation B</code>.</p>
</div>
</div>
<div class="sect3">
<h4 id="Xassociation">3.2.3. association,&#8201;&#8212;&#8201;</h4>
<div class="paragraph">
<p>For terms A and B, <code>A association B</code> or <code>A&#8201;&#8212;&#8201;B</code> indicates that A and B are associated in an unspecified manner. This relationship is used when not enough information about the association is available to describe it using more specific relationships, like <code>increases</code> or <code>positiveCorrelation</code>. The order of the subject and object does not affect the interpretation of the statement, thus <code>B&#8201;&#8212;&#8201;A</code> is equivalent to <code>A&#8201;&#8212;&#8201;B</code>.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_genomic_relationships">3.3. Genomic Relationships</h3>
<div class="paragraph">
<p>These relationship types link related terms, like orthologous terms from two different species or the <code>geneAbundance()</code> and <code>rnaAbundance()</code> terms for the same namespace value.</p>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
<div class="paragraph">
<p>In most cases, these relationships will be introduced by the BEL Namespace resources, and are not needed for creation of BEL Statements and BEL Documents.</p>
</div>
</td>
</tr>
</table>
</div>
<div class="sect3">
<h4 id="_orthologous">3.3.1. orthologous</h4>
<div class="paragraph">
<p>For terms A and B, <code>A orthologous B</code> indicates that A and B represent entities in different species which are sequence similar and which are therefore presumed to share a common ancestor. For example,</p>
</div>
<div class="literalblock">
<div class="content">
<pre>g(HGNC:AKT1) orthologous g(MGI:AKT1)</pre>
</div>
</div>
<div class="paragraph">
<p>indicates that the mouse and human AKT1 genes are orthologs.</p>
</div>
</div>
<div class="sect3">
<h4 id="_transcribedto">3.3.2. transcribedTo, :&gt;</h4>
<div class="paragraph">
<p>For RNA abundance term R and gene abundance term G, <code>G transcribedTo R</code> or <code>G :&gt; R</code> indicates that members of R are produced by the transcription of members of G. For example:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>g(HGNC:AKT1) :&gt; r(HGNC:AKT1)</pre>
</div>
</div>
<div class="paragraph">
<p>indicates that the human AKT1 RNA is transcribed from the human AKT1 gene.</p>
</div>
</div>
<div class="sect3">
<h4 id="_translatedto">3.3.3. translatedTo, &gt;&gt;</h4>
<div class="paragraph">
<p>For RNA abundance term R and protein abundance term P, <code>R translatedTo P</code> or <code>R &gt;&gt; P</code> indicates that members of P are produced by the translation of members of R. For example:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>r(HGNC:AKT1) &gt;&gt; p(HGNC:AKT1)</pre>
</div>
</div>
<div class="paragraph">
<p>indicates that AKT1 protein is produced by translation of AKT1 RNA.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_other_relationships">3.4. Other Relationships</h3>
<div class="paragraph">
<p>Additional miscellaneous relationship types.
Icon
In most cases, these relationships will be introduced by the BEL Namespace resources, and are not needed for creation of BEL Statements and BEL Documents.</p>
</div>
<div class="sect3">
<h4 id="_hasmember">3.4.1. hasMember</h4>
<div class="paragraph">
<p>For term abundances A and B, <code>A hasMember B</code> designates B as a member class of A. A member class is a distinguished sub-class. A is defined as a group by all of the members assigned to it. The member classes may or may not be overlapping and may or may not entirely cover all instances of A. A term may not appear in both the subject and object of the same hasMember statement.</p>
</div>
</div>
<div class="sect3">
<h4 id="_hasmembers">3.4.2. hasMembers</h4>
<div class="paragraph">
<p>The <code>hasMembers</code> relationship is a special form which enables the assignment of multiple member classes in a single statement where the object of the statement is a set of abundance terms. A statement using <code>hasMembers</code> is exactly equivalent to multiple <code>hasMember</code> statements. A term may not appear in both the subject and object of the same <code>hasMembers</code> statement.</p>
</div>
<div class="paragraph">
<p>For the abundance terms A, B, C and D, <code>A hasMembers list(B, C, D)</code> indicates that A is defined by its member abundance classes B, C and D.</p>
</div>
</div>
<div class="sect3">
<h4 id="_hascomponent">3.4.3. hasComponent</h4>
<div class="paragraph">
<p>For complex abundance term A and abundance term B, <code>A hasComponent B</code> designates B as a component of A, that complexes that are instances of A have instances of B as possible components. Note that, the stoichiometry of A is not described, nor is it stated that B is a required component.
The use of <code>hasComponent</code> relationships is complementary to the use of functionally composed complexes and is intended to enable the assignment of components to complexes designated by names in external vocabularies. The assignment of components can potentially enable the reconciliation of equivalent complexes at knowledge assembly time.</p>
</div>
</div>
<div class="sect3">
<h4 id="_hascomponents">3.4.4. hasComponents</h4>
<div class="paragraph">
<p>The <code>hasComponents</code> relationship is a special form which enables the assignment of multiple complex components in a single statement where the object of the statement is a set of abundance terms. A statement using <code>hasComponents</code> is exactly equivalent to multiple <code>hasComponent</code> statements. A term may not appear in both the subject and object of the same hasComponents statement.</p>
</div>
<div class="paragraph">
<p>For the abundance terms A, B, C and D, <code>A hasComponents list(B, C, D)</code> indicates that A has components B, C and D.</p>
</div>
</div>
<div class="sect3">
<h4 id="_isa">3.4.5. isA</h4>
<div class="paragraph">
<p>For terms A and B, <code>A isA B</code> indicates that A is a subset of B.</p>
</div>
<div class="paragraph">
<p>All terms in BEL 1.0 represent classes, but given that classes implicitly have instances, <code>A isA B</code> is interpreted to mean that any instance of A must also be an instance of B. This relationship can be used to represent GO and MeSH hierarchies:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>pathology(MESH:Psoriasis) isA pathology(MESH:"Skin Diseases")</pre>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_subprocessof">3.4.6. subProcessOf</h4>
<div class="paragraph">
<p>For process, activity, or transformation term A and process term P, <code>A subProcessOf P</code> indicates that instances of process P, by default, include one or more instances of A in their composition. For example, the reduction of HMG-CoA to mevalonate is a subprocess of cholesterol biosynthesis:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>rxn(reactants(a(CHEBI:"(S)-3-hydroxy-3-methylglutaryl-CoA"),a(CHEBI:NADPH), a(CHEBI:hydron)),\
 products(a(CHEBI:mevalonate), a(CHEBI:"CoA-SH"), a(CHEBI:"NADP(+)"))) subProcessOf\
 bp(GOBP:"cholesterol biosynthetic process")</pre>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_deprecated_relationships">3.5. Deprecated Relationships</h3>
<div class="admonitionblock warning">
<table>
<tr>
<td class="icon">
<div class="title">Warning</div>
</td>
<td class="content">
<div class="paragraph">
<p>These BEL v1.0 relationships are supported in BEL v2.0, but are slated to be removed in the next major version.</p>
</div>
</td>
</tr>
</table>
</div>
<div class="sect3">
<h4 id="_analogous">3.5.1. analogous</h4>
<div class="paragraph">
<p>For terms A and B, <code>A analogousTo B</code> indicates that A and B represent abundances or molecular activities which function in a similar manner, but do not share sequence similarity or a common ancestor.</p>
</div>
</div>
<div class="sect3">
<h4 id="_biomarkerfor">3.5.2. biomarkerFor</h4>
<div class="paragraph">
<p>For term A and process term P, <code>A biomarkerFor P</code> indicates that changes in or detection of A is used in some way to be a biomarker for pathology or biological process P.</p>
</div>
</div>
<div class="sect3">
<h4 id="_prognosticbiomarkerfor">3.5.3. prognosticBiomarkerFor</h4>
<div class="paragraph">
<p>For term A and process term P, <code>A prognosticBiomarkerFor P</code> indicates that changes in or detection of A is used in some way to be a prognostic biomarker for the subsequent development of pathology or biological process P.</p>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_appendices">4. Appendices</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Additional information supporting the BEL Language specification.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_namespaces_used_in_examples">Namespaces Used in Examples</a></p>
</li>
<li>
<p><a href="#_bel_examples">BEL Examples</a></p>
</li>
<li>
<p><a href="#_bel_best_practices_updated_for_bel_v2">BEL Best Practices - Updated for BEL v2</a></p>
</li>
</ul>
</div>
<div class="sect2">
<h3 id="_namespaces_used_in_examples">4.1. Namespaces Used in Examples</h3>
<div class="paragraph">
<p>Namespaces are a reference to the specific vocabulary that a value used in a BEL Term comes from. The examples in this documentation use the following set of BEL Namespaces (<a href="http://wiki.openbel.org/display/BELNA/Namespaces+Overview">v20131211</a>) to reference external ontologies and vocabularies:</p>
</div>
<table class="tableblock frame-all grid-all spread">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Namespace Abbreviation</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Namespace Description</strong></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">EGID</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Entrez Gene IDs</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">HGNC</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">HGNC human gene symbols</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MGI</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MGI mouse gene symbols</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">RGD</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RGD rat gene symbols</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SP</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SwissProt accession numbers</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MESHD</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Medical Subject Heading Disease names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MESHCS</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Medical Subject Heading Cellular Structure names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MESHPP</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Medical Subject Heading Process names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">CHEBI</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chemicals of Biological Interest names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">GOBP</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gene Ontology Biological Process names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">GOCC</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Gene Ontology Cellular Component names</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SCOMP</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Selventa Named Complexes</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">SFAM</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Selventa Protein Families</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_bel_examples">4.2. BEL Examples</h3>
<div class="paragraph">
<p>The following pages contain examples of BEL Terms and BEL Statements. BEL Terms are used to represent biological entities including abundances and processes. These terms are used as the basis of BEL Statements that link one or more BEL Terms together with a relationship and/or additional context information to represent biological knowledge.</p>
</div>
<div class="paragraph">
<p>These examples are written in BEL Script format; see documentation for more information.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_bel_term_examples">BEL Term Examples</a></p>
</li>
<li>
<p><a href="#_bel_statement_examples">BEL Statement Examples</a></p>
</li>
<li>
<p><a href="#_other_examples">Other Examples</a></p>
</li>
</ul>
</div>
<div class="sect3">
<h4 id="_bel_term_examples">4.2.1. BEL Term Examples</h4>
<div class="ulist">
<ul>
<li>
<p><a href="#_abundance_term_examples">Abundance Term Examples</a></p>
</li>
<li>
<p><a href="#_activity_term_examples">Activity Term Examples</a></p>
</li>
<li>
<p><a href="#_binding_interaction_term_examples">Binding Interaction Term Examples</a></p>
</li>
<li>
<p><a href="#_biological_processes_and_pathologies_term_examples">Biological Processes and Pathologies Term Examples</a></p>
</li>
<li>
<p><a href="#_post_translationally_modified_protein_term_examples">Post-Translationally Modified Protein Term Examples</a></p>
</li>
<li>
<p><a href="#Xtte">Transformation Term Examples (Reactions, Translocations, Degradation)</a></p>
</li>
<li>
<p><a href="#_variant_mutant_protein_examples">Variant (Mutant) Protein Examples</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_abundance_term_examples">Abundance Term Examples</h5>
<div class="paragraph">
<p>Measurable entities like genes, RNAs, proteins, and small molecules are represented as abundances in BEL. BEL Terms for abundances have the general form <code>a(ns:v)</code>, where <code>a</code> is an abundance function, <code>ns</code> is <code>a</code> namespace reference and <code>v</code> is a value from the namespace vocabulary. See <a href="#_namespaces_used_in_examples">Namespaces Used in Examples</a>.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_chemicals_and_small_molecules">Chemicals and Small Molecules</a></p>
</li>
<li>
<p><a href="#Xgenesrnasproteins">Genes, RNAs, and proteins</a></p>
</li>
<li>
<p><a href="#_protein_families">Protein families</a></p>
</li>
<li>
<p><a href="#_micrornas">microRNAs</a></p>
</li>
<li>
<p><a href="#_complexes">Complexes</a></p>
</li>
<li>
<p><a href="#_composite_abundances">Composite abundances</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_chemicals_and_small_molecules">Chemicals and Small Molecules</h6>
<div class="paragraph">
<p>The general abundance function <code><a href="#Xabundancea">abundance()</a></code> is used to represent abundances of chemicals, small molecules, and any other entities that cannot be represented by a more specific abundance function.</p>
</div>
</div>
<div class="sect5">
<h6 id="_examples_8">Examples</h6>

</div>
<div class="sect5">
<h6 id="_long_form">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>abundance(CHEBI:"nitrogen atom")
abundance(CHEBI:"prostaglandin J2")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>a(CHEBI:"nitrogen atom")
a(CHEBI:"prostaglandin J2")</pre>
</div>
</div>
<div class="paragraph">
<p>These BEL Terms represent the abundance of the entities specified by <em>nitrogen atom</em> and by <em>prostaglandin J2</em> in the CHEBI namespace.</p>
</div>
</div>
<div class="sect5">
<h6 id="Xgenesrnasproteins">Genes, RNAs, and proteins</h6>
<div class="paragraph">
<p>The abundance functions <code><a href="#XgeneA">geneAbundance()</a></code>, <code><a href="#XrnaA">rnaAbundance()</a></code>, and <code><a href="#XproteinA">proteinAbundance()</a></code> are used with namespace values like HGNC human gene symbols, EntrezGene IDs, SwissProt accession numbers to designate the type of molecule represented.</p>
</div>
</div>
<div class="sect5">
<h6 id="_examples_9">Examples</h6>
<div class="paragraph">
<p>Abundances of the gene, RNA, and protein encoded by the human AKT1 gene are represented as:</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_2">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>geneAbundance(HGNC:AKT1)
rnaAbundance(HGNC:AKT1)
proteinAbundance(HGNC:AKT1)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_2">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>g(HGNC:AKT1)
r(HGNC:AKT1)
p(HGNC:AKT1)</pre>
</div>
</div>
<div class="paragraph">
<p>These BEL Terms represent the gene, RNA, and protein abundances of the entity specified by <em>AKT1</em> in the HGNC namespace. Equivalent terms can be constructed using a corresponding value from a different namespace. For example, the abundance of the human AKT1 RNA can also be represented by referencing the EntrezGene ID or SwissProt accession namespaces:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>r(EGID:207)
r(SP:P31749)</pre>
</div>
</div>
<div class="paragraph">
<p>The BEL Framework identifies and merges corresponding terms created using different namespaces into a single term through namespace equivalencing.</p>
</div>
</div>
<div class="sect5">
<h6 id="_protein_families">Protein families</h6>
<div class="paragraph">
<p>Protein families are used to represent a group of functionally similar proteins. For example, AKT1, AKT2, and AKT3 together form the AKT family. Like other proteins, abundances of protein families are represented using the <code><a href="#XproteinA">proteinAbundance()</a></code> function, with namespace values from the Selventa named protein families namespace.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_6">Example</h6>
<div class="paragraph">
<p>This term represents the protein abundance of the AKT protein family.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"AKT Family")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_micrornas">microRNAs</h6>
<div class="paragraph">
<p>The abundance function <code><a href="#XmicroRNAA">microRNAAbundance()</a></code> is used to represent the fully processed, active form of a microRNA. The specific abundance functions allow distinct representations of the gene, RNA, and microRNA abundances for a given namespace value.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_7">Example</h6>
<div class="paragraph">
<p>These BEL Terms represent the abundances of the gene, RNA, and processed microRNA, respectively, for the entity specified by <em>Mir21</em> in the MGI mouse gene symbol namespace.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_3">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>geneAbundance(MGI:Mir21)
rnaAbundance(MGI:Mir21)
microRNAAbundance(MGI:Mir21)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_3">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>g(MGI:Mir21)
r(MGI:Mir21)
m(MGI:Mir21)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_complexes">Complexes</h6>
<div class="paragraph">
<p>The abundances of molecular complexes are represented using the <code><a href="#XcomplexA">complexAbundance()</a></code> function. This function can take either a list of abundance terms or a value from a namespace of molecular complexes as its argument.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_8">Example</h6>
<div class="paragraph">
<p>Both BEL Terms represent the IkappaB kinase complex. The first by referencing a named protein complex within the <a href="http://geneontology.org/page/cellular-component-ontology-guidelines">GO Cellular Component namespace</a>, and the second by enumerating the individual protein abundances that compose the IkappaB kinase complex, CHUK, IKBKB, and IKBKG.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_4">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>complexAbundance(GOCC:"IkappaB kinase complex")
complexAbundance(proteinAbundance(HGNC:CHUK), proteinAbundance(HGNC:IKBKB), proteinAbundance(HGNC:IKBKG))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_4">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>complex(GOCC:"IkappaB kinase complex")
complex(p(HGNC:CHUK), p(HGNC:IKBKB), p(HGNC:IKBKG))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_composite_abundances">Composite abundances</h6>
<div class="paragraph">
<p>Multiple abundance terms can be represented together as the subject of a BEL Statement by using the <code><a href="#XcompositeA">compositeAbundance()</a></code> function. This function takes a list of abundances as its argument and is used when the individual abundances do not act alone, but rather synergize to produce an effect.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_9">Example</h6>
<div class="paragraph">
<p>This term represents the combined abundances of TGFB1 and IL6 proteins.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_5">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>compositeAbundance(proteinAbundance(HGNC:TGFB1), proteinAbundance(HGNC:IL6))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_5">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>composite(p(HGNC:TGFB1), p(HGNC:IL6))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_activity_term_examples">Activity Term Examples</h5>
<div class="paragraph">
<p>Term activity functions are applied to protein, complex, and RNA abundances to specify the frequency of events resulting from the molecular activity of the abundance. This distinction is particularly useful for proteins whose activities are regulated by post-translational modification. Specific activity types can be indicated using the <code><a href="#XmolecularA">molecularActivity()</a></code> process modifier function. The default BEL namespace includes molecular activity values corresponding to the <a href="http://wiki.openbel.org/display/BLD/Activities">BEL v1.0 activity functions</a>, and <a href="http://geneontology.org/page/molecular-function-ontology-guidelines">GO Molecular Function</a> namespace values can be used to indicate more specific molecular activities.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_non_specified_activities">Non-Specified Activities</a></p>
</li>
<li>
<p><a href="#_catalytic_activity">Catalytic Activity</a></p>
</li>
<li>
<p><a href="#_peptidase_activity">Peptidase Activity</a></p>
</li>
<li>
<p><a href="#_g_proteins_in_the_active_gtp_bound_state">G-proteins in the active (GTP-bound) state</a></p>
</li>
<li>
<p><a href="#_transporter_activity">Transporter Activity</a></p>
</li>
<li>
<p><a href="#_chaperone_activity">Chaperone Activity</a></p>
</li>
<li>
<p><a href="#_transcription_activity">Transcription Activity</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_non_specified_activities">Non-Specified Activities</h6>
<div class="paragraph">
<p>If the type of molecular activity is not reported, it does not need to be specified. The <code>activity()</code> function is sufficient for distinguishing the frequency of events mediated by an abundance from the amount of the abundance. This term represents the ligand-bound activity of the human non-catalytic receptor protein TLR7.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_6">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(HGNC:TLR7))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_6">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:TLR7))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_catalytic_activity">Catalytic Activity</h6>
<div class="paragraph">
<p>A protein, complex, or ribozymes has catalytic activity when it acts as an enzymatic catalyst of biochemical reactions. Catalytic activity includes kinase, phosphatase, peptidase, and ADP-ribosylase activities, though these can be represented by more specific molecular activity terms.</p>
</div>
<div class="paragraph">
<p>This term represents the frequency of events in which the protein abundance of rat Sod1 acts as a catalyst.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(RGD:Sod1), ma(cat))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_gomf_namespace">Long Form - GO Molecular Function (GOMF) namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(RGD:Sod1), molecularActivity(GOMF:"catalytic activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(RGD:Sod1), ma(cat))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_go_molecular_function_namespace">short Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(RGD:Sod1), ma(GOMF:"catalytic activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_peptidase_activity">Peptidase Activity</h6>
<div class="paragraph">
<p>This term represents the frequency of events in which the protein abundance of mouse Casp3 acts as a peptidase.The more specific GO Molecular Function term "cysteine-type endopeptidase activity" is also applicable.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace_2">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(MGI:Casp3), molecularActivity(pep))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_namespace">Long Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(MGI:Casp3), molecularActivity(GOMF:"peptidase activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace_2">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Casp3), ma(pep))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_go_molecular_function_namespace_2">Short Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Casp3), ma(GOMF:"peptidase activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_g_proteins_in_the_active_gtp_bound_state">G-proteins in the active (GTP-bound) state</h6>
<div class="paragraph">
<p>The activity of guanine nucleotide-binding proteins (G-proteins) like RAS in the active, GTP-bound state. This term represents the frequency of events caused by the active, GTP-bound form of the RAS protein family.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace_3">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(SFAM:"RAS Family"), molecularActivity(gtp))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_namespace_2">Long Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(SFAM:"RAS Family"), molecularActivity(GOMF:"GTP binding"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace_3">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(SFAM:"RAS Family"), ma(gtp))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_go_molecular_function_namespace_3">Short Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(SFAM:"RAS Family"), ma(GOMF:"GTP binding"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_transporter_activity">Transporter Activity</h6>
<div class="paragraph">
<p>Molecular translocation events mediated by transporter proteins like ion channels or glucose transporters. This term represents the frequency of ion transport events mediated by the epithelial sodium channel (ENaC) complex.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace_4">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(complexAbundance(SCOMP:"ENaC Complex"), molecularActivity(tport))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_namespace_3">Long Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(complexAbundance(SCOMP:"ENaC Complex"), molecularActivity(GOMF:"transporter activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace_4">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(complex(NCH:"ENaC Complex"), ma(tport))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_go_molecular_function_namespace_4">Short Form - GO Molecular Function namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(complex(NCH:"ENaC Complex"), ma(GOMF:"transporter activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_chaperone_activity">Chaperone Activity</h6>
<div class="paragraph">
<p>This term represents the events in which the human Calnexin protein functions as a chaperone to aid the folding of other proteins.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace_5">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(HGNC:CANX), molecularActivity(chap))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace_5">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:CANX), ma(chap))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_transcription_activity">Transcription Activity</h6>
<div class="paragraph">
<p>Events in which a protein or molecular complex acts to directly control transcription, including proteins acting directly as transcription factors, as well as transcriptional co-activators and co-repressors. This term represents the frequency of events in which the mouse p53 protein controls RNA expression.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_default_bel_namespace_6">Long Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(MGI:Trp53), molecularActivity(tscript))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_namespace_4">Long Form - GO Molecular Function Namespace</h6>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(MGI:Trp53), molecularActivity(GOMF:"nucleic acid binding transcription factor activity"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_default_bel_namespace_6">Short Form - default BEL namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Trp53), ma(tscript))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_go_molecular_function_namespace_5">Long Form - GO Molecular Function Namespace</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Trp53), ma(GOMF:"nucleic acid binding transcription factor activity"))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_binding_interaction_term_examples">Binding Interaction Term Examples</h5>
<div class="paragraph">
<p>The <code><a href="#XcomplexA">complexAbundance()</a></code> function can be used to specify molecular interactions between abundances. This function can take either a list of abundances that define a molecular complex or a namespace value that represents a molecular complex (e.g., many GO Cellular Component values) as an argument. These examples demonstrate the use of the <code>complexAbundance()</code> function to represent protein-protein, protein-chemical, and protein-DNA interactions.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_protein_protein_interactions">Protein  protein interactions</a></p>
</li>
<li>
<p><a href="#_protein_dna_interactions">Protein  DNA interactions</a></p>
</li>
<li>
<p><a href="#_protein_small_molecule_interactions">Protein  small molecule interactions</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_protein_protein_interactions">Protein  protein interactions</h6>

</div>
<div class="sect5">
<h6 id="_example_protein_protein_interaction_as_bel_statement">Example - protein-protein interaction as BEL statement</h6>
<div class="paragraph">
<p>This statement represents that MTOR and AKT1S1 proteins physically interact. Note that this statement has only an object term and no subject term and relationship.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_7">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Nat Cell Biol 2007 Mar 9(3) 316-23", "17277771"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Here, we identify PRAS40 (proline-rich Akt/PKB substrate
 40 kDa) as a novel mTOR binding partner"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation PRAS40 = HGNC AKT1S1</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complexAbundance(proteinAbundance(HGNC:AKT1S1), proteinAbundance(HGNC:MTOR))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_7">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:AKT1S1), p(HGNC:MTOR))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_protein_protein_interaction_as_statement_object">Example - protein-protein interaction as Statement object</h6>
<div class="paragraph">
<p>Here, a protein-protein interaction is the object of a BEL Statement.This statement expresses that the MTOR and STAT3 proteins associate and that increases in the protein abundance of BMP4 can increase the abundance of the complex comprised of MTOR and STAT3.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_8">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "J Cell Biol. 2003 Jun 9;161(5):911-21.", "12796477"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Upon BMP4 treatment, the serine-threonine kinase
FKBP12/rapamycin-associated protein (FRAP), mammalian target of
rapamycin (mTOR), associates with Stat3 and facilitates STAT activation."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:BMP4) increases complexAbundance(proteinAbundance(HGNC:MTOR), proteinAbundance(HGNC:STAT3))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_8">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:BMP4) -&gt; complex(p(HGNC:MTOR), p(HGNC:STAT3))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_protein_dna_interactions">Protein  DNA interactions</h6>

</div>
<div class="sect5">
<h6 id="_example_transcription_factor_protein_binding_to_dna">Example - transcription factor protein binding to DNA</h6>
<div class="paragraph">
<p>This statement expresses that STAT3 protein binds to the CCL11 gene DNA, and that this association is increased by IL17A.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_9">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "J Immunol 2009 Mar 15 182(6) 3357-65", "19265112"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "IL-17A induced at 1 h a marked enrichment of
 STAT3- associated CCL11 promoter DNA"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:IL17A) increases \
 complexAbundance(proteinAbundance(HGNC:STAT3), geneAbundance(HGNC:CCL11))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_9">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:IL17A) -&gt; complex(p(HGNC:STAT3), g(HGNC:CCL11))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_protein_small_molecule_interactions">Protein  small molecule interactions</h6>

</div>
<div class="sect5">
<h6 id="_example_protein_binding_to_a_small_molecule">Example - protein binding to a small molecule</h6>
<div class="paragraph">
<p>This statement represents that PIP3 binds AKT1 protein.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_10">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Breast Cancer Res 2005 7(4) R394-401", "15987444"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Evidence = "After PIP3 binding, Akt1 is activated"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation PIP3 = CHEBI 1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complexAbundance(abundance(CHEBI:"1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate"), proteinAbundance(HGNC:AKT1))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_10">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>complex(a(CHEBI:"1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate"), p(HGNC:AKT1))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_biological_processes_and_pathologies_term_examples">Biological Processes and Pathologies Term Examples</h5>
<div class="paragraph">
<p>Biological phenomena that occur at the level of the cell or the organism are considered processes. These terms are represented by values from namespaces like GO and MeSH.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_biological_processes">Biological Processes</a></p>
</li>
<li>
<p><a href="#_diseases_and_pathologies">Diseases and Pathologies</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_biological_processes">Biological Processes</h6>
<div class="paragraph">
<p>Cellular senescence can be represented by:</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_11">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>biologicalProcess(GOBP:"cellular senescence")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_11">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>bp(GOBP:"cellular senescence")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_diseases_and_pathologies">Diseases and Pathologies</h6>
<div class="paragraph">
<p>Disease pathologies like muscle hypotonia can be represented by:</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_12">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>pathology(MESHD:"Muscle Hypotonia")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_12">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>path(MESHD:"Muscle Hypotonia")</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_post_translationally_modified_protein_term_examples">Post-Translationally Modified Protein Term Examples</h5>
<div class="paragraph">
<p>The <code>proteinModification()</code> or <code>pmod()</code> function is used within a protein abundance to specify post-translational modifications. Types of post-translational modification are specified by a namespace value; the default BEL namespace provides many commonly used modification types. Abundances of modified proteins take the form <code>p(ns:v, pmod(ns:type_value, &lt;code&gt;, &lt;pos&gt;))</code>, where <code>&lt;type&gt;</code> (required) is the kind of modification, <code>&lt;code&gt;</code> (optional) is the one- or three- letter <a href="#_supported_one_and_three_letter_amino_acid_codes">amino acid code</a> for the modified residue, and <code>&lt;pos&gt;</code> (optional) is the sequence position of the modification.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_hydroxylation">Hydroxylation</a></p>
</li>
<li>
<p><a href="#_phosphorylation">Phosphorylation</a></p>
</li>
<li>
<p><a href="#_acetylation">Acetylation</a></p>
</li>
<li>
<p><a href="#_glycosylation">Glycosylation</a></p>
</li>
<li>
<p><a href="#_methylation">Methylation</a></p>
</li>
<li>
<p><a href="#_ubiquitination">Ubiquitination</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_hydroxylation">Hydroxylation</h6>
<div class="paragraph">
<p>This term represents the abundance of human HIF1A protein hydroxylated at asparagine 803.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_13">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:HIF1A, proteinModification(Hy, Asn, 803))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_13">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:HIF1A, pmod(Hy, N, 803))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_phosphorylation">Phosphorylation</h6>
<div class="paragraph">
<p>This term represents the phosphorylation of the human AKT protein family at an unspecified amino acid residue.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"AKT Family", pmod(Ph))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_acetylation">Acetylation</h6>
<div class="paragraph">
<p>This term represents the abundance of mouse RELA protein acetylated at lysine 315.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(MGI:Rela, pmod(Ac, Lys, 315))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_glycosylation">Glycosylation</h6>
<div class="paragraph">
<p>This term represents the abundance of human SP1 protein glycosylated at an unspecified amino acid residue.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:SP1, pmod(Glyco))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_methylation">Methylation</h6>
<div class="paragraph">
<p>This term represents the abundance of rat STAT1 protein methylated at an unspecified arginine residue:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(RGD:STAT1, pmod(Me, Arg))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_ubiquitination">Ubiquitination</h6>
<div class="paragraph">
<p>This term represents the abundance of human MYC protein ubiquitinated at an unspecified lysine residue:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:MYC, pmod(Ub, Lys))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="Xtte">Transformation Term Examples (Reactions, Translocations, Degradation)</h5>
<div class="ulist">
<ul>
<li>
<p><a href="#_reactions">Reactions</a></p>
</li>
<li>
<p><a href="#_translocations">Translocations</a></p>
</li>
<li>
<p><a href="#_degradation">Degradation</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_reactions">Reactions</h6>
<div class="paragraph">
<p>The <code>reaction()</code> or <code>rxn()</code> function expresses the transformation of products into reactants, each defined by a list of abundances.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_10">Example</h6>
<div class="paragraph">
<p>This BEL Term represents the reaction in which the reactants phosphoenolpyruvate and ADP are converted into pyruvate and ATP.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_14">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>reaction(reactants(abundance(CHEBI:phosphoenolpyruvate), abundance(CHEBI:ADP)),\
 products(abundance(CHEBI:pyruvate), abundance(CHEBI:ATP)))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_14">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>rxn(reactants(a(CHEBI:phophoenolpyruvate), a(CHEBI:ADP)),\
 products(a(CHEBI:pyruvate), a(CHEBI:ATP)))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_translocations_2">Translocations</h6>
<div class="paragraph">
<p>Translocations, or the movement of abundances from one location to another, are represented in BEL Terms by the <code>translocation()</code> or <code>tloc()</code> function. For convenience, the frequently used translocations of abundances from inside the cell to cell surface or extracellular space are represented by the <code>cellSurface()</code> and <code>cellSecretion()</code> functions, respectively.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_11">Example</h6>
<div class="paragraph">
<p>This term represents the event in which human NFE2L2 protein is translocated from the cytoplasm to the nucleus.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_15">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>translocation(proteinAbundance(HGNC:NFE2L2), fromLoc(MESHCS:Cytoplasm), toLoc(MESHCS:"Cell Nucleus"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_15">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>tloc(p(HGNC:NFE2L2), fromLoc(MESHCL:Cytoplasm), toLoc(MESHCL:"Cell Nucleus"))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_cell_secretion">Example - cell secretion</h6>
<div class="paragraph">
<p>This term represents secretion of mouse IL6 protein.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_16">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>cellSecretion(proteinAbundance(MGI:Il6))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_16">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>sec(p(MGI:Il6))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_cell_surface_expression">Example - cell surface expression</h6>
<div class="paragraph">
<p>This term represents cell surface expression of rat Fas protein.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_17">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>cellSurfaceExpression(proteinAbudance(RGD:Fas))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_17">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>surf(p(RGD:Fas))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_degradation">Degradation</h6>
<div class="paragraph">
<p>Events in which an abundance is degraded can be represented by the <code>degradation()</code> or <code>deg()</code> function.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_12">Example</h6>
<div class="paragraph">
<p>This term represents the degradation of MYC RNA. Degradation decreases the amount of the abundance - when degradation statements are compiled, a directlyDecreases relationship edge is added between the degradation term and the degraded entity.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_18">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>degradation(rnaAbundance(HGNC:MYC))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_18">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>deg(r(HGNC:MYC))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_variant_mutant_protein_examples">Variant (Mutant) Protein Examples</h5>
<div class="paragraph">
<p>The abundances of mutated and variant proteins can be represented in BEL using the abundance modifier function <code>variant()</code> and the other function <code>fusion()</code>.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_amino_acid_substitutions">Amino Acid Substitutions</a></p>
</li>
<li>
<p><a href="#_truncated_proteins">Truncated Proteins</a></p>
</li>
<li>
<p><a href="#_fusion_proteins">Fusion Proteins</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_amino_acid_substitutions">Amino Acid Substitutions</h6>
<div class="paragraph">
<p>The abundances of proteins with amino acid sequence variations, such as those resulting from missense mutations or polymorphisms can be specified by using the <code>variant()</code> or <code>var()</code> function within a protein abundance term.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_13">Example</h6>

</div>
<div class="sect5">
<h6 id="_long_form_19">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:PIK3CA, variant(p.Glu545Lys))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_19">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:PIK3CA, var(p.Glu545Lys))</pre>
</div>
</div>
<div class="paragraph">
<p>This term represents the abundance of the human PIK3CA protein in which the glutamic acid residue at position 545 has been substituted with a lysine.</p>
</div>
</div>
<div class="sect5">
<h6 id="_truncated_proteins">Truncated Proteins</h6>
<div class="paragraph">
<p>The abundances of proteins that are truncated by the introduction of a stop codon can be specified by using the <code>variant()</code> or <code>var()</code> function within a protein abundance term.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_14">Example</h6>

</div>
<div class="sect5">
<h6 id="_long_form_20">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:ABCA1, variant(p.Arg1851*))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_20">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:ABCA1, var(p.Arg1851*))</pre>
</div>
</div>
<div class="paragraph">
<p>This term represents the abundance of human ABCA1 protein that has been truncated by substitution of Arginine 1851 with a stop codon.</p>
</div>
</div>
<div class="sect5">
<h6 id="_fusion_proteins">Fusion Proteins</h6>
<div class="paragraph">
<p>The abundances of fusion proteins resulting from chromosomal translocation mutations can be specified by using the <code>fusion()</code> or <code>fus()</code> function within a protein abundance term.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_15">Example</h6>

</div>
<div class="sect5">
<h6 id="_long_form_21">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(fusion(HGNC:BCR, p.1_426, HGNC:JAK2, p.812_1132))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_21">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(fus(HGNC:BCR, p.1_426, HGNC:JAK2, p.812_1132))</pre>
</div>
</div>
<div class="paragraph">
<p>This term represents the abundance of a fusion protein of the 5' partner BCR and 3' partner JAK2, with the breakpoint for BCR at amino acid 426 and JAK2 at 812. <em>p.</em> indicates that the protein sequence is used for the range coordinates provided. If the breakpoint is not specified, the fusion protein abundance can be represented as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(fus(HGNC:BCR, ?, HGNC:JAK2, ?))</pre>
</div>
</div>
<div class="paragraph">
<p>The <code>fusion()</code> function can also be used within <code>geneAbundance</code> and <code>rnaAbundance</code> terms to represent genes and RNAs modified by fusion mutations.</p>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_bel_statement_examples">4.2.2. BEL Statement Examples</h4>
<div class="ulist">
<ul>
<li>
<p><a href="#_causal_statement_examples">Causal Statement Examples</a></p>
</li>
<li>
<p><a href="#_correlative_statement_examples">Correlative Statement Examples</a></p>
</li>
<li>
<p><a href="#_direct_causal_statement_examples">Direct Causal Statement Examples</a></p>
</li>
<li>
<p><a href="#_nested_statement_example">Nested Statement Example</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_causal_statement_examples">Causal Statement Examples</h5>
<div class="paragraph">
<p>Causal statements connect subject and object terms with a causal <code><a href="#Xincreases">increases</a></code>, <code><a href="#Xdecreases">decreases</a></code>, or <code><a href="#Xcnc">causesNoChange</a></code> relationship. Subject terms can be an abundance or process (including activities and transformations) and object terms can be either an abundance, a process, or a second BEL Statement.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_causal_increase">Causal increase</a></p>
</li>
<li>
<p><a href="#_causal_decrease">Causal decrease</a></p>
</li>
<li>
<p><a href="#_causes_no_change">Causes no change</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_causal_increase">Causal increase</h6>

</div>
<div class="sect5">
<h6 id="_example_16">Example</h6>
<div class="paragraph">
<p>These statements use the causal <code><a href="#Xincreases">increases</a></code> relationship. These statements are annotated with a citation and supporting evidence text, as well as with the cell line and species context for the experimental observations represented by the statements. These two statements represent the observation that increases in IL6 protein abundance cause increases in the RNA abundance of ENO1 and XBP1. These statements are annotated with CellLine and Species to indicate that the experimental observation was made in the context of the cell line "U266" and species "9606" (Homo sapiens).</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_22">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Int J Oncol 1999 Jul 15(1) 173-8", "10375612"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Northern blot analysis documented that two
 transcription factor genes chosen for further study, c-myc
 promoter-binding protein (MBP-1) and X-box binding protein 1
 (XBP-1), were up-regulated in U266 cells about 3-fold relative
 to the cell cycle-dependent beta-actin gene 12 h after IL-6
 treatment"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET CellLine = "U266"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Species = "9606"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation MBP-1 = HNGC ENO1</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:IL6) increases rnaAbundance(HGNC:ENO1)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:IL6) increases rnaAbundance(HGNC:XBP1)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_22">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:IL6) -&gt; r(HGNC:ENO1)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:IL6) -&gt; r(HGNC:XBP1)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_causal_decrease">Causal decrease</h6>

</div>
<div class="sect5">
<h6 id="_example_17">Example</h6>
<div class="paragraph">
<p>This statement demonstrates a causal statement using the <code><a href="#Xdecreases">decreases</a></code> relationship. The statement expresses that increases in the abundance of corticosteroid molecules cause decreases in the frequency or intensity of the biological process inflammation. This statement is annotated with an Anatomy and Disease to indicate that the relationship was observed in the context of the <em>cardiovascular system</em> and the disease <em>Stroke</em>.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_23">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "J Mol Med. 2003 Mar;81(3):168-74. Epub 2003 Mar 14.", "12682725"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "high-dose steroid treatment decreases vascular
 inflammation and ischemic tissue damage after myocardial
 infarction and stroke through direct vascular effects involving
 the nontranscriptional activation of eNOS"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Anatomy = "cardiovascular system"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET MeSHDisease = "Stroke"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>abundance(CHEBI:corticosteroid) decreases biologicalProcess(MESHD:Inflammation)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_23">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>a(CHEBI:corticosteroid) -| path(MESHD:Inflammation)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_causes_no_change">Causes no change</h6>
<div class="paragraph">
<p>The <code><a href="#Xcnc">causesNoChange</a></code> relationship can be used to record the lack of an observed effect.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_18">Example</h6>
<div class="paragraph">
<p>The epidermal growth factor receptor (EGFR) ligand amphiregulin (AREG) is observed to increase NF-kappaB transcriptional activity while the EGFR ligand EGF has no effect.These statements express that an increase of AREG protein abundance causes an observed increase in the transcriptional activity of the NF-kappaB complex, and that an increase EGF does not.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_24">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Mol Cancer Res 2007 Aug 5(8) 847-61", "17670913"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Furthermore, EGFR, activated by amphiregulin but not
 epidermal growth factor, results in the prompt activation of the
 transcription factor nuclear factor-kappaB (NF-kappaB)"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation Amphiregulin = HGNC AREG</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:AREG) increases activity(complexAbundance(GOCC:"NF-kappaB complex"), molecularActivity(tscript))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:EGF) causesNoChange activity(complexAbundance(GOCC:"NF-kappaB complex"), molecularActivity(tscript))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_24">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AREG) -&gt; act(complex(GOCC:"NF-kappaB complex"), ma(tscript))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:EGF) causesNoChange act(complex(GOCC:"NF-kappaB complex"), ma(tscript))</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_correlative_statement_examples">Correlative Statement Examples</h5>
<div class="paragraph">
<p><a href="#XcorRels">Correlative Relationships</a> link abundances and biological processes when no causal relationship is known.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_negativecorrelation">negativeCorrelation</a></p>
</li>
<li>
<p><a href="#XassociationCR">association</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_negativecorrelation">negativeCorrelation</h6>
<div class="paragraph">
<p>This statement expresses that an increase in cytoplasmic FGF2 protein positively correlates with an increase in the pathology Chronic Obstructive Pulmonary Disease. The subject and object terms of correlative statements are interchangeable. The <code><a href="#XnegCor">negativeCorrelation</a></code> relationship is used to represent inverse correlative relationships, i.e., a decrease in A is correlated with an increase in B.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "J Pathol. 2005 May;206(1):28-38.", "15772985"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Quantitative digital image analysis revealed
increased cytoplasmic expression of FGF-2 in bronchial epithelium
(0.35 +/- 0.03 vs 0.20 +/- 0.04, p &lt; 0.008) and nuclear
localization in ASM (p &lt; 0.0001) in COPD patients compared with
controls."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Tissue = "epithelium"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:FGF2, location(GOCC:cytoplasm)) positiveCorrelation \
 pathology(MESHD:"Pulmonary Disease, Chronic Obstructive")</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="XassociationCR">association</h6>
<div class="paragraph">
<p>The direction of causal effect or correlation of two abundance or biological process terms is not always specified. The <code><a href="#Xassociation">association</a></code> relationship can be used in these cases.</p>
</div>
<div class="paragraph">
<p>This statement represents that abundance of protein designated by the name Nr2f2 in the MGI namespace is associated in an unspecified manner with the biological process angiogenesis.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_25">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Mech Ageing Dev. 2004 Oct-Nov;125(10-11):719-32.", "15541767"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "COUP-TFII is involved in the angiogenic process in the developing embryos."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation - COUP-TFII refers to MGI Nr2f2</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET MeSHAnatomy = "Embryo, Mammalian"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(MGI:Nr2f2) association biologicalProcess(GOBP:angiogenesis)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_25">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(MGI:NR2F2) -- bp(GOBP:angiogenesis)</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_direct_causal_statement_examples">Direct Causal Statement Examples</h5>
<div class="paragraph">
<p>The following examples demonstrate the use of <a href="#_direct_causal_relationships">direct casual relationships</a> in causal statements. The direct causal relationships <code><a href="#XdIncreases">directlyIncreases</a></code> and <code><a href="#XdDecreases">directlyDecreases</a></code> are special forms of the causal <code><a href="#Xincreases">increases</a></code> and <code><a href="#Xdecreases">decreases</a></code> relationships where the mechanism of the causal relationship involves the physical interaction of entities related to the BEL Statement subject and object terms.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_example_ligand_and_receptor">Example - Ligand and Receptor</a></p>
</li>
<li>
<p><a href="#_example_kinase_and_substrate">Example - Kinase and Substrate</a></p>
</li>
<li>
<p><a href="#_example_catalyst_and_reaction">Example - Catalyst and Reaction</a></p>
</li>
<li>
<p><a href="#_example_self_referential_relationships">Example - Self-Referential Relationships</a></p>
</li>
<li>
<p><a href="#_example_direct_transcriptional_control">Example - Direct Transcriptional Control</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_example_ligand_and_receptor">Example - Ligand and Receptor</h6>
<div class="paragraph">
<p>In this example, the <code><a href="#XdIncreases">directlyIncreases</a></code> relationship is used to represent activation of a receptor by its ligand. This statement expresses that amphiregulin (AREG) activates its receptor, the Epidermal Growth Factor Receptor (EGFR). This relationship is direct because ligands directly interact with their receptors.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_26">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Mol Cancer Res 2007 Aug 5(8) 847-61", "17670913"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Furthermore, EGFR, activated by amphiregulin"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># disambiguation Amphiregulin = HGNC AREG</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre># EGFR is known to have kinase activity</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:AREG) directlyIncreases activity(proteinAbundance(HGNC:EGFR), molecularActivity(kin))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_26">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AREG) =&gt; act(p(HGNC:EGFR), ma(kin))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_kinase_and_substrate">Example - Kinase and Substrate</h6>
<div class="paragraph">
<p>In this example, the <code><a href="#XdIncreases">directlyIncreases</a></code> relationship is used to represent the phosphorylation of a protein substrate by a kinase. This statement expresses that the kinase activity of CDK1 protein causes an increase in the modification of FOXO1 protein by phosphorylation at serine 249. The relationship is direct because the kinase physically interacts with its target.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_27">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Science 2008 Mar 21 319(5870) 1665-8.", "18356527"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "We found that Cdk1 phosphorylated the
 transcription factor FOXO1 at Ser249 in vitro and in vivo."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(HGNC:CDK1), molecularActivity(kin)) directlyIncreases \
 proteinAbundance(HGNC:FOXO1, proteinModification(Ph, Ser, 249))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_27">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:CDK1), ma(kin)) =&gt; p(HGNC:FOXO1, pmod(Ph, S, 249))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_catalyst_and_reaction">Example - Catalyst and Reaction</h6>
<div class="paragraph">
<p>In this example, the direct activation of a reaction by a catalytic enzyme is represented. The statement indicates that an increase in the catalytic activity of ALOX5 increase the transformation of the reactant '5(S)-HPETE' to the products 'leukotriene A4' and 'water'. The relationship is considered direct because ALOX5 protein is the catalyzing enzyme.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_28">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"Other", "Reactome: Leukotriene synthesis", "REACT_15354.1"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Dehydration of 5-HpETE to leukotriene A4. In the
 second step, 5-lipoxygenase converts 5-HpETE to an allylic
 epoxide, leukotriene A4."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(HGNC:ALOX5), molecularActivity(cat)) directlyIncreases \
 reaction(reactants(abundance(CHEBI:"5(S)-HPETE")), \
 products(abundance(CHEBI:"leukotriene A4"), abundance(CHEBI:water)))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_28">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:ALOX5), ma(cat)) =&gt; rxn(reactants(a(CHEBI:"5(S)-HPETE")), products(a(CHEBI:"leukotriene A4"), a(CHEBI:water)))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_self_referential_relationships">Example - Self-Referential Relationships</h6>
<div class="paragraph">
<p>In this example, the <code><a href="#XdDecreases">directlyDecreases</a></code> relationship is used to represent the effect of a protein modification on the activity of the same protein. These statements express that the modification of GSK3A and GSK3B protein by phosphorylation on serines 9 and 21, respectively, inhibits the activity of GSK3A and GSK3B. These relationships are considered direct, because they are self-referential. The modification of the protein abundance by phosphorylation inhibits the activity of the same protein abundance.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_29">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Proc Natl Acad Sci U S A 2000 Oct 24 97(22) 11960-5", "11035810"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "GSK-3 activity is inhibited through phosphorylation
 of serine 21 in GSK-3 alpha and serine 9 in GSK-3 beta."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:GSK3A, proteinModification(Ph, Ser, 21)) \
 directlyDecreases activity(proteinAbundance(HGNC:GSK3A))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:GSK3B, proteinModification(Ph, Ser, 9)) \
 directlyDecreases activity(proteinAbundance(HGNC:GSK3B))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_29">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:GSK3A, pmod(Ph, S, 21)) =| act(p(HGNC:GSK3A))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:GSK3B, pmod(Ph, S, 9)) =| act(p(HGNC:GSK3B))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_example_direct_transcriptional_control">Example - Direct Transcriptional Control</h6>
<div class="paragraph">
<p>In this example, the direct activation of a RNA transcription is encoded. The statement expresses that increases in the transcriptional activity of FOXO1 protein directly increase the RNA abundance of CEBPB. This relationship is considered direct because the transcription factor, FOXO1, directly binds the promoter of the CEBPB gene, increasing the expression of CEBPB RNA.</p>
</div>
</div>
<div class="sect5">
<h6 id="_long_form_30">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Biochem Biophys Res Commun. 2009 Jan 9;378(2):290-5. Epub 2008 Nov 21.", "19026986"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "We found that Foxo1 increased the expression of
 CCAAT/enhancer binding protein (C/EBPbeta, a positive regulator
 of monocyte chemoattractant protein (MCP)-1 and interleukin
 (IL)-6 genes, through directly binding to its promoter."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>activity(proteinAbundance(HGNC:FOXO1), molecularActivity(tscript)) \
 directlyIncreases rnaAbundance(HGNC:CEBPB)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_30">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:FOXO1), ma(tscript)) =&gt; r(HGNC:CEBPB)</pre>
</div>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_nested_statement_example">Nested Statement Example</h5>
<div class="paragraph">
<p>This example demonstrates use of a nested causal statement in which the object of a causal statement is itself a causal statement.
In the relationship described by the evidence text, CLSPN specifically increases the activity of ATR to phosphorylate the target protein CHEK1 and does not affect the kinase activity of ATR towards its other targets. The use of the nested statement allows the representation of the information that CLSPN increases the phosphorylation of CHEK1 via the kinase activity of ATR, without incorrectly indicating that CLSPN generally increases the kinase activity of ATR.</p>
</div>
<div class="sect5">
<h6 id="_long_form_31">Long Form</h6>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Mol Cell Biol 2006 Aug 26(16) 6056-64.", "16880517"}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Species = "9606"</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET SupportingText = "Consistently, the RNAi-mediated ablation of Claspin
 selectively abrogated ATR's ability to phosphorylate Chk1 but not
 other ATR targets."</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>proteinAbundance(HGNC:CLSPN) increases \
(activity(proteinAbundance(HGNC:ATR), molecularActivity(kin)) directlyIncreases proteinAbundance(HGNC:CHEK1, proteinModification(Ph)))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_short_form_31">Short Form</h6>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CLSPN) -&gt; (act(p(HGNC:ATR), ma(kin)) =&gt; p(HGNC:CHEK1, pmod(Ph)))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_other_examples">4.2.3. Other Examples</h4>
<div class="ulist">
<ul>
<li>
<p><a href="#_bel_statement_annotation_examples">BEL Statement Annotation Examples</a></p>
</li>
<li>
<p><a href="#_membership_assignment_examples">Membership Assignment Examples</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_bel_statement_annotation_examples">BEL Statement Annotation Examples</h5>
<div class="paragraph">
<p>Annotations associate context information with BEL Statements, including citation of the source material, evidence text supporting the statement, and the experimental context for the scientific observations represented by the statement. To associate Annotations with statements, Annotations are <code>SET</code> and <code>UNSET</code> within a BEL Document. In the BEL Script syntax, once an Annotation has been <code>SET</code> all following statements inherit the annotation until is explicitly <code>UNSET</code> or a new Annotation of the same type is <code>SET</code>.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_citation">Citation</a></p>
</li>
<li>
<p><a href="#SupportingText">[SupportingText]</a></p>
</li>
<li>
<p><a href="#_species">Species</a></p>
</li>
<li>
<p><a href="#_other_annotation_types">Other Annotation Types</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_citation">Citation</h6>
<div class="paragraph">
<p>Citations are a special type of annotation that references the knowledge source that reports the observation that the statement is based on. Citations are composed of a document type, a document name, a document reference ID, and an optional publication date, authors list and comment field. For example, the citation for a journal article indexed by PubMed would be encoded as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"PubMed", "Genes Cancer. 2010 Jun;1(6):560-567.", "21533016"}</pre>
</div>
</div>
<div class="paragraph">
<p>The document name is a text string containing the reference information, the type is PubMed, and the document reference is the PubMed ID.</p>
</div>
<div class="paragraph">
<p>The citation for a Reactome pathway would be encoded as:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Citation = {"Online Resource", "p53-Dependent G1 DNA Damage Response", "REACT_1625.1"}</pre>
</div>
</div>
<div class="paragraph">
<p>In this case, the document name is the pathway name, the type is <em>Online Resource</em>, and the reference is the Reactome identifier.</p>
</div>
</div>
<div class="sect5">
<h6 id="_support_previously_known_as_supporting_text">Support (previously known as Supporting Text)</h6>
<div class="paragraph">
<p>Support annotations provide the specific text that the statement is derived from. Text should come directly from the abstract or full text of the source referenced by the citation annotation. For example, a support line from the Reactome pathway cited above is:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Supporting = "The p53 protein activates the transcription of cyclin-dependent kinase inhibitor, p21.
p21 inactivates the CyclinE:Cdk2 complexes, and prevent entry of the cell into S phase, leading to G1 arrest."</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_species">Species</h6>
<div class="paragraph">
<p>Species annotations indicate the species context for experimental observation represented by the statement. It is good practice to unambiguously assign species context to BEL Statements, even though many BEL Terms are derived from a species-specific namespace (e.g., HGNC, MGI, RGD). Species annotation uses the <a href="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/">NCBI taxonomy ID</a>:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Species = "9606"</pre>
</div>
</div>
<div class="paragraph">
<p>Sets the species as Homo sapiens.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Species = "10090"</pre>
</div>
</div>
<div class="paragraph">
<p>Sets the species as Mus musculus</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Species = "10116"</pre>
</div>
</div>
<div class="paragraph">
<p>Sets the species as Rattus norvegicus.</p>
</div>
</div>
<div class="sect5">
<h6 id="_other_annotation_types">Other Annotation Types</h6>
<div class="paragraph">
<p>Other types of annotations can be added to statements to indicate the context of the experimental observation supported by the statement, including cell line, cell type, and cellular location. For example:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET Cell = "Adipocytes, White"
SET CellLine = "LoVo"
SET Disease = "Lupus Erythematosus, Systemic"
SET Anatomy = "Pulmonary Artery"</pre>
</div>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
<div class="paragraph">
<p>In a BEL Document each Annotation Type that will be used, except for Citation and SupportingText, must be defined in the document header, along with the values allowed for each.</p>
</div>
</td>
</tr>
</table>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_membership_assignment_examples">Membership Assignment Examples</h5>
<div class="paragraph">
<p>These examples demonstrate the assignment of members to groups. Because all BEL terms denote classes, membership in a group is an important special case where subsets of a class that define the class are designated.</p>
</div>
<div class="admonitionblock tip">
<table>
<tr>
<td class="icon">
<div class="title">Tip</div>
</td>
<td class="content">
<div class="paragraph">
<p>The BEL Framework adds family members to protein families and complex components to named complexes during network compilation.</p>
</div>
</td>
</tr>
</table>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_protein_family">Protein Family</a></p>
</li>
<li>
<p><a href="#_complex_component">Complex Component</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_protein_family">Protein Family</h6>
<div class="paragraph">
<p>In this example, members of a protein family are assigned using the <code><a href="#_hasmember">hasMember</a></code> and <code><a href="#_hasmembers">hasMembers</a></code> relationships.</p>
</div>
<div class="paragraph">
<p>The <code>hasMembers</code> relationship is used to assign a list of protein abundances as members of a protein family. This relationship is a syntactic convenience that is equivalent to the set of two statements using the <code>hasMember</code> relationship. These statements designate the protein abundances of MAPK8 and MAPK9 as members of the JNK MAPK protein family. The term representing the JNK family is a protein abundance based on the name <em>MAPK JNK Family</em> in the Selventa Protein Families namespace.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"MAPK JNK Family") hasMembers list(p(HGNC:MAPK8), p(HGNC:MAPK9))</pre>
</div>
</div>
<div class="paragraph">
<p>The <code>hasMember</code> relationship is used to assign individual protein abundances to a protein family.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"MAPK JNK Family") hasMember p(HGNC:MAPK8)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"MAPK JNK Family") hasMember p(HGNC:MAPK9)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_complex_component">Complex Component</h6>
<div class="paragraph">
<p>In this example components are assigned to a named protein complex using the <a href="#_hascomponent">hasComponent</a> and <a href="#_hascomponents">hasComponents</a> relationships.</p>
</div>
<div class="paragraph">
<p>The <code>hasComponents</code> relationship is similar to the <code>hasMembers</code> relationship and is used to assign a list of abundances as components of a complex.These statements designate the protein abundances of RAD9A, RAD1, and HUS1 as components of the complex abundance of the <em>checkpoint clamp complex</em>.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(GOCC:"checkpoint clamp complex") hasComponents list(p(HGNC:RAD9A), p(HGNC:RAD1), p(HGNC:HUS1))</pre>
</div>
</div>
<div class="paragraph">
<p>The <code>hasComponent</code> relationship is used to assign individual abundances to a named protein complex.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:RAD9A)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:RAD1)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:HUS1)</pre>
</div>
</div>
<div class="paragraph">
<p>The single <code>hasComponents</code> statement is equivalent to the set of three <code>hasComponent</code> statements.</p>
</div>
</div>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_bel_best_practices_updated_for_bel_v2">4.3. BEL Best Practices - Updated for BEL v2</h3>
<div class="paragraph">
<p>These pages contain suggestions and guidelines for representing scientific findings in BEL.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_representation_of_experimental_data">Representation of Experimental Data</a></p>
</li>
<li>
<p><a href="#_statement_annotations">Statement Annotations</a></p>
</li>
<li>
<p><a href="#_modified_proteins">Modified Proteins</a></p>
</li>
<li>
<p><a href="#_reactions">Reactions</a></p>
</li>
<li>
<p><a href="#_protein_protein_interactions_2">Protein-Protein Interactions</a></p>
</li>
<li>
<p><a href="#_protein_families_2">Protein Families</a></p>
</li>
</ul>
</div>
<div class="sect3">
<h4 id="_representation_of_experimental_data">4.3.1. Representation of Experimental Data</h4>
<div class="paragraph">
<p>In a causal BEL Statement, the subject term frequently represents an experimentally manipulated entity while the object term represents a measured entity. Our best practices apply different levels of inference for mapping subject and object terms, particularly for representing 'omic data.</p>
</div>
<div class="ulist">
<ul>
<li>
<p><a href="#_subject_terms_perturbations">Subject Terms (Perturbations)</a></p>
</li>
<li>
<p><a href="#_relationships">Relationships</a></p>
</li>
<li>
<p><a href="#_object_terms_measurements">Object Terms (Measurements)</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_subject_terms_perturbations">Subject Terms (Perturbations)</h5>
<div class="ulist">
<ul>
<li>
<p><a href="#BELv2How should I represent chemical inhibitor experiments?">[BELv2How should I represent chemical inhibitor experiments?]</a></p>
</li>
<li>
<p><a href="#_how_do_i_represent_experiments_that_use_site_directed_mutants">How do I represent experiments that use site-directed mutants?</a></p>
</li>
<li>
<p><a href="#_how_do_i_represent_observations_resulting_from_manipulation_of_two_or_more_entities">How do I represent observations resulting from manipulation of two or more entities?</a></p>
</li>
<li>
<p><a href="#_how_should_i_represent_gene_knock_out_or_rnai_experiments">How should I represent gene knock out or RNAi experiments?</a></p>
</li>
<li>
<p><a href="#_how_should_i_represent_overexpression_experiments">How should I represent overexpression experiments?</a></p>
</li>
<li>
<p><a href="#_when_should_i_use_the_protein_abundance_vs_the_activity_of_a_protein">When should I use the protein abundance vs. the activity of a protein?</a></p>
</li>
</ul>
</div>
<div class="sect5">
<h6 id="_belv2_how_should_i_represent_chemical_inhibitor_experiments">BELv2 How should I represent chemical inhibitor experiments?</h6>
<div class="paragraph">
<p>For experiments where protein activity is perturbed with a chemical inhibitor, we generally use the chemical as the subject term and not the activity of the target protein. In many cases, the effects of the chemical are not specific to the intended target. This representation approach avoids unintended attribution of off-target effects of a chemical to the target protein.</p>
</div>
<div class="paragraph">
<p>For example, treatment of cells with the PI3 kinase inhibitor LY294002 significantly decreases expression of TGFB2 RNA (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20629536">PMID 20629536</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>a(SCHEM:"LY 294002") -| r(HGNC:TGFB2)</pre>
</div>
</div>
<div class="paragraph">
<p>In a case where more information is available, the protein activity targeted by the inhibitor can be used as the subject term. For example, if the effect of LY 29004 on TGFB2 RNA expression was demonstrated to require the PIK3CA gene, we could represent the subject term as the kinase activity of the PIK3CA protein.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:PIK3CA), ma(kin)) -&gt; r(HGNC:TGFB2)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_how_do_i_represent_experiments_that_use_site_directed_mutants">How do I represent experiments that use site-directed mutants?</h6>
<div class="paragraph">
<p>The artificial (laboratory) creation of sequence variants is often used to investigate the effects of protein activity or specific post-translational modifications. These include proteins altered to be constitutively active or dominant negative, as well as proteins with specific amino acid residues altered to prevent phosphorylation. While many of these sequence alterations can be precisely represented with BEL, this may not be the best approach to capturing the observations from experiments that use these constructs.</p>
</div>
</div>
<div class="sect5">
<h6 id="_non_phosphorylatable_mutant">Non-phosphorylatable mutant</h6>
<div class="paragraph">
<p>In this example, mutation of FOXO1 serine 256 to alanine is used to block phosphorylation at 256 (S256A), a site phosphorylated by AKT. The S256A mutation was found to impair phosphorylation of threonine 24 and serine 319 by AKT (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11237865">PMID 11237865</a>).
We could represent this observation as follows:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:FOXO1, var(p.Ser256Ala)) =| p(HGNC:FOXO1, pmod(Ph, Ser, 256))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:FOXO1, var(p.Ser256Ala)) =| (p(SFAM:"AKT Family") =&gt; p(HGNC:FOXO1, pmod(Ph, Thr, 24)))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:FOXO1, var(p.Ser256Ala)) =| (p(SFAM:"AKT Family") =&gt; p(HGNC:FOXO1, pmod(Ph, Ser, 319)))</pre>
</div>
</div>
<div class="paragraph">
<p>The first statement indicates that phosphorylation at S256 is blocked by mutation of S256 to alanine. The next two statements indicate that the S256A mutation decreases phosphorylation of FOXO1 threonine 24 and serine 319 by AKT.
However, we are not generally interested in the effects of a lab-created mutant like S256A so much as the role of phosphorylation at serine 256 on phosphorylation of the other two sites. Thus, we recommend the following representation:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:FOXO1, pmod(Ph, Ser, 256)) =&gt;(p(SFAM:"AKT Family") =&gt; p(HGNC:FOXO1, pmod(Ph, Thr, 24)))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:FOXO1, pmod(Ph, Ser, 256)) =&gt;(p(SFAM:"AKT Family") =&gt; p(HGNC:FOXO1, pmod(Ph, Ser, 319)))</pre>
</div>
</div>
<div class="paragraph">
<p>Here, the statements indicate that phosphorylation of FOXO1 at S256 increases the phosphorylation of T24 and S319 by the kinase activity of AKT.
While both representations are accurate, the second version is better suited to integrating other information about the role of FOXO1 phosphorylation at S256 into a cohesive, traversable model.</p>
</div>
</div>
<div class="sect5">
<h6 id="_how_do_i_represent_observations_resulting_from_manipulation_of_two_or_more_entities">How do I represent observations resulting from manipulation of two or more entities?</h6>
<div class="paragraph">
<p>In some cases an experiment has a complex perturbation, where manipulations of multiple biological entities are required for an effect. Multiple BEL abundance terms can be represented together as the subject of a BEL Statement by using the <code>compositeAbundance()</code> or <code>composite()</code> function.</p>
</div>
<div class="paragraph">
<p>In this example, TGF-beta cooperates with IL-6 to generate T-helper 17 cells (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17918200">PMID 17918200</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>composite(p(MGI:Tgfb1), p(MGI:Il6)) -&gt; bp(GOBP:"T-helper 17 cell differentiation")</pre>
</div>
</div>
<div class="paragraph">
<p>If the two manipulated components are known to physically interact (such as a receptor and it&#8217;s ligand), we recommend inferring their effects rather than using a composite term.</p>
</div>
<div class="paragraph">
<p>In this example, both Met and Hgf (the Met ligand) are required for increased expression of integrin Itgav RNA (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16710476">PMID 16710476</a>):</p>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
<div class="paragraph">
<p>Not recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>composite(p(MGI:Hgf), p(MGI:Met)) -&gt; r(MGI:Itgav)</pre>
</div>
</div>
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>Recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>kin(p(MGI:Met) ) -&gt; r(MGI:Itgav)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(MGI:Hgf) -&gt; r(MGI:Itgav)</pre>
</div>
</div>
<div class="paragraph">
<p>Because Hgf binds to and directly activates Met, the effect of Met and Hgf together on Itgav RNA expression can be inferred to result from Met activity.</p>
</div>
</div>
<div class="sect5">
<h6 id="_how_should_i_represent_gene_knock_out_or_rnai_experiments">How should I represent gene knock out or RNAi experiments?</h6>
<div class="paragraph">
<p>Our general practice is to represent the subject term for experiments where the perturbation is a gene deletion or RNAi knockdown as the abundance of the corresponding protein.</p>
</div>
</div>
<div class="sect5">
<h6 id="_gene_knockouts">Gene knockouts</h6>
<div class="paragraph">
<p>In this example, mice with a gene deletion of Nfe2l2 express reduced mRNA of the glutathione S transferase Gsta1 compared to wild-type mice (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11991805">PMID 11991805</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(MGI:Nfe2l2) -&gt; r(MGI:Gsta1)</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_rna_interference">RNA interference</h6>
<div class="paragraph">
<p>In this example, knockdown of PTEN using RNA interference results in increased CDKN1A protein levels (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17300726">PMID 17300726</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:PTEN) -| p(HGNC:CDKN1A)</pre>
</div>
</div>
<div class="paragraph">
<p>We assume that the effects of PTEN RNAi are due to knock down of PTEN protein. Decreased PTEN protein resulting in increased CDKN1A protein is interpreted as PTEN decreases CDKN1A protein.</p>
</div>
<div class="paragraph">
<p>It is generally preferable to represent the subject term as the protein abundance and not an activity of the protein, particularly for 'omic experiments. See also <a href="#_when_should_i_use_the_protein_abundance_vs_the_activity_of_a_protein">When should I use the protein abundance vs. the activity of a protein?</a></p>
</div>
</div>
<div class="sect5">
<h6 id="_how_should_i_represent_overexpression_experiments">How should I represent overexpression experiments?</h6>
<div class="paragraph">
<p>For experiments where the perturbation is overexpression of DNA or RNA for the purpose of overexpressing a protein, we generally represent the subject term as a protein abundance.</p>
</div>
<div class="paragraph">
<p>In this example, SIAH2 and repp86 (TPX2) proteins interact, and overexpression of SIAH2 by transfection increases degradation of TPX2 protein (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17716627">PMID 17716627</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:SIAH2) =&gt; deg(p(HGNC:TPX2))</pre>
</div>
</div>
<div class="paragraph">
<p>The statement is modeled as direct because the subject and object term proteins interact.</p>
</div>
<div class="paragraph">
<p>While it would be technically correct to represent overexpressions achieved via DNA transfection as gene abundances and those from mRNA transfections as RNA abundances, this distinction is not useful for applications like Whistle and pathfinding. It is generally preferable to represent the subject term as the protein abundance and not an activity of the protein, particularly for 'omic experiments, if it is not clear that the activity is required or responsible for the effect.</p>
</div>
</div>
<div class="sect5">
<h6 id="_when_should_i_use_the_protein_abundance_vs_the_activity_of_a_protein">When should I use the protein abundance vs. the activity of a protein?</h6>
<div class="paragraph">
<p>Many experimental perturbations involve the overexpression or knockdown of protein abundance. We generally represent this type of experiment using a protein abundance as the subject term instead of an activity (e.g., kinase or phosphatase) of the protein. While many proteins have a known activity, this activity is not always responsible for all downstream effects of overexpression or knockdown; the same effects occur when either wild-type or catalytically inactive forms are expressed.</p>
</div>
<div class="paragraph">
<p>Below are examples of BEL statements for cases where:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>the effects of increased or decreased protein abundance are not due to the catalytic activity of the protein,</p>
</li>
<li>
<p>effects are likely due to an increase or decrease in the activity of the protein, and</p>
</li>
<li>
<p>not enough information is available.</p>
</li>
</ol>
</div>
</div>
<div class="sect5">
<h6 id="_effects_are_not_due_to_the_catalytic_activity_of_the_protein">Effects are not due to the catalytic activity of the protein</h6>
<div class="paragraph">
<p><em>Example 1.</em> A mutant telomerase (TERT) protein lacking telomerase activity retains its effects on keratinocyte proliferation (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18208333">PMID 18208333</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(MGI:Tert) -&gt; bp(GOBP:"cell proliferation")</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Tert), ma(cat)) causesNoChange bp(GOBP:"cell proliferation")</pre>
</div>
</div>
<div class="paragraph">
<p><em>Example 2.</em> The serine-threonine kinase RIPK1 activates NF-kappaB through a mechanism that does not involve the protein&#8217;s kinase activity (<a href="http://www.ncbi.nlm.nih.gov/pubmed/20354226">PMID 20354226</a>).</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:RIPK1) -&gt; act(p(GOCC:"NF-kappaB complex"), ma(tscript))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:RIPK1), ma(kin)) causesNoChange act(p(GOCC:"NF-kappaB complex"), ma(tscript))</pre>
</div>
</div>
</div>
<div class="sect5">
<h6 id="_effects_are_likely_due_to_the_activity_of_the_protein">Effects are likely due to the activity of the protein</h6>
<div class="paragraph">
<p><em>Example.</em> Genes are differentially expressed in wild-type vs. Met knock out mouse hepatocytes, only after treatment with Hgf, the Met ligand (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16710476">PMID 16710476</a>). These genes include integrins Itgav, Itga3, and Itgb1.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Met), ma(kin)) -&gt; r(Mgi:Itgav)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Met), ma(kin)) -&gt; r(Mgi:Itga3)</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(MGI:Met), ma(kin)) -&gt; r(Mgi:Itgb1)</pre>
</div>
</div>
<div class="paragraph">
<p>Because both Met and the Met ligand are required for the increase in gene expression, we use the activity of Met as the subject term.</p>
</div>
</div>
<div class="sect5">
<h6 id="_not_enough_information_is_available">Not enough information is available</h6>
<div class="paragraph">
<p><em>Example.</em> Genes are differentially expressed in the pancreas of mice with a pancreas-specific beta-catenin deletion (<a href="http://www.ncbi.nlm.nih.gov/pubmed/17222338">PMID 17222338</a>). These include decreased expression of the hedgehog interacting protein Hhip in the knock-out compared to wild-type.</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(MGI:Ctnnb1) -&gt; r(MGI:Hhip)</pre>
</div>
</div>
<div class="paragraph">
<p>In this case, not enough information is available to determine if the change in Hhip expression is due to Ctnnb1 function in transcription, cell adhesion, or another role.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_relationships">Relationships</h5>
<div class="sect5">
<h6 id="_when_should_i_use_a_correlative_relationship">When should I use a correlative relationship?</h6>
<div class="paragraph">
<p><a href="#XcorRels">Correlative relationships</a> are more appropriate than <a href="#_causal_relationships">causal relationships</a> to represent observations that do not clearly result from an experimental perturbation. BEL causal relationships include <code>increases</code>, <code>decreases</code>, <code>directlyIncreases</code>, <code>directlyDecreases</code>, and <code>regulates</code>. Correlative relationships include <code>positiveCorrelation</code> and <code>negativeCorrelation</code>.</p>
</div>
</div>
<div class="sect5">
<h6 id="_correlative">Correlative</h6>
<div class="paragraph">
<p>If the observation comes from the comparison of human tumors grouped by the occurrence of a specific mutation, then the relationship should generally be expressed as correlative. In this case there is no experimental perturbation.
In this example, most patient tumor samples with an EGFR L858R mutation were observed to exhibit a reduction in ERBB2 tyrosine 1248 phosphorylation compared to wild-type samples (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18687633">PMID 18687633</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:EGFR, var(p.Leu858Arg)) negativeCorrelation p(HGNC:ERBB2, pmod(Ph,Tyr,1248))</pre>
</div>
</div>
<div class="paragraph">
<p>In this case, no evidence is presented to suggest that the differences in ERBB2 phosphorylation are causally related to the EGFR mutation, only that the two observations are inversely correlated. Note that the subject and object terms are interchangeable for correlative relationships.</p>
</div>
</div>
<div class="sect5">
<h6 id="_causal_3">Causal</h6>
<div class="paragraph">
<p>If the observation comes from the comparison of experimentally controlled states, like gene deletion, overexpression, or introduction of a mutant allele into a cell line or animal, the experimental perturbation can generally be represented as the subject term of a causal statement.
In this example, DUSP6 RNA is observed to be upregulated in immortalized human bronchial epithelial cells transfected with EGFR mutant L858R, as compared to WT EGFR (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16489012">PMID 16489012</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:EGFR, var(p.Leu858Arg)) -&gt; r(HGNC:DUSP6)</pre>
</div>
</div>
<div class="paragraph">
<p>In this case, the EGFR mutation is introduced as an experimentally-controlled perturbation.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_object_terms_measurements">Object Terms (Measurements)</h5>
<div class="sect5">
<h6 id="_how_should_i_represent_microarray_data">How should I represent microarray data?</h6>
<div class="paragraph">
<p>We record the results of experiments like microarrays and RT-PCR, which measure RNA abundances, by representing the object terms as RNA abundances. Only significant effects (e.g., meeting minimum criteria for fold change and statistical significance) should be recorded in BEL Statements.</p>
</div>
<div class="paragraph">
<p>In a causal BEL Statement, the subject term generally represents an experimentally manipulated entity while the object term represents a measured entity. Our general practice is to represent the object terms in BEL Statements with the terms most closely related to the experimental measurement.</p>
</div>
<div class="paragraph">
<p>This direct representation of the measurement in BEL supports the creation of KAMs to which 'omic data can be mapped directly and analyzed using automated reasoning applications like Whistle. Inference of the potential downstream consequences of RNA expression changes is supported by connection of RNA abundances to the corresponding proteins during knowledge network compilation</p>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_statement_annotations">4.3.2. Statement Annotations</h4>
<div class="sect4">
<h5 id="_how_do_i_annotate_a_relationship_observed_in_multiple_biological_contexts">How do I annotate a relationship observed in multiple biological contexts?</h5>
<div class="paragraph">
<p>Often, the scientific literature reports a relationship as occurring across several biological contexts.</p>
</div>
<div class="paragraph">
<p>Our general practice is to represent each observation with a separate statement. Several annotations can be used to describe the same context, e.g., 'lung' and 'fibroblast', but distinct BEL statements should be used to describe each experimental context that the relationship is observed in.</p>
</div>
<div class="sect5">
<h6 id="_example_19">Example</h6>
<div class="paragraph">
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/18650932">PMID 18650932</a> - siRNA knockdown of the atypical PKC-interacting protein Par-4 (PAWR) increases phosphorlyation of AKT at Serine 473 in both human 293 and A549 cells.</p>
</div>
<div class="paragraph">
<p>"To test whether this is also true in human cells, we used a Par-4 siRNA to deplete endogenous Par-4 levels in human 293 cells and in the A549 human lung adenocarcinoma cell line. Cells were treated with control or Par-4-specific siRNAs, after which they were kept for 24 h in serum-free medium conditions and then stimulated with serum. Data in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519103/figure/f5/">Figure 5E and F</a> clearly demonstrate that the knockdown of Par-4 provokes enhanced serum-activated phospho-Akt-Ser473 levels in A549 and 293 human cells, respectively."</p>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
<div class="paragraph">
<p>Not recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET CellLine = {A549, 293}</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:PAWR) -| p(SFAM:"AKT Family", pmod(Ph, Ser, 473))</pre>
</div>
</div>
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>Recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>SET CellLine = A549</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:PAWR) -| p(SFAM:"AKT Family", pmod(Ph, Ser, 473))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>SET CellLine</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:PAWR) -| p(SFAM:"AKT Family", pmod(Ph, Ser, 473))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_modified_proteins">4.3.3. Modified Proteins</h4>
<div class="ulist">
<ul>
<li>
<p><a href="#_how_do_i_represent_a_protein_modification_when_specific_information_is_not_available">How do I represent a protein modification when specific information is not available?</a></p>
</li>
<li>
<p><a href="#_how_do_i_represent_a_protein_modification_within_a_complex">How do I represent a protein modification within a complex?</a></p>
</li>
<li>
<p><a href="#Xmultphosphforprotein">How do I represent a situation where multiple phosphorylations are required for a protein&#8217;s activity?</a></p>
</li>
<li>
<p><a href="#_how_do_i_represent_a_situation_where_one_protein_modification_initiates_additional_modifications">How do I represent a situation where one protein modification initiates additional modifications?</a></p>
</li>
<li>
<p><a href="#Xremovalofproteinmod">How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_how_do_i_represent_a_protein_modification_when_specific_information_is_not_available">How do I represent a protein modification when specific information is not available?</h5>
<div class="paragraph">
<p>BEL terms for <a href="#_protein_modifications">post-translational modifications</a> of proteins specify the type of modification, the modified amino acid, and the position of the modified amino acid. The modified amino acid and position are not required, so protein modifications can be represented with less specific information.</p>
</div>
<div class="sect5">
<h6 id="_example_20">Example</h6>
<div class="paragraph">
<p>Human AKT1 protein modified by phosphorylation at serine 473</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(Ph, Ser, 473))</pre>
</div>
</div>
<div class="paragraph">
<p>Human AKT1 protein modified by phosphorylation at an unspecified serine residue</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(Ph, Ser))</pre>
</div>
</div>
<div class="paragraph">
<p>Human AKT1 protein that has been modified by phosphorylation at an unspecified amino acid residue</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:AKT1, pmod(Ph))</pre>
</div>
</div>
<div class="paragraph">
<p>As a general rule, if specific information is available, it should be used. In some cases, this involves investigation sections of a paper outside of the evidence text or other referenced papers to determine which specific modifications have been measured.</p>
</div>
<div class="paragraph">
<p>Non-specific protein modification terms have limited value in the context of a knowledge network. For example, phosphorylation at different sites of the same protein can have opposing effects.
For example: "Akt-phosphorylated FOXO interacts with the ubiquitin ligase Skp2 and is targeted for proteasomal degradation" (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15917664">PMID 15917664</a>)</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_21">Example</h6>
<div class="paragraph">
<p>Recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(SFAM:"AKT Family"), ma(kin)) =&gt; (act(p(HGNC:SKP2)) =&gt; deg(p(SFAM:"FOXO Family")))</pre>
</div>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
<div class="paragraph">
<p>Not recommended:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(SFAM:"FOXO Family", pmod(Ph)) =&gt; (act(p(HGNC:SKP2)) =&gt; deg(p(SFAM:"FOXO Family")))</pre>
</div>
</div>
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>The first BEL Statement indicates that the kinase activity of AKT increases the degradation of FOXO by SKP2. The second statement indicates that phosphorylation of FOXO increases the degradation of FOXO by SKP2. In this case, more information is captured by using the phosphorylating kinase AKT as the subject term instead of the non-specified phosphorylation of FOXO.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_how_do_i_represent_a_protein_modification_within_a_complex">How do I represent a protein modification within a complex?</h5>
<div class="paragraph">
<p>In many cases, complexes include proteins with post-translational modifications and these modifications influence complex formation. For example, HIF1A that has been hydroxylated on proline residues 402 and 564 interacts with VHL (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17925579">PMID 17925579</a>).</p>
</div>
<div class="paragraph">
<p>Our general practice is to represent this type of event as a causal statement in BEL, with the modified protein as the subject term and the complex with no specified modifications as the object term. Because the modified protein is a component of the complex, we use a direct causal relationship:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:HIF1A, pmod(Hy, Pro, 402)) =&gt; complex(p(HGNC:HIF1A), p(HGNC:VHL))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:HIF1A, pmod(Hy, Pro, 564)) =&gt; complex(p(HGNC:HIF1A), p(HGNC:VHL))</pre>
</div>
</div>
<div class="admonitionblock important">
<table>
<tr>
<td class="icon">
<div class="title">Important</div>
</td>
<td class="content">
<div class="paragraph">
<p>While BEL allows representation of complexes with the modified proteins as components, we do not recommend this approach:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:HIF1A, pmod(Hy, Pro, 402)), p(HGNC:VHL))</pre>
</div>
</div>
</td>
</tr>
</table>
</div>
<div class="paragraph">
<p>The practice of composing a complex using protein abundances without any specified modifications provides a standardized representation for complexes and allows the effects of modifications on complex formation to be captured as causal relationships. The modified forms of HIF1A, <code>p(HGNC:HIF1A, pmod(Hy, Pro, 402))</code> and <code>p(HGNC:HIF1A, pmod(Hy, Pro, 564))</code> are considered a subset of the total <code>p(HGNC:HIF1A)</code>.</p>
</div>
<div class="paragraph">
<p>This approach enables representation of the effects of multiple protein modifications on complex formation by using a causal statement for each modification.</p>
</div>
<div class="admonitionblock warning">
<table>
<tr>
<td class="icon">
<div class="title">Warning</div>
</td>
<td class="content">
<div class="paragraph">
<p>BELv2.0 does not provide a specific representation of unmodified protein abundances.</p>
</div>
</td>
</tr>
</table>
</div>
<div class="sect5">
<h6 id="__strong_exception_modified_histones_bound_to_the_promoter_of_a_specific_gene_strong"><strong>Exception  modified histones bound to the promoter of a specific gene</strong></h6>
<div class="paragraph">
<p>One exception to our general practice of not specifying protein modifications within complex abundances is the interaction of specific modified histones with the promoter of a specific gene. In this example, cigarette smoke is observed to increase H3K27me3 levels at the DKK1 promoter (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19351856">PMID 19351856</a>).</p>
</div>
<div class="literalblock">
<div class="content">
<pre>a(SCHEM:"smoke condensate, cigarette (gas phase)") -&gt; \
   complex(p(PFH:"Histone H3 Family",pmod(Me3, Lys, 27)), g(HGNC:DKK1))</pre>
</div>
</div>
<div class="paragraph">
<p>In this example, the modification of the histone by trimethylation does not affect its binding to the gene DKK1. In addition, cigarette smoke does not increase or decrease the overall abundance of the modified histone, only the abundance of the modified histone at the DKK1 promoter.</p>
</div>
</div>
</div>
<div class="sect4">
<h5 id="Xmultphosphforprotein">How do I represent a situation where multiple phosphorylations are required for a protein&#8217;s activity?</h5>
<div class="paragraph">
<p>In many cases two or more distinct modifications are required simultaneously for protein activity, and neither modification alone is sufficient. For example, MAPK3 must be phosphorylated at two sites, Threonine 202 and Tyrosine 204, to be active. Our general practice is to take the simple, most general approach, and model the effect of each site separately:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:MAPK3, pmod(Ph, Thr, 202)) =&gt; act(p(HGNC:MAPK3), ma(kin))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:MAPK3, pmod(Ph, Tyr, 204)) =&gt; act(p(HGNC:MAPK3), ma(kin))</pre>
</div>
</div>
<div class="paragraph">
<p>A multiply-modified abundance term can be used if it is of high importance to capture the requirement for both modifications:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:MAPK3, pmod(Ph, Thr, 202)), pmod(Ph, Tyr, 204)) =&gt; act(p(HGNC:MAPK3), ma(kin))</pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_how_do_i_represent_a_situation_where_one_protein_modification_initiates_additional_modifications">How do I represent a situation where one protein modification initiates additional modifications?</h5>
<div class="paragraph">
<p>In many cases a specific protein modification may be dependent on another modification of the same protein. In this case, the first protein modification can be modeled as the upstream cause of the second. In this example, phosphorylation of CTNNB1 at Serine 45 initiates phosphorylation of CTNNB1 at other sites including Threonine 41 by GSK3 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16618120">PMID 16618120</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CTNNB1, pmod(Ph, Ser, 45)) =&gt; p(HGNC:CTNNB1, pmod(Ph, Thr, 41))</pre>
</div>
</div>
<div class="paragraph">
<p>We represent this relationship as direct, because the subject and object terms have the same root abundance node.</p>
</div>
<div class="paragraph">
<p>Because the kinase mediating the second phosphorylation is known, this relationship can be modeled alternatively as a nested statement:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>p(HGNC:CTNNB1, pmod(Ph, Ser, 45)) =&gt; \
   (kin(p(SFAM:"GSK3 Family")) =&gt; p(HGNC:CTNNB1, pmod(Ph, Thr, 41)))</pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="Xremovalofproteinmod">How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?</h5>
<div class="paragraph">
<p>Removal of a specific protein modification is represented simply as a decrease in the abundance of the modified protein.</p>
</div>
<div class="sect5">
<h6 id="_deubiquitination">Deubiquitination</h6>
<div class="paragraph">
<p>In this example, STAMBP deubiquitinates F2RL1 protein (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19684015">PMID 19684015</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:STAMBP)) =| p(HGNC:F2RL1, pmod(Ub))</pre>
</div>
</div>
<div class="paragraph">
<p>Deubiquitination is represented simply as a decrease in the ubiquitinated form of the protein. Because in this example STAMBP is the deubiquitinating enzyme, we used a directlyDecreases relationship.</p>
</div>
</div>
<div class="sect5">
<h6 id="_dephosphorylation">Dephosphorylation</h6>
<div class="paragraph">
<p>In this example, the phosphatase CDC25C dephosphorylates CDK1 at tyrosine 15 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1384126">PMID 1384126</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:CDC25C), ma(phos)) =| p(HGNC:CDK1, pmod(Ph, Tyr, 15))</pre>
</div>
</div>
<div class="paragraph">
<p>Similar to deubiquitination, dephosphorylation is represented simply as a decrease in the modified form of the protein.</p>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_reactions_2">4.3.4. Reactions</h4>
<div class="ulist">
<ul>
<li>
<p><a href="#_how_can_i_represent_a_reversible_metabolic_reaction">How can I represent a reversible metabolic reaction?</a></p>
</li>
<li>
<p><a href="#_when_and_why_should_i_use_a_reaction_term">When and why should I use a reaction term?</a></p>
</li>
</ul>
</div>
<div class="sect4">
<h5 id="_how_can_i_represent_a_reversible_metabolic_reaction">How can I represent a reversible metabolic reaction?</h5>
<div class="paragraph">
<p>A reversible reaction can be represented by modeling the reaction with the products and reactants interchanged.</p>
</div>
<div class="paragraph">
<p>For example, HSD11B1 acts primarily to convert cortisone to active cortisol, but in some cell types the reverse reaction is favored (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12530648">PMID 12530648</a>):</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:HSD11B1), ma(cat)) =&gt; \
 rxn(reactants(a(CHEBI:NADPH), a(CHEBI:cortisone)), products(a(CHEBI:"NADP(+)"), a(CHEBI:cortisol)))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:HSD11B1), ma(cat)) =&gt; \
 rxn(reactants(a(CHEBI:"NADP(+)"), a(CHEBI:cortisol)), products(a(CHEBI:NADPH), a(CHEBI:cortisone)))</pre>
</div>
</div>
<div class="paragraph">
<p>The top statement represents the forward reaction and the bottom statement represents the reverse reaction.</p>
</div>
</div>
<div class="sect4">
<h5 id="_when_and_why_should_i_use_a_reaction_term">When and why should I use a reaction term?</h5>
<div class="sect5">
<h6 id="_when_should_i_use_a_reaction_term">When_ should I use a reaction term</h6>
<div class="paragraph">
<p>Reaction terms allow the representation of a transformation of a list of reactants into a list of products. In this example, the superoxide dismutase SOD1 converts superoxide to hydrogen peroxide:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:SOD1), ma(cat)) =&gt; rxn(reactants(a(CHEBI:superoxide)), products(a(CHEBI:"hydrogen peroxide")))</pre>
</div>
</div>
<div class="paragraph">
<p>It is not necessary to include all reactants and products, especially if they are ubiquitous small molecules. In the above example the reactant hydrogen and product oxygen have been omitted from the reaction.</p>
</div>
</div>
<div class="sect5">
<h6 id="_why_should_i_use_a_reaction_term">Why_ should I use a reaction term</h6>
<div class="paragraph">
<p>It is possible to represent the above reaction with separate statements linking the activity of SOD1 to decreased abundances of the reactants and increased abundances of the products:</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(HGNC:SOD1), ma(cat)) =| a(CHEBI:superoxide)
act(p(HGNC:SOD1), ma(cat)) =&gt; a(CHEBI:"hydrogen peroxide")</pre>
</div>
</div>
<div class="paragraph">
<p>While this representation describes the function of the catalytic enzyme SOD1, it does not link the product hydrogen peroxide to the reactant superoxide.</p>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_protein_protein_interactions_2">4.3.5. Protein-Protein Interactions</h4>
<div class="sect4">
<h5 id="_how_do_i_represent_a_physical_interaction_between_two_entities">How do I represent a physical interaction between two entities?</h5>
<div class="paragraph">
<p>You can use a complex abundance to represent binding events between two or more abundance terms. The subject term of your BEL Statement will be the complex. BEL Statements do not require a relationship and object term.</p>
</div>
<div class="admonitionblock warning">
<table>
<tr>
<td class="icon">
<div class="title">Warning</div>
</td>
<td class="content">
<div class="paragraph">
<p>The order in which the members of the complex are listed is not important.</p>
</div>
</td>
</tr>
</table>
</div>
<div class="sect5">
<h6 id="_examples_10">Examples</h6>
<div class="paragraph">
<p>EPOR physically interacts with CSF2RB, the common beta-receptor (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15456912">PMID 15456912</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(MGI:EPOR), p(MGI:CSF2RB))</pre>
</div>
</div>
<div class="paragraph">
<p>KEAP1 binds 15-deoxy-Delta(12,14)-prostaglandin J2 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15917255">PMID 15917255</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:KEAP1), a(CHEBI:"15-deoxy-Delta(12,14)-prostaglandin J2"))</pre>
</div>
</div>
<div class="paragraph">
<p>KEAP1, CUL3, and RBX1 copurify and are part of a functional E3 ubiquitin ligase complex (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15572695">PMID 15572695</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(HGNC:KEAP1), p(HGNC:RBX1), p(HGNC:CUL3))</pre>
</div>
</div>
<div class="paragraph">
<p>The AP-1 transcription complex binds the CCL23 promoter (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17368823">PMID 17368823</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(GOCC:"AP1 complex"), g(HGNC:CCL23))</pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect3">
<h4 id="_protein_families_2">4.3.6. Protein Families</h4>
<div class="sect4">
<h5 id="_when_should_i_use_a_protein_family_instead_of_a_specific_protein">When should I use a protein family instead of a specific protein?</h5>
<div class="paragraph">
<p>Protein families can be used to represent protein abundances in cases where the information presented by the source does not allow identification of the specific protein. For example:</p>
</div>
<div class="sect5">
<h6 id="_example_1">Example 1:</h6>
<div class="paragraph">
<p>"Akt physically associates with MDM2 and phosphorylates it at Ser166 and Ser186." (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11715018">PMID 11715018</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(SFAM:"AKT Family"), ma(kin)) =&gt; p(HGNC:MDM2, pmod(Ph,Ser,166))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(SFAM:"AKT Family"), ma(kin)) =&gt; p(HGNC:MDM2, pmod(Ph,Ser,186))</pre>
</div>
</div>
<div class="paragraph">
<p>Here, Akt may refer to AKT1, AKT2, and/or AKT3.</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_2_2">Example 2:</h6>
<div class="paragraph">
<p>"We show that Siah2 is subject to phosphorylation by p38 MAPK &#8230;&#8203; Phosphopeptide mapping identified T24 and S29 as the primary phospho-acceptor sites." (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17003045">PMID 17003045</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(PFH:"MAPK p38 Family"), ma(kin)) =&gt; p(HGNC:SIAH2, pmod(Ph, Thr, 24))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>act(p(PFH:"MAPK p38 Family"), ma(kin)) =&gt; p(HGNC:SIAH2, pmod(Ph, Ser, 29))</pre>
</div>
</div>
<div class="paragraph">
<p>Here, it is not clear which specific p38 MAPK is responsible (MAPK11, MAPK12, MAPK13, or MAPK14).</p>
</div>
</div>
<div class="sect5">
<h6 id="_example_3_2">Example 3:</h6>
<div class="paragraph">
<p>"Hip encodes a membrane glycoprotein that binds to all three mammalian Hedgehog proteins." (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10050855">PMID 10050855</a>)</p>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(SFAM:"Hedgehog Family"), p(MGI:Hhip))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(MGI:Ihh), p(MGI:Hhip))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(MGI:Shh), p(MGI:Hhip))</pre>
</div>
</div>
<div class="literalblock">
<div class="content">
<pre>complex(p(MGI:Dhh), p(MGI:Hhip))</pre>
</div>
</div>
<div class="paragraph">
<p>In this case, all three hedgehog family members are reported to bind to the hedgehog interacting protein (Hhip). Statements can be modeled using the family as well as each individual member.</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_implementation">5. Implementation</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_formats">5.1. Formats</h3>
<div class="sect3">
<h4 id="_bel_script">5.1.1. BEL Script</h4>

</div>
<div class="sect3">
<h4 id="_xbel">5.1.2. XBEL</h4>

</div>
<div class="sect3">
<h4 id="_evidence_json">5.1.3. Evidence JSON</h4>

</div>
<div class="sect3">
<h4 id="_json_graph_format_jgf">5.1.4. JSON Graph Format (JGF)</h4>

</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_tools">6. Tools</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_java">6.1. Java</h3>

</div>
<div class="sect2">
<h3 id="_ruby">6.2. Ruby</h3>
<div class="dlist">
<dl>
<dt class="hdlist1">BEL Parameter</dt>
<dd>
<p>The corresponding (indented) definition.</p>
</dd>
<dt class="hdlist1">BEL Term</dt>
<dd>
<p>The corresponding (indented) definition.</p>
</dd>
<dt class="hdlist1">BEL Statement</dt>
<dd>
<p>The corresponding (indented) definition.</p>
</dd>
</dl>
</div>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Version 2.0<br>
Last updated 2016-05-23 19:16:12 UTC
</div>
</div>
</body>
</html>